<!DOCTYPE html>
<html>
  <head>
    <meta charset="utf-8">
    <title>Puglisi Lab</title>
    <link href="https://fonts.googleapis.com/css?family=Open+Sans+Condensed:300" rel="stylesheet">
    <link href="https://fonts.googleapis.com/css?family=Arimo" rel="stylesheet">
    <script src="https://use.fontawesome.com/7ccc97b483.js"></script>
    <script type="text/javascript" src="jquery-3.2.0.min.js"></script>
    <!-- <link rel="stylesheet" href="assets/reset.css" /> -->
    <link rel="stylesheet" href="assets/main.css" />
    <!-- <link rel="stylesheet" href="assets/publications.css" /> -->
  </head>
  <body>
    <div id="page-wrapper">

      <header class="header">
        <h1 id="logo"><a href="index.html">
          <div id="logo-img">
          <img src="images/Logo_of_Stanford_University.png" height="50" width="50"></div>
          <div id="puglisi">Puglisi</div>
          <div id="laboratory">Lab</div></a></h1>
          <div id="bars"><i class="fa fa-bars fa-2x" aria-hidden="true"></i>
            <div class="bars-nav" class="hidden">
              <a href="research.html"><ul id="nav-title">Research</ul></a>
              <a href="people.html"><ul id="nav-title">People</ul></a>
              <a href="publications.html"><ul id="nav-title">Publications</ul></a>
              <a href="abroad.html"><ul id="nav-title">Abroad</ul></a>
              <a href="contact.html"><ul id="nav-title">Contact</ul></a>
            </div></div>

        <div class="nav">
          <a href="research.html"><ul id="nav-title">Research</ul></a>
          <a href="people.html"><ul id="nav-title">People</ul></a>
          <a href="publications.html"><ul id="nav-title">Publications</ul></a>
          <a href="abroad.html"><ul id="nav-title">Abroad</ul></a>
          <a href="contact.html"><ul id="nav-title">Contact</ul></a>
        </div>
      </header>

      <div id="background-photo">
        <div class="welcome 1">Our Publications</div></br>
      </div>

      <div id="publication-header-container">
        <div id="publication-years">Publication Years </div>
        <div id="years-list">
          <div id="first-half">
          <a href="#2016">2016</a>
          <a href="#2015">2015</a>
          <a href="#2014">2014</a>
          <a href="#2013">2013</a>
          <a href="#2012">2012</a>
          <a href="#2011">2011</a>
          <a href="#2010">2010</a>
          <a href="#2009">2009</a>
          <a href="#2008">2008</a>
          <a href="#2007">2007</a>
          <a href="#2006">2006</a>
          <a href="#2005">2005</a>
          <a href="#2004">2004</a>
          <a href="#2003">2003</a>
          <a href="#2002">2002</a>
        </div></br>
          <div id="second-half">
            <a href="#2001">2001</a>
            <a href="#2000">2000</a>
          <a href="#1999">1999</a>
          <a href="#1998">1998</a>
          <a href="#1997">1997</a>
          <a href="#1996">1996</a>
          <a href="#1995">1995</a>
          <a href="#1994">1994</a>
          <a href="#1993">1993</a>
          <a href="#1992">1992</a>
          <a href="#1991">1991</a>
          <a href="#1990">1990</a>
          <a href="#1989">1989</a>
          <a href="#1987">1987</a>
          <a href="#1986">1986</a></div>
        </div>

      </div>

      <div id="publications-wrapper">
        <div id="year-wrapper">

          <div class="year" id="2016">2016</div>
          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Coey, A., Larsen, K., Puglisi, J. D., Puglisi, E. V.</span>
            <a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/2016-RNA-Coey.pdf" download><span id="title">Heterogeneous structures formed by conserved RNA sequences within the HIV reverse transcription initiation site</span></a>
            <span id="publisher">RNA, 2016; 22 (11): 1689-1698</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/2016-RNA-Coey.pdf" download><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span></a>
Reverse transcription is a key process in the early steps of HIV infection. This process initiates within a specific complex formed by the 5' UTR of the HIV genomic RNA (vRNA) and a host primer tRNA(Lys)3 Using nuclear magnetic resonance (NMR) spectroscopy and single-molecule fluorescence spectroscopy, we detect two distinct conformers adopted by the tRNA/vRNA initiation complex. We directly show that an interaction between the conserved 8-nucleotide viral RNA primer activation signal (PAS) and the primer tRNA occurs in one of these conformers. This intermolecular PAS interaction likely induces strain on a vRNA intramolecular helix, which must be broken for reverse transcription to initiate. We propose a mechanism by which this vRNA/tRNA conformer relieves the kinetic block formed by the vRNA intramolecular helix to initiate reverse transcription.
              <div id="webof-science"></div>
              <div id="pubmed"></div>
              </div>

            </div>
          </div>



          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Navon, S. P., Kornberg, G., Chen, J., Schwartzman, T., Tsai, A., Puglisi, E. V., Puglisi, J. D., Adir, N.</span>
            <a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/2016-PNAS-Navon-AminoAcidSequence.pdf" download><span id="title">Amino acid sequence repertoire of the bacterial proteome and the occurrence of untranslatable sequences</span></a>
            <span id="publisher"> PROCEEDINGS OF THE NATIONAL ACADEMY OF SCIENCES OF THE UNITED STATES OF AMERICA, 2016; 113 (26): 7166-7170</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/2016-PNAS-Navon-AminoAcidSequence.pdf" download><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span></a>
                Bioinformatic analysis of Escherichia coli proteomes revealed that all possible amino acid triplet sequences occur at their expected frequencies, with four exceptions. Two of the four underrepresented sequences (URSs) were shown to interfere with translation in vivo and in vitro. Enlarging the URS by a single amino acid resulted in increased translational inhibition. Single-molecule methods revealed stalling of translation at the entrance of the peptide exit tunnel of the ribosome, adjacent to ribosomal nucleotides A2062 and U2585. Interaction with these same ribosomal residues is involved in regulation of translation by longer, naturally occurring protein sequences. The E. coli exit tunnel has evidently evolved to minimize interaction with the exit tunnel and maximize the sequence diversity of the proteome, although allowing some interactions for regulatory purposes. Bioinformatic analysis of the human proteome revealed no underrepresented triplet sequences, possibly reflecting an absence of regulation by interaction with the exit tunnel.
                <div id="webof-science"></div>
                <div id="pubmed"></div>

              </div>

            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Chen, J., Choi, J., O'Leary, S. E., Prabhakar, A., Petrov, A., Grosely, R., Puglisi, E. V., Puglisi, J. D.</span>
            <a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/2016-QRB-Chen-MolecularChoreography.pdf" download><span id="title">The molecular choreography of protein synthesis: translational control, regulation, and pathways </span></a>
            <span id="publisher">QUARTERLY REVIEWS OF BIOPHYSICS, 2016; 49</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/2016-QRB-Chen-MolecularChoreography.pdf" download><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span></a>
                Translation of proteins by the ribosome regulates gene expression, with recent results underscoring the importance of translational control. Misregulation of translation underlies many diseases, including cancer and many genetic diseases. Decades of biochemical and struc- tural studies have delineated many of the mechanistic details in prokaryotic translation, and sketched the outlines of eukaryotic translation. However, translation may not proceed linearly through a single mechanistic pathway, but likely involves multiple pathways and branchpoints. The stochastic nature of biological processes would allow different pathways to occur during translation that are biased by the interaction of the ribosome with other translation factors, with many of the steps kinetically controlled. These multiple pathways and branchpoints are po- tential regulatory nexus, allowing gene expression to be tuned at the translational level. As research focus shifts toward eukaryotic translation, certain themes will be echoed from studies on prokaryotic translation. This review provides a general overview of the dynamic data related to prokaryotic and eukaryotic translation, in particular recent findings with single-molecule methods, complemented by biochemical, kinetic, and structural findings. We will underscore the importance of viewing the process through the viewpoints of regulation, translational control, and heterogeneous pathways.
              <div id="webof-science"></div>
              <div id="pubmed"></div>
              </div>
            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Petrov, A., Grosely, R., Chen, J., O'Leary, S. E., Puglisi, J. D.</span>
            <a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/2016-MolecularCell-Petrov_MultipleParallel.pdf" download><span id="title">Multiple Parallel Pathways of Translation Initiation on the CrPV IRES.</span></a>
            <span id="publisher">Molecular cell, 2016; 62 (1): 92-103</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/2016-MolecularCell-Petrov_MultipleParallel.pdf" download><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span></a>
                The complexity of eukaryotic translation allows fine-tuned regulation of protein synthesis. Viruses use internal ribosome entry sites (IRESs) to minimize or, like the CrPV IRES, eliminate the need for initiation factors. Here, by exploiting the CrPV IRES, we observed the entire process of initiation and transition to elongation in real time. We directly tracked the CrPV IRES, 40S and 60S ribosomal subunits, and tRNA using single-molecule fluorescence spectroscopy and identified multiple parallel initiation pathways within the system. Our results distinguished two pathways of 80S:CrPV IRES complex assembly that produce elongation-competent complexes. Following 80S assembly, the requisite eEF2-mediated translocation results in an unstable intermediate that is captured by binding of the elongator tRNA. Whereas initiation can occur in the 0 and +1 frames, the arrival of the first tRNA defines the reading frame and strongly favors 0 frame initiation. Overall, even in the simplest system, an intricate reaction network regulates translation initiation.

              <div id="webof-science"></div>
              <div id="pubmed"></div>
              </div>
            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Choi, J., Ieong, K., Demirci, H., Chen, J., Petrov, A., Prabhakar, A., O'Leary, S. E., Dominissini, D., Rechavi, G., Soltis, S. M., Ehrenberg, M., Puglisi, J. D.</span>
            <a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/2015_NSMB_Choi_N6Methyladenosine.pdf" download><span id="title">N(6)-methyladenosine in mRNA disrupts tRNA selection and translation-elongation dynamics.</span></a>
            <span id="publisher">Nature structural & molecular biology, 2016; 23 (2): 110-115</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/2015_NSMB_Choi_N6Methyladenosine.pdf" download><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span></a>
N(6)-methylation of adenosine (forming m(6)A) is the most abundant post-transcriptional modification within the coding region of mRNA, but its role during translation remains unknown. Here, we used bulk kinetic and single-molecule methods to probe the effect of m(6)A in mRNA decoding. Although m(6)A base-pairs with uridine during decoding, as shown by X-ray crystallographic analyses of Thermus thermophilus ribosomal complexes, our measurements in an Escherichia coli translation system revealed that m(6)A modification of mRNA acts as a barrier to tRNA accommodation and translation elongation. The interaction between an m(6)A-modified codon and cognate tRNA echoes the interaction between a near-cognate codon and tRNA, because delay in tRNA accommodation depends on the position and context of m(6)A within codons and on the accuracy level of translation. Overall, our results demonstrate that chemical modification of mRNA can change translational dynamics.
              <div id="webof-science"></div>
              <div id="pubmed"></div>
              </div>
            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Tsai, A., Puglisi, J. D., Uemura, S.</span>
            <a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/2016-PMBTS-Tsai-ProbingtheTranslational.pdf" download><span id="title">Probing the Translation Dynamics of Ribosomes Using Zero-Mode Waveguides. </span></a>
            <span id="publisher">Progress in molecular biology and translational science, 2016; 139: 1-43</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/2016-PMBTS-Tsai-ProbingtheTranslational.pdf" download><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span></a>
                In order to coordinate the complex biochemical and structural feat of converting triple-nucleotide codons into their corresponding amino acids, the ribosome must physically manipulate numerous macromolecules including the mRNA, tRNAs, and numerous translation factors. The ribosome choreographs binding, dissociation, physical movements, and structural rearrangements so that they synergistically harness the energy from biochemical processes, including numerous GTP hydrolysis steps and peptide bond formation. Due to the dynamic and complex nature of translation, the large cast of ligands involved, and the large number of possible configurations, tracking the global time evolution or dynamics of the ribosome complex in translation has proven to be challenging for bulk methods. Conventional single-molecule fluorescence experiments on the other hand require low concentrations of fluorescent ligands to reduce background noise. The significantly reduced bimolecular association rates under those conditions limit the number of steps that can be observed within the time window available to a fluorophore. The advent of zero-mode waveguide (ZMW) technology has allowed the study of translation at near-physiological concentrations of labeled ligands, moving single-molecule fluorescence microscopy beyond focused model systems into studying the global dynamics of translation in realistic setups. This chapter reviews the recent works using the ZMW technology to dissect the mechanism of translation initiation and elongation in prokaryotes, including complex processes such as translational stalling and frameshifting. Given the success of the technology, similarly complex biological processes could be studied in near-physiological conditions with the controllability of conventional in vitro experiments.

              <div id="webof-science"></div>
              <div id="pubmed"></div>
              </div>
            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Siprashvili, Z., Webster, D. E., Johnston, D., Shenoy, R. M., Ungewickell, A. J., Bhaduri, A., Flockhart, R., Zarnegar, B. J., Che, Y., Meschi, F., Puglisi, J. D., Khavari, P. A.</span>
            <a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/2016_NatureGenetics_Siprashvili_TheNoncodingRNAs.pdf" download><span id="title">The noncoding RNAs SNORD50A and SNORD50B bind K-Ras and are recurrently deleted in human cancer.</span></a>
            <span id="publisher"> Nature genetics, 2016; 48 (1): 53-58</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/2016_NatureGenetics_Siprashvili_TheNoncodingRNAs.pdf" download><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span></a>
                Small nucleolar RNAs (snoRNAs) are conserved noncoding RNAs best studied as ribonucleoprotein (RNP) guides in RNA modification. To explore their role in cancer, we compared 5,473 tumor-normal genome pairs to identify snoRNAs with frequent copy number loss. The SNORD50A-SNORD50B snoRNA locus was deleted in 10-40% of 12 common cancers, where its loss was associated with reduced survival. A human protein microarray screen identified direct SNORD50A and SNORD50B RNA binding to K-Ras. Loss of SNORD50A and SNORD50B increased the amount of GTP-bound, active K-Ras and hyperactivated Ras-ERK1/ERK2 signaling. Loss of these snoRNAs also increased binding by farnesyltransferase to K-Ras and increased K-Ras prenylation, suggesting that KRAS mutation might synergize with SNORD50A and SNORD50B loss in cancer. In agreement with this hypothesis, CRISPR-mediated deletion of SNORD50A and SNORD50B in KRAS-mutant tumor cells enhanced tumorigenesis, and SNORD50A and SNORD50B deletion and oncogenic KRAS mutation co-occurred significantly in multiple human tumor types. SNORD50A and SNORD50B snoRNAs thus directly bind and inhibit K-Ras and are recurrently deleted in human cancer.
              <div id="webof-science"></div>
              <div id="pubmed"></div>
              </div>
            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Sierra, R. G., Gati, C., Laksmono, H., Dao, E. H., Gul, S., Fuller, F., Kern, J., Chatterjee, R., Ibrahim, M., Brewster, A. S., Young, I. D., Michels-Clark, T., Aquila, A., Liang, M., Hunter, M. S., Koglin, J. E., Boutet, S., Junco, E. A., Hayes, B., Bogan, M. J., Hampton, C. Y., Puglisi, E. V., Sauter, N. K., Stan, C. A., Zouni, A., Yano, J., Yachandra, V. K., Soltis, S. M., Puglisi, J. D., DeMirci, H.</span>
            <a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/2016_NatureMethods_Sierra_Concentric-flow.pdf" download><span id="title">Concentric-flow electrokinetic injector enables serial crystallography of ribosome and photosystem II</span></a>
            <span id="publisher">NATURE METHODS,2016; 13 (1): 59-?</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/2016_NatureMethods_Sierra_Concentric-flow.pdf" download><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span></a>
                We describe a concentric-flow electrokinetic injector for efficiently delivering microcrystals for serial femtosecond X-ray crystallography analysis that enables studies of challenging biological systems in their unadulterated mother liquor. We used the injector to analyze microcrystals of Geobacillus stearothermophilus thermolysin (2.2-Å structure), Thermosynechococcus elongatus photosystem II (<3-Å diffraction) and Thermus thermophilus small ribosomal subunit bound to the antibiotic paromomycin at ambient temperature (3.4-Å structure).
              <div id="webof-science"></div>
              <div id="pubmed"></div>
              </div>
            </div>
          </div>

          <div class="year" id="2015">2015</div>
          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Chen, J., Coakley, A., O'Connor, M., Petrov, A., O'Leary, S. E., Atkins, J. F., Puglisi, J. D.</span>
            <a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/2015-Cell-Chen_Coupling.pdf" download><span id="title">Coupling of mRNA Structure Rearrangement to Ribosome Movement during Bypassing of Non-coding Regions</span></a>
            <span id="publisher">Cell, 2015; 163 (5): 1267-1280</span></div>
            <div id="more-title"><i id="triangle" class="fa fa-caret-right" aria-hidden="true"></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/2015-Cell-Chen_Coupling.pdf" download><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span></a>
                Nearly half of the ribosomes translating a particular bacteriophage T4 mRNA bypass a region of 50 nt, resuming translation 3′ of this gap. How this large-scale, specific hop occurs and what determines whether a ribosome bypasses remain unclear. We apply single-molecule fluorescence with zero-mode waveguides to track individual Escherichia coli ribosomes during translation of T4’s gene 60 mRNA. Ribosomes that bypass are characterized by a 10- to 20-fold longer pause in a non-canonical rotated state at the take-off codon. During the pause, mRNA secondary structure rearrangements are coupled to ribosome forward movement, facilitated by nascent peptide interactions that disengage the ribosome anticodon-codon interactions for slippage. Close to the landing site, the ribosome then scans mRNA in search of optimal base-pairing interactions. Our results provide a mechanistic and conformational framework for bypassing, highlighting a non-canonical ribosomal state to allow for mRNA structure refolding to drive large-scale ribosome movements.
                </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>

            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Nilsson, O. B., Hedman, R., Marino, J., Wickles, S., Bischoff, L., Johansson, M., Mueller-Lucks, A., Trovato, F., Puglisi, J. D., O'Brien, E. P., Beckmann, R., Von Heijne, G.</span>
            <a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/2015_CellReports_Nilsson_Cotranslational.pdf" download><span id="title">Cotranslational Protein Folding inside the Ribosome Exit Tunnel</span></a>
            <span id="publisher">Cell Reports, 2015; 12 (10): 1533-1540</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/2015_CellReports_Nilsson_Cotranslational.pdf" download><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span></a>
                At what point during translation do proteins fold? It is well established that proteins can fold cotranslationally outside the ribosome exit tunnel, whereas studies of folding inside the exit tunnel have so far detected only the formation of helical secondary structure and collapsed or partially structured folding intermediates. Here, using a combination of cotranslational nascent chain force measurements, inter-subunit fluorescence resonance energy transfer studies on single translating ribosomes, molecular dynamics simulations, and cryoelectron microscopy, we show that a small zinc-finger domain protein can fold deep inside the vestibule of the ribosome exit tunnel. Thus, for small protein domains, the ribosome itself can provide the kind of sheltered folding environment that chaperones provide for larger proteins.
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>

            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Puglisi, J. D.</span>
            <a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/2015-Nature-Puglisi-Ribosomal.pdf" download><span id="title">SYNTHETIC BIOLOGY Ribosomal ties that bind</span></a>
            <span id="publisher">Nature, 2015; 524 (7563): 45-46</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/2015-Nature-Puglisi-Ribosomal.pdf" download><span id="title"><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span></a>
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>

            </div>
          </div>


          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Puglisi, J. D.</span>
            <a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/2015-Science-Puglisi-TheDelicate.pdf" download><span id="title">Protein synthesis. The delicate dance of translation and folding.</span></a>
            <span id="publisher">Science, 2015; 348 (6233): 399-400</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/2015-Science-Puglisi-TheDelicate.pdf" download><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span></a>

The past decade has seen rapid growth in our knowledge of how proteins are synthesized in cells. This includes the basic step of how transfer RNAs (tRNAs) decode messenger RNAs (mRNAs) with high fidelity and speed, how the ribosome moves along mRNA from codon to codon during translation, and how synthesis of the corresponding polypeptide chain is initiated and terminated at specific points on the mRNA (1). Structures of the ribosomal particles—megadalton RNA-protein assemblies—have provided detailed molecular views of the active sites for mRNA decoding and peptide bond formation, and suggested pathways for movement of ligands, factors, and the ribosomal subunits themselves. Obscured in this rich cache of knowledge is the fate of the protein product. How does it fold during mRNA translation and how might protein folding affect translation itself? These questions are addressed by elegant biophysical and biochemical approaches reported by Goldman et al. (2) and Kim et al. (3) on pages 457 and 444, respectively, of this issue, adding to a growing appreciation of cotranslational protein folding (4–6).
                </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>

            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Puglisi, J. D.</span>
            <a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/RNA-2015-Puglisi-RNADances.pdf" download><span id="title">RNA dances to center stage</span></a>
            <span id="publisher">RNA - A Publication of the RNA Society,2015; 21 (4): 712-713 </span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/RNA-2015-Puglisi-RNADances.pdf" download><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span></a>
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>

            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Capece, M. C., Kornberg, G. L., Petrov, A., Puglisi, J. D.</span>
            <a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/2015-RNA-Capece-ASimple.pdf" download><span id="title">A simple real-time assay for in vitro translation</span></a>
            <span id="publisher">RNA, 2015; 21 (2): 296-305 </span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/2015-RNA-Capece-ASimple.pdf" download><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span></a>
                A high-throughput assay for real-time measurement of translation rates in cell-free protein synthesis (SNAP assay) is described. The SNAP assay enables quantitative, real-time measurement of overall translation rates in vitro via the synthesis of O(6)-alkylguanine DNA O(6)-alkyltransferase (SNAP). SNAP production is continuously detected by fluorescence produced by the reaction of SNAP with a range of quenched fluorogenic substrates. The capabilities of the assay are exemplified by measurements of the activities of Escherichia coli MRE600 ribosomes and fluorescently labeled E. coli mutant ribosomes in the PURExpress translation system and by determination of the 50% inhibitory concentrations (IC50) of three common macrolide antibiotics.
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>

            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Fuchs, G., Petrov, A. N., Marceau, C. D., Popov, L. M., Chen, J., O'Leary, S. E., Wang, R., Carette, J. E., Sarnow, P., Puglisi, J. D.</span>
            <a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/2014-PNAS-Fuchs-Kinetic.pdf" download><span id="title">Kinetic pathway of 40S ribosomal subunit recruitment to hepatitis C virus internal ribosome entry site</span></a>
            <span id="publisher">Proceedings of the National Academy of Sciences of the United States of America, 2015; 112 (2): 319-325</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/2014-PNAS-Fuchs-Kinetic.pdf" download><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span></a>
                Translation initiation can occur by multiple pathways. To delineate these pathways by single-molecule methods, fluorescently labeled ribosomal subunits are required. Here, we labeled human 40S ribosomal subunits with a fluorescent SNAP-tag at ribosomal protein eS25 (RPS25). The resulting ribosomal subunits could be specifically labeled in living cells and in vitro. Using single-molecule Förster resonance energy transfer (FRET) between RPS25 and domain II of the hepatitis C virus (HCV) internal ribosome entry site (IRES), we measured the rates of 40S subunit arrival to the HCV IRES. Our data support a single-step model of HCV IRES recruitment to 40S subunits, irreversible on the initiation time scale. We furthermore demonstrated that after binding, the 40S:HCV IRES complex is conformationally dynamic, undergoing slow large-scale rearrangements. Addition of translation extracts suppresses these fluctuations, funneling the complex into a single conformation on the 80S assembly pathway. These findings show that 40S:HCV IRES complex formation is accompanied by dynamic conformational rearrangements that may be modulated by initiation factors.
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>

            </div>
          </div>


          <div class="year" id="2014" >2014</div>
          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Noriega, T. R., Chen, J., Walter, P., Puglisi, J. D.
</span>
            <a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/2014_eLife_thomas_Real-time.pdf" download><span id="title">Real-time observation of signal recognition particle binding to actively translating ribosomes</span></a>
            <span id="publisher">eLife, 2014; 3 </span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/2014_eLife_thomas_Real-time.pdf" download><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span></a>
                The signal recognition particle (SRP) directs translating ribosome-nascent chain complexes (RNCs) that display a signal sequence to protein translocation channels in target membranes. All previous work on the initial step of the targeting reaction, when SRP binds to RNCs, used stalled and non-translating RNCs. This meant that an important dimension of the co-translational process remained unstudied. We apply single-molecule fluorescence measurements to observe directly and in real-time E. coli SRP binding to actively translating RNCs. We show at physiologically relevant SRP concentrations that SRP-RNC association and dissociation rates depend on nascent chain length and the exposure of a functional signal sequence outside the ribosome. Our results resolve a long-standing question: how can a limited, sub-stoichiometric pool of cellular SRP effectively distinguish RNCs displaying a signal sequence from those that are not? The answer is strikingly simple: as originally proposed, SRP only stably engages translating RNCs exposing a functional signal sequence.
                </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>

            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Chen, J., Petrov, A., Johansson, M., Tsai, A., O'Leary, S. E., Puglisi, J. D.</span>
            <a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/2014-Nature-Chen-Dynamic.pdf" download><span id="title">Dynamic pathways of -1 translational frameshifting.</span></a>
            <span id="publisher">Nature, 2014; 512 (7514): 328-332</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/2014-Nature-Chen-Dynamic.pdf" download><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span></a>
                Spontaneous changes in the reading frame of translation are rare (frequency of 10(-3) to 10(-4) per codon), but can be induced by specific features in the messenger RNA (mRNA). In the presence of mRNA secondary structures, a heptanucleotide 'slippery sequence' usually defined by the motif X XXY YYZ, and (in some prokaryotic cases) mRNA sequences that base pair with the 3' end of the 16S ribosomal rRNA (internal Shine-Dalgarno sequences), there is an increased probability that a specific programmed change of frame occurs, wherein the ribosome shifts one nucleotide backwards into an overlapping reading frame (-1 frame) and continues by translating a new sequence of amino acids. Despite extensive biochemical and genetic studies, there is no clear mechanistic description for frameshifting. Here we apply single-molecule fluorescence to track the compositional and conformational dynamics of individual ribosomes at each codon during translation of a frameshift-inducing mRNA from the dnaX gene in Escherichia coli. Ribosomes that frameshift into the -1 frame are characterized by a tenfold longer pause in elongation compared to non-frameshifted ribosomes, which translate through unperturbed. During the pause, interactions of the ribosome with the mRNA stimulatory elements uncouple EF-G catalysed translocation from normal ribosomal subunit reverse-rotation, leaving the ribosome in a non-canonical intersubunit rotated state with an exposed codon in the aminoacyl-tRNA site (A site). tRNA(Lys) sampling and accommodation to the empty A site and EF-G action either leads to the slippage of the tRNAs into the -1 frame or maintains the ribosome into the 0 frame. Our results provide a general mechanistic and conformational framework for -1 frameshifting, highlighting multiple kinetic branchpoints during elongation.
                </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>

            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Noriega, T. R., Tsai, A., Elvekrog, M. M., Petrov, A., Neher, S. B., Chen, J., Bradshaw, N., Puglisi, J. D., Walter, P.</span>
            <a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/2014-JournalBioChem-Noriega-SignalRecognition.pdf" download><span id="title">Signal Recognition Particle-ribosome Binding Is Sensitive to Nascent Chain Length</span></a>
            <span id="publisher">Journal of Biological Chemistry, 2014; 289 (28): 19294-19305</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/2014-JournalBioChem-Noriega-SignalRecognition.pdf" download><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span></a>
                The signal recognition particle (SRP) directs ribosome-nascent chain complexes (RNCs) displaying signal sequences to protein translocation channels in the plasma membrane of prokaryotes and endoplasmic reticulum of eukaryotes. It was initially proposed that SRP binds the signal sequence when it emerges from an RNC and that successful binding becomes impaired as translation extends the nascent chain, moving the signal sequence away from SRP on the ribosomal surface. Later studies drew this simple model into question, proposing that SRP binding is unaffected by nascent chain length. Here, we reinvestigate this issue using two novel and independent fluorescence resonance energy transfer assays. We show that the arrival and dissociation rates of SRP binding to RNCs vary according to nascent chain length, resulting in the highest affinity shortly after a functional signal sequence emerges from the ribosome. Moreover, we show that SRP binds RNCs in multiple and interconverting conformations, and that conversely, RNCs exist in two conformations distinguished by SRP interaction kinetics.
                </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>

            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Tsai, A., Kornberg, G., Johansson, M., Chen, J., Puglisi, J. D.</span>
            <a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/2014_CellReports_Tsai_TheDynamics.pdf" download><span id="title">The Dynamics of SecM-Induced Translational Stalling </span></a>
            <span id="publisher">Cell Reports, 2014; 7 (5): 1521-1533</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/2014_CellReports_Tsai_TheDynamics.pdf" download><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span></a>
                SecM is an E. coli secretion monitor capable of stalling translation on the prokaryotic ribosome without cofactors. Biochemical and structural studies have demonstrated that the SecM nascent chain interacts with the 50S subunit exit tunnel to inhibit peptide bond formation. However, the timescales and pathways of stalling on an mRNA remain undefined. To provide a dynamic mechanism for stalling, we directly tracked the dynamics of elongation on ribosomes translating the SecM stall sequence (FSTPVWISQAQGIRAGP) using single-molecule fluorescence techniques. Within 1 min, three peptide-ribosome interactions work cooperatively over the last five codons of the SecM sequence, leading to severely impaired elongation rates beginning from the terminal proline and lasting four codons. Our results suggest that stalling is tightly linked to the dynamics of elongation and underscore the roles that the exit tunnel and nascent chain play in controlling fundamental steps in translation.
                </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>

            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Johansson, M., Chen, J., Tsai, A., Kornberg, G., Puglisi, J. D.</span>
            <a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/2014_CellReports_Johansson_Sequence-Dependent.pdf" download><span id="title">Sequence-Dependent Elongation Dynamics on Macrolide-Bound Ribosomes</span></a>
            <span id="publisher">Cell Reports, 2014; 7 (5): 1534-1546</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/2014_CellReports_Johansson_Sequence-Dependent.pdf" download><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span></a>
                The traditional view of macrolide antibiotics as plugs inside the ribosomal nascent peptide exit tunnel (NPET) has lately been challenged in favor of a more complex, heterogeneous mechanism, where drug-peptide interactions determine the fate of a translating ribosome. To investigate these highly dynamic processes, we applied single-molecule tracking of elongating ribosomes during inhibition of elongation by erythromycin of several nascent chains, including ErmCL and H-NS, which were shown to be, respectively, sensitive and resistant to erythromycin. Peptide sequence-specific changes were observed in translation elongation dynamics in the presence of a macrolide-obstructed NPET. Elongation rates were not severely inhibited in general by the presence of the drug; instead, stalls or pauses were observed as abrupt events. The dynamic pathways of nascent-chain-dependent elongation pausing in the presence of macrolides determine the fate of the translating ribosome stalling or readthrough.
                </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>

            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Chen, J., Dalal, R. V., Petrov, A. N., Tsai, A., O'Leary, S. E., Chapin, K., Cheng, J., Ewan, M., Hsiung, P., Lundquist, P., Turner, S. W., Hsu, D. R., Puglisi, J. D.</span>
            <a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/2014-PNAS-Chen-High-throughput.pdf" download><span id="title">High-throughput platform for real-time monitoring of biological processes by multicolor single-molecule fluorescence</span></a>
            <span id="publisher">Proceedings of the National Academy of Sciences of the United States of America, 2014; 111 (2): 664-669</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/2014-PNAS-Chen-High-throughput.pdf" download><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span></a>
                Zero-mode waveguides provide a powerful technology for studying single-molecule real-time dynamics of biological systems at physiological ligand concentrations. We customized a commercial
                 zero-mode waveguide-based DNA sequencer for use as a versatile instrument for single-molecule fluorescence detection and showed that the system provides long fluorophore lifetimes with good
                 signal to noise and low spectral cross-talk. We then used a ribosomal translation assay to show real-time fluidic delivery during data acquisition, showing it is possible to follow the conformation
                 and composition of thousands of single biomolecules simultaneously through four spectral channels. This instrument allows high-throughput multiplexed dynamics of single-molecule biological processes
                  over long timescales. The instrumentation presented here has broad applications to single-molecule studies of biological systems and is easily accessible to the biophysical community.
                </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>

            </div>
          </div>

          <div class="year" id="2013">2013</div>


          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">O'Leary, S. E., Petrov, A., Chen, J., Puglisi, J. D.</span>
            <a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/2013-Structure-O'Leary-Dynamic.pdf" download><span id="title">Dynamic Recognition of the mRNA Cap by Saccharomyces cerevisiae eIF4E</span></a>
            <span id="publisher">Structure, 2013; 21 (12): 2197-2207</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/2013-Structure-O'Leary-Dynamic.pdf" download><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span></a>
                Recognition of the mRNA 5' m⁷G(5')ppp(5')N cap is key to translation initiation for most eukaryotic mRNAs. The cap is bound by the eIF4F complex, consisting
                of a cap-binding protein (eIF4E), a "scaffold" protein (eIF4G), and an RNA helicase (eIF4A). As a central early step in initiation, regulation of eIF4F is crucial for cellular viability.
                 Although the structure and function of eIF4E have been defined, a dynamic mechanistic picture of its activity at the molecular level in the eIF4F·mRNA complex is still unavailable.
                 Here, using single-molecule fluorescence, we measured the effects of Saccharomyces cerevisiae eIF4F factors, mRNA secondary structure, and the poly(A)-binding protein Pab1p on eIF4E-mRNA
                  binding dynamics. Our data provide an integrated picture of how eIF4G and mRNA structure modulate eIF4E-mRNA interaction, and uncover an eIF4G- and poly(A)-independent activity of poly(A)-binding
                  protein that prolongs the eIF4E·mRNA complex lifetime.
                </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>

            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Simonetti, A., Marzi, S., Billas, I. M., Tsai, A., Fabbretti, A., Myasnikov, A. G., Roblin, P., Vaiana, A. C., Hazemann, I., Eiler, D., Steitz, T. A., Puglisi, J. D., Gualerzi, C. O., Klaholz, B. P.</span>
            <a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/PNAS-2013-Simonetti-Involvement.pdf" download><span id="title">Involvement of protein IF2 N domain in ribosomal subunit joining revealed from architecture and function of the full-length initiation factor</span></a>
            <span id="publisher">PROCEEDINGS OF THE NATIONAL ACADEMY OF SCIENCES OF THE UNITED STATES OF AMERICA,2013; 110 (39): 15656-15661 </span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/PNAS-2013-Simonetti-Involvement.pdf" download><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span></a>
                Translation initiation factor 2 (IF2) promotes 30S initiation complex (IC) formation and 50S subunit joining, which produces the 70S IC. The architecture of full-length IF2, determined by small angle X-ray diffraction and cryo electron microscopy, reveals a more extended conformation of IF2 in solution and on the ribosome than in the crystal. The N-terminal domain is only partially visible in the 30S IC, but in the 70S IC, it stabilizes interactions between IF2 and the L7/L12 stalk of the 50S, and on its deletion, proper N-formyl-methionyl(fMet)-tRNA(fMet) positioning and efficient transpeptidation are affected. Accordingly, fast kinetics and single-molecule fluorescence data indicate that the N terminus promotes 70S IC formation by stabilizing the productive sampling of the 50S subunit during 30S IC joining. Together, our data highlight the dynamics of IF2-dependent ribosomal subunit joining and the role played by the N terminus of IF2 in this process.
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>

            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Chen, J., Petrov, A., Tsai, A., O'Leary, S. E., Puglisi, J. D.</span>
            <a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/2013_NatureStructuralMolecularBiology_Chen_Coordinated.pdf" download><span id="title">Coordinated conformational and compositional dynamics drive ribosome translocation. </span></a>
            <span id="publisher">Nature structural & molecular biology, 2013; 20 (6): 718-727</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/2013_NatureStructuralMolecularBiology_Chen_Coordinated.pdf" download><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span></a>
                During translation elongation, the ribosome compositional factors elongation factor G (EF-G; encoded by fusA) and tRNA alternately bind to the ribosome to direct protein synthesis and regulate the conformation of the ribosome. Here, we use single-molecule fluorescence with zero-mode waveguides to directly correlate ribosome conformation and composition during multiple rounds of elongation at high factor concentrations in Escherichia coli. Our results show that EF-G bound to GTP (EF-G-GTP) continuously samples both rotational states of the ribosome, binding with higher affinity to the rotated state. Upon successful accommodation into the rotated ribosome, the EF-G-ribosome complex evolves through several rate-limiting conformational changes and the hydrolysis of GTP, which results in a transition back to the nonrotated state and in turn drives translocation and facilitates release of both EF-G-GDP and E-site tRNA. These experiments highlight the power of tracking single-molecule conformation and composition simultaneously in real time.
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>

            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Tsai, A., Uemura, S., Johansson, M., Puglisi, E. V., Marshall, R. A., Aitken, C. E., Korlach, J., Ehrenberg, M., Puglisi, J. D.</span>
            <a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/2013_CellReports_Tsai_TheImpact.pdf" download><span id="title">The Impact of Aminoglycosides on the Dynamics of Translation Elongation</span></a>
            <span id="publisher">CELL REPORTS, 2013; 3 (2): 497-508</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/2013_CellReports_Tsai_TheImpact.pdf" download><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span></a>
                Inferring antibiotic mechanisms on translation through static structures has been challenging, as biological systems are highly dynamic. Dynamic single-molecule methods are also limited to few simultaneously measurable parameters. We have circumvented these limitations with a multifaceted approach to investigate three structurally distinct aminoglycosides that bind to the aminoacyl-transfer RNA site (A site) in the prokaryotic 30S ribosomal subunit: apramycin, paromomycin, and gentamicin. Using several single-molecule fluorescence measurements combined with structural and biochemical techniques, we observed distinct changes to translational dynamics for each aminoglycoside. While all three drugs effectively inhibit translation elongation, their actions are structurally and mechanistically distinct. Apramycin does not displace A1492 and A1493 at the decoding center, as demonstrated by a solution nuclear magnetic resonance structure, causing only limited miscoding; instead, it primarily blocks translocation. Paromomycin and gentamicin, which displace A1492 and A1493, cause significant miscoding, block intersubunit rotation, and inhibit translocation. Our results show the power of combined dynamics, structural, and biochemical approaches to elucidate the complex mechanisms underlying translation and its inhibition.
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>

            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Tsai, A., Chen, J., Kornberg, G., Korlach, J., Uemura, S., Puglisi, J.</span>
            <a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/2013-CallPress-Tsai-Observing.pdf" download><span id="title">Observing Prokaryotic Translation Elongation in Real-Time using Single-Molecule Fluorescence</span></a>
            <span id="publisher">CELL PRESS, 2013: 257A-257A</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/2013-CallPress-Tsai-Observing.pdf" download><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span></a>
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>

            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Petrov, A., Tsa, A., Puglisi, J. D.</span>
            <a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/2013-MethodsEnzymol-Petrov-Analysis.pdf" download><span id="title">Analysis of RNA by Analytical Polyacrylamide Gel Electrophoresis</span></a>
            <span id="publisher">LABORATORY METHODS IN ENZYMOLOGY: RNA, 2013; 530: 301-313</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/2013-MethodsEnzymol-Petrov-Analysis.pdf" download><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span></a>
                Polyacrylamide gel electrophoresis (PAGE) is a powerful tool for analyzing RNA samples. Denaturing PAGE provides information on the sample composition and structural integrity of the individual RNA species. Nondenaturing gel electrophoresis allows separation of the conformers and alternatively folded RNA species. It also can be used to resolve RNA protein complexes and to detect RNA complex formation by analyzing changes in the electrophoretic mobility of the RNA. RNA can be visualized within gels by different methods depending on the nature of the detection reagent. RNA molecules can be stained with various dyes, including toluidine blue, SYBR green, and ethidium bromide. Radioactively labeled RNA molecules are visualized by autoradiography, and fluorescently labeled RNA molecules can be observed with a fluorescence scanner. Generally, gels between 0.4 and 1.5mm thick are used for analytical PAGE. Gels thinner than 1mm are fragile and thus usually are not stained but rather are used for radiolabeled RNA. The gels are dried and the radiolabeled RNA is visualized by autoradiography.
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>

            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Petrov, A., Wu, T., Puglisi, E. V., Puglisi, J. D.</span>
            <a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/2013-MethodsEnzymol-Petrov_RNAPurification.pdf" download><span id="title">RNA Purification by Preparative Polyacrylamide Gel Electrophoresis</span></a>
            <span id="publisher">LABORATORY METHODS IN ENZYMOLOGY: RNA, 2013; 530: 315-330</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/2013-MethodsEnzymol-Petrov_RNAPurification.pdf" download><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span></a>
                Preparative polyacrylamide gel electrophoresis (PAGE) is a powerful tool for purifying RNA samples. Denaturing PAGE allows separation of nucleic acids that differ by a single nucleotide in length. It is commonly used to separate and purify RNA species after in vitro transcription, to purify naturally occurring RNA variants such as tRNAs, to remove degradation products, and to purify labeled RNA species. To preserve RNA integrity following purification, RNA is usually visualized by UV shadowing or stained with ethidium bromide or SYBR green dyes.
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>

            </div>
          </div>


        <div class="year" id="2012">2012</div>
          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Chen, J., Tsai, A., O'Leary, S. E., Petrov, A., Puglisi, J. D.</span>
            <a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/2012-CurrOpinStructBiol-Chen-Unraveling.pdf" download><span id="title">Unraveling the dynamics of ribosome translocation</span></a>
            <span id="publisher">CURRENT OPINION IN STRUCTURAL BIOLOGY,2012; 22 (6): 804-814 </span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/2012-CurrOpinStructBiol-Chen-Unraveling.pdf" download><span id="title"><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span></a>
                Translocation is one of the key events in translation, requiring large-scale conformational changes in the ribosome, movements of two transfer RNAs (tRNAs) across a distance of more than 20?, and the coupled movement of the messenger RNA (mRNA) by one codon, completing one cycle of peptide-chain elongation. Translocation is catalyzed by elongation factor G (EF-G in bacteria), which hydrolyzes GTP in the process. However, how the conformational rearrangements of the ribosome actually drive the movements of the tRNAs and how EF-G GTP hydrolysis plays a role in this process are still unclear. Fluorescence methods, both single-molecule and bulk, have provided a dynamic view of translocation, allowing us to follow the different conformational changes of the ribosome in real-time. The application of electron microscopy has revealed new conformational intermediates during translocation and important structural rearrangements in the ribosome that drive tRNA movement, while computational approaches have added quantitative views of the translational pathway. These recent advances shed light on the process of translocation, providing insight on how to resolve the different descriptions of translocation in the current literature.
                </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>

            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Petrov, A., Chen, J., O'Leary, S., Tsai, A., Puglisi, J. D.</span>
            <a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/2012-Cold+Spring+Harb+Perspect+Biol-Petrov-Single.pdf" download><span id="title">Single-Molecule Analysis of Translational Dynamics</span></a>
            <span id="publisher">COLD SPRING HARBOR PERSPECTIVES IN BIOLOGY,2012; 4 (9) </span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/2012-Cold+Spring+Harb+Perspect+Biol-Petrov-Single.pdf" download><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span></a>
                Decades of extensive biochemical and biophysical research have outlined the mechanism of translation. Rich structural studies have provided detailed snapshots of the translational machinery at all phases of the translation cycle. However, the relationship between structural dynamics, composition, and function remains unknown. The multistep nature of each stage of the translation cycle results in rapid desynchronization of individual ribosomes, thus hindering elucidation of the underlying mechanisms by conventional bulk biophysical and biochemical methods. Single-molecule approaches unsusceptible to these complications have led to the first glances at both compositional and conformational dynamics on the ribosome and their impact on translational control. These experiments provide the necessary link between static structure and mechanism, often providing new perspectives. Here we review recent advances in the field and their relationship to structural and biochemical data.
                </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>

            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Harvey, C. J., Puglisi, J. D., Pande, V. S., Cane, D. E., Khosla, C.</span>
            <a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/2012-JACS-Harvey-Precursor.pdf" download><span id="title">Precursor Directed Biosynthesis of an Orthogonally Functional Erythromycin Analogue: Selectivity in the Ribosome Macrolide Binding Pocket</span></a>
            <span id="publisher">JOURNAL OF THE AMERICAN CHEMICAL SOCIETY, 2012; 134 (29): 12259-12265</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/2012-JACS-Harvey-Precursor.pdf" download><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span></a>
                The macrolide antibiotic erythromycin A and its semisynthetic analogues have been among the most useful antibacterial agents for the treatment of infectious diseases. Using a recently developed chemical genetic strategy for precursor-directed biosynthesis and colony bioassay of 6-deoxyerythromycin D analogues, we identified a new class of alkynyl- and alkenyl-substituted macrolides with activities comparable to that of the natural product. Further analysis revealed a marked and unexpected dependence of antibiotic activity on the size and degree of unsaturation of the precursor. Based on these leads, we also report the precursor-directed biosynthesis of 15-propargyl erythromycin A, a novel antibiotic that not only is as potent as erythromycin A with respect to its ability to inhibit bacterial growth and cell-free ribosomal protein biosynthesis but also harbors an orthogonal functional group that is capable of facile chemical modification.
                </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>

            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Tsai, A., Petrov, A., Marshall, R. A., Korlach, J., Uemura, S., Puglisi, J. D.</span>
            <a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/2012_Nature_Tsai__Heterogeneous.pdf" download><span id="title">Heterogeneous pathways and timing of factor departure during translation initiation</span></a>
            <span id="publisher">NATURE,2012; 487 (7407): 390-394 </span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/2012_Nature_Tsai__Heterogeneous.pdf" download><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span></a>
                The initiation of translation establishes the reading frame for protein synthesis and is a key point of regulation. Initiation involves factor-driven assembly at a start codon of a messenger RNA of an elongation-competent 70S ribosomal particle (in bacteria) from separated 30S and 50S subunits and initiator transfer RNA. Here we establish in Escherichia coli, using direct single-molecule tracking, the timing of initiator tRNA, initiation factor 2 (IF2; encoded by infB) and 50S subunit joining during initiation. Our results show multiple pathways to initiation, with orders of arrival of tRNA and IF2 dependent on factor concentration and composition. IF2 accelerates 50S subunit joining and stabilizes the assembled 70S complex. Transition to elongation is gated by the departure of IF2 after GTP hydrolysis, allowing efficient arrival of elongator tRNAs to the second codon presented in the aminoacyl-tRNA binding site (A site). These experiments highlight the power of single-molecule approaches to delineate mechanisms in complex multicomponent systems.
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>

            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Puglisi, J. D., Williamson, J. R.</span>
            <a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/2012-CurrOpinStructBiol-Puglisi-Digging.pdf" download><span id="title">Digging deep into nucleic acid structure and nucleic acid protein recognition </span></a>
            <span id="publisher">CURRENT OPINION IN STRUCTURAL BIOLOGY, 2012; 22 (3): 249-250</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/2012-CurrOpinStructBiol-Puglisi-Digging.pdf" download><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span></a>
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>

            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Chen, J., Tsai, A., Petrov, A., Puglisi, J. D.</span>
            <a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/2012-JACS-Chen-Nonfluorescent.pdf" download><span id="title">Nonfluorescent Quenchers To Correlate Single-Molecule Conformational and Compositional Dynamics</span></a>
            <span id="publisher">JOURNAL OF THE AMERICAN CHEMICAL SOCIETY, 2012; 134 (13): 5734-5737</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/2012-JACS-Chen-Nonfluorescent.pdf" download><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span></a>
                Single-molecule Förster resonance energy transfer (smFRET) is a powerful method for studying the conformational dynamics of a biomolecule in real-time. However, studying how interacting ligands correlate with and regulate the conformational dynamics of the biomolecule is extremely challenging because of the availability of a limited number of fluorescent dyes with both high quantum yield and minimal spectral overlap. Here we report the use of a nonfluorescent quencher (Black Hole Quencher, BHQ) as an acceptor for smFRET. Using a Cy3/BHQ pair, we can accurately follow conformational changes of the ribosome during elongation in real time. We demonstrate the application of single-color FRET to correlate the conformational dynamics of the ribosome with the compositional dynamics of tRNA. We use the normal Cy5 FRET acceptor to observe arrival of a fluorescently labeled tRNA with a concomitant transition of the ribosome from the locked to the unlocked conformation. Our results illustrate the potential of nonfluorescent quenchers in single-molecule correlation studies.
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>

            </div>
          </div>


          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Puglisi, J. D., Chen, J., Kornberg, G., O'Leary, S., Petrov, A., Tsai, A.</span>
            <a href="" download><span id="title">Real-Time Dynamics of Translation</span></a>
            <span id="publisher">FEDERATION AMER SOC EXP BIOL. 2012 </span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span>
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>

            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Masuda, T., Petrov, A. N., Iizuka, R., Funatsu, T., Puglisi, J. D., Uemura, S.</span>
            <a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/2012-PNAS-Masuda-Initiation.pdf" download><span id="title">Initiation factor 2, tRNA, and 50S subunits cooperatively stabilize mRNAs on the ribosome during initiation </span></a>
            <span id="publisher">PROCEEDINGS OF THE NATIONAL ACADEMY OF SCIENCES OF THE UNITED STATES OF AMERICA,2012; 109 (13): 4881-4885 </span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/2012-PNAS-Masuda-Initiation.pdf" download><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span></a>
                Initiation factor 2 (IF2) is a key factor in initiation of bacterial protein synthesis. It recruits initiator tRNA to the small ribosomal subunit and facilitates joining of the large ribosomal subunit. Using reconstituted translation system of Escherichia coli and optical tweezers, we directly measure the rupture force between single ribosomal complexes and mRNAs for initiation complexes in the presence and the absence of IF2. We demonstrate that IF2 together with codon recognition by initiator tRNA increases the force required to dislocate mRNA from the ribosome complexes; mRNA stabilization by IF2 required the presence of a joined 50S subunit, and was independent of bound guanine nucleotide. IF2 thus helps lock the 70S ribosome over the start codon during initiation, thus maintaining reading frame. Our results show how mRNA is progressively stabilized on the ribosome through distinct steps of initiation.
                </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>

            </div>
          </div>


          <div class="year" id="2011">2011</div>
          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">
                Puglisi, E. V., Puglisi, J. D.</span>
            <a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/2011-JMB-PuglisiE-Secondary.pdf" download><span id="title">Secondary Structure of the HIV Reverse Transcription Initiation Complex by NMR</span></a>
            <span id="publisher">JOURNAL OF MOLECULAR BIOLOGY,2011; 410 (5): 863-874 </span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/2011-JMB-PuglisiE-Secondary.pdf" download><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span></a>
                Initiation of reverse transcription of genomic RNA is a key early step in replication of the human immunodeficiency virus (HIV) upon infection of a host cell. Viral reverse transcriptase initiates from a specific RNA-RNA complex formed between a host transfer RNA (tRNA(Lys)(3)) and a region at the 5' end of genomic RNA; the 3' end of the tRNA acts as a primer for reverse transcription of genomic RNA. We report here the secondary structure of the HIV genomic RNA-human tRNA(Lys)(3) initiation complex using heteronuclear nuclear magnetic resonance methods. We show that both RNAs undergo large-scale conformational changes upon complex formation. Formation of the 18-bp primer helix with the 3' end of tRNA(Lys)(3) drives large conformational rearrangements of the tRNA at the 5' end while maintaining the anticodon loop for potential loop-loop interactions. HIV RNA forms an intramolecular helix adjacent to the intermolecular primer helix. This helix, which must be broken by reverse transcription, likely acts as a kinetic block to reverse transcription.
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>

            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Petrov, A., Kornberg, G., O'Leary, S., Tsai, A., Uemura, S., Puglisi, J. D.</span>
            <a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/2011-CurrOpinStructBiol-Petrov-Dynamics.pdf" download><span id="title">Dynamics of the translational machinery</span></a>
            <span id="publisher"> CURRENT OPINION IN STRUCTURAL BIOLOGY, 2011; 21 (1): 137-145</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/2011-CurrOpinStructBiol-Petrov-Dynamics.pdf" download><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span></a>
                The recent growth in single molecule studies of translation has provided an insight into the molecular mechanism of ribosomal function. Single molecule fluorescence approaches allowed direct observation of the structural rearrangements occurring during translation and revealed dynamic motions of the ribosome and its ligands. These studies demonstrated how ligand binding affects dynamics of the ribosome, and the role of the conformational sampling in large-scale rearrangements intrinsic to translation elongation. The application of time-resolved cryo-electron microscopy revealed new conformational intermediates during back-translocation providing an insight into ribosomal dynamics from an alternative perspective. Recent developments permitted examination of conformational and compositional dynamics of the ribosome in real-time through multiple cycles of elongation at the single molecule level. The zero-mode waveguide approach allowed direct observation of the compositional dynamics of tRNA occupancy on the elongating ribosome. The emergence of single molecule in vivo techniques provided insights into the mechanism and regulation of translation at the organismal level.
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>

            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Uemura, S., Puglisi, J. D.</span>
            <a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/2011-Ribosomes-Uemura.pdf" download><span id="title">Real-time monitoring of single-molecule translation</span></a>
            <span id="publisher">RIBOSOMES: STRUCTURE, FUNCTION, AND DYNAMICS, 2011: 295-302</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/2011-Ribosomes-Uemura.pdf" download><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span></a>
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>

            </div>
          </div>


          <div class="year" id="2010">2010</div>
          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Petrov, A., Puglisi, J. D.</span>
            <a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/2010-NucleicAcidResearch-Petrov-Site-specific.pdf" download><span id="title">Site-specific labeling of Saccharomyces cerevisiae ribosomes for single-molecule manipulations</span></a>
            <span id="publisher">NUCLEIC ACIDS RESEARCH, 2010; 38 (13)</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/2010-NucleicAcidResearch-Petrov-Site-specific.pdf" download><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span></a>
                Site-specific labeling of Escherichia coli ribosomes has allowed application of single-molecule fluorescence spectroscopy and force methods to probe the mechanism of translation. To apply these approaches to eukaryotic translation, eukaryotic ribosomes must be specifically labeled with fluorescent labels and molecular handles. Here, we describe preparation and labeling of the small and large yeast ribosomal subunits. Phylogenetically variable hairpin loops in ribosomal RNA are mutated to allow hybridization of oligonucleotides to mutant ribosomes. We demonstrate specific labeling of the ribosomal subunits, and their use in single-molecule fluorescence and force experiments.
                </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>

            </div>
          </div>


          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Aitken, C. E., Puglisi, J. D.</span>
            <a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/2010-nsmb-Aitken-Following.pdf" download><span id="title">Following the intersubunit conformation of the ribosome during translation in real time</span></a>
            <span id="publisher">NATURE STRUCTURAL & MOLECULAR BIOLOGY, 2010; 17 (7): 793-U35</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/2010-nsmb-Aitken-Following.pdf" download><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span></a>
                We report the direct observation of conformational rearrangements of the ribosome during multiple rounds of elongation. Using
single-molecule fluorescence resonance energy transfer, we monitored the intersubunit conformation of the ribosome in real
time as it proceeds from codon to codon. During each elongation cycle, the ribosome unlocks upon peptide bond formation,
then reverts to the locked state upon translocation onto the next codon. Our data reveal both the specific and cumulative effects
of antibiotics on individual steps of translation and uncover the processivity of the ribosome as it elongates. Our approach
interrogates the precise molecular events occurring at each codon of the mRNA within the full context of ongoing translation.
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>

            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Puglisi, J. D., Williamson, J. R.</span>
            <a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/2010-CurrOpinStructBio-Puglisi-Nucleic.pdf" download><span id="title">Nucleic acids continue to surprise Editorial overview</span></a>
            <span id="publisher">CURRENT OPINION IN STRUCTURAL BIOLOGY, 2010; 20 (3): 259-261</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/2010-CurrOpinStructBio-Puglisi-Nucleic.pdf" download><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span></a>
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>

            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Uemura, S., Aitken, C. E., Korlach, J., Flusberg, B. A., Turner, S. W., Puglisi, J. D.</span>
            <a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/2012-Nature-Uemura-Real-time.pdf" download><span id="title">Real-time tRNA transit on single translating ribosomes at codon resolution</span></a>
            <span id="publisher">NATURE, 2010; 464 (7291): 1012-U73</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/2012-Nature-Uemura-Real-time.pdf" download><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span></a>
                Translation by the ribosome occurs by a complex mechanism involving the coordinated interaction of multiple nucleic acid and protein ligands. Here we use zero-mode waveguides (ZMWs) and sophisticated detection instrumentation to allow real-time observation of translation at physiologically relevant micromolar ligand concentrations. Translation at each codon is monitored by stable binding of transfer RNAs (tRNAs)-labelled with distinct fluorophores-to translating ribosomes, which allows direct detection of the identity of tRNA molecules bound to the ribosome and therefore the underlying messenger RNA (mRNA) sequence. We observe the transit of tRNAs on single translating ribosomes and determine the number of tRNA molecules simultaneously bound to the ribosome, at each codon of an mRNA molecule. Our results show that ribosomes are only briefly occupied by two tRNA molecules and that release of deacylated tRNA from the exit (E) site is uncoupled from binding of aminoacyl-tRNA site (A-site) tRNA and occurs rapidly after translocation. The methods outlined here have broad application to the study of mRNA sequences, and the mechanism and regulation of translation.
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>

            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Bokoch, M. P., Zou, Y., Rasmussen, S. G., Liu, C. W., Nygaard, R., Rosenbaum, D. M., Fung, J. J., Choi, H., Thian, F. S., Kobilka, T. S., Puglisi, J. D., Weis, W. I., Pardo, L., Prosser, R. S., Mueller, L., Kobilka, B. K.</span>
            <a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/2010-Nature-Bokoch-Ligand.pdf" download><span id="title">Ligand-specific regulation of the extracellular surface of a G-protein-coupled receptor</span></a>
            <span id="publisher">NATURE, 2010; 463 (7277): 108-U121</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/2010-Nature-Bokoch-Ligand.pdf" download><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span></a>
                G-protein-coupled receptors (GPCRs) are seven-transmembrane proteins that mediate most cellular responses to hormones and neurotransmitters. They are the largest group of therapeutic targets for a broad spectrum of diseases. Recent crystal structures of GPCRs have revealed structural conservation extending from the orthosteric ligand-binding site in the transmembrane core to the cytoplasmic G-protein-coupling domains. In contrast, the extracellular surface (ECS) of GPCRs is remarkably diverse and is therefore an ideal target for the discovery of subtype-selective drugs. However, little is known about the functional role of the ECS in receptor activation, or about conformational coupling of this surface to the native ligand-binding pocket. Here we use NMR spectroscopy to investigate ligand-specific conformational changes around a central structural feature in the ECS of the beta(2) adrenergic receptor: a salt bridge linking extracellular loops 2 and 3. Small-molecule drugs that bind within the transmembrane core and exhibit different efficacies towards G-protein activation (agonist, neutral antagonist and inverse agonist) also stabilize distinct conformations of the ECS. We thereby demonstrate conformational coupling between the ECS and the orthosteric binding site, showing that drugs targeting this diverse surface could function as allosteric modulators with high subtype selectivity. Moreover, these studies provide a new insight into the dynamic behaviour of GPCRs not addressable by static, inactive-state crystal structures.
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>

            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Aitken, C. E., Petrov, A., Puglisi, J. D.</span>
            <a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/2010-ANRV411-Aitken-Single.pdf" download><span id="title">Single Ribosome Dynamics and the Mechanism of Translation</span></a>
            <span id="publisher">ANNUAL REVIEW OF BIOPHYSICS VOL 39, 2010; 39: 491-513</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/2010-ANRV411-Aitken-Single.pdf" download><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span></a>
                Our current understanding of the mechanism of translation is based on nearly fifty years of biochemical and biophysical studies. This mechanism, which requires the ribosome to manipulate tRNA and step repetitively along the mRNA, implies movement. High-resolution structures of the ribosome and its ligands have recently described translation in atomic detail, capturing the endpoints of large-scale rearrangements of the ribosome. Direct observation of the dynamic events that underlie the mechanism of translation is challenged by ensemble averaging in bulk solutions. Single-molecule methods, which eliminate these averaging effects, have emerged as powerful tools to probe the mechanism of translation. Single-molecule fluorescence experiments have described the dynamic motion of the ribosome and tRNA. Single-molecule force measurements have directly probed the forces stabilizing ribosomal complexes. Recent developments have allowed real-time observation of ribosome movement and dynamics during translation. This review covers the contributions of single-molecule studies to our understanding of the dynamic nature of translation.
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>
            </div>
          </div>

        <div class="year" id="2009">2009</div>
          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Puglisi, J. D.</span>
            <a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/2009-Cell-Puglisi-Resolving.pdf" download><span id="title">Resolving the Elegant Architecture of the Ribosome</span></a>
            <span id="publisher">MOLECULAR CELL, 2009; 36 (5): 720-723</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/2009-Cell-Puglisi-Resolving.pdf" download><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span></a>
                This year's Nobel Prize in Chemistry rewards Ada Yonath, Tom Steitz, and Venki Ramakrishnan for their groundbreaking structural studies on the ribosome.
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>

            </div>
          </div>


          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Elazar, M., Liu, M., Mckenna, S. A., Liu, P., Gehrig, E. A., Puglisi, J. D., Rossignol, J., Glenn, J. S.</span>
            <a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/2009-GASTROENTEROLOGY-Elazar.pdf" download><span id="title">
The Anti-Hepatitis C Agent Nitazoxanide Induces Phosphorylation of Eukaryotic Initiation Factor 2 alpha Via Protein Kinase Activated by Double-Stranded RNA Activation</span></a>
            <span id="publisher">GASTROENTEROLOGY, 2009; 137 (5): 1827-1835 </span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/2009-GASTROENTEROLOGY-Elazar.pdf" download><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span></a>
                New therapies are needed to treat patients infected with hepatitis C virus (HCV), a major worldwide cause of chronic liver disease. Nitazoxanide (NTZ), originally used to treat cryptosporidiosis infection, recently was shown to have unexpected antiviral activity in the HCV replicon system and in chronically infected patients. A pilot clinical study suggested that NTZ can augment the antiviral effect of interferon (IFN), although the molecular basis for its effect was unknown.We analyzed the effects of NTZ on the regulation of eukaryotic initiation factor-2alpha (eIF2alpha) and its IFN-induced kinase, protein kinase activated by double-stranded RNA (PKR), in cells that support HCV RNA replication and in vitro biochemical assays.NTZ increased eIF2alpha phosphorylation, a modification known to mediate host cell antiviral defenses. The addition of IFN to cell cultures increased NTZ-induced eIF2alpha phosphorylation. NTZ also increased PKR phosphorylation. In vitro, NTZ promoted PKR autophosphorylation, a key step in activating PKR's kinase activity for eIF2alpha. Finally, NTZ-induced eIF2alpha phosphorylation was reduced in the presence of specific inhibitors of PKR autophosphorylation.An important mechanism of NTZ's action involves activation of PKR, a key kinase that regulates the cell's innate antiviral response. These observations could explain the clinical antiviral effect of NTZ. NTZ might represent a new class of small molecules capable of potentiating and recapitulating important antiviral effects of IFN.
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>

            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Shimoike, T., Mckenna, S. A., Lindhout, D. A., Puglisi, J. D.</span>
            <a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/2009-AntiviralResearch-Shimoike-Translational.pdf" download><span id="title">Translational insensitivity to potent activation of PKR by HCV IRES RNA</span></a>
            <span id="publisher">ANTIVIRAL RESEARCH, 2009; 83 (3): 228-237</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/2009-AntiviralResearch-Shimoike-Translational.pdf" download><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span></a>
                Translation of hepatitis C virus (HCV) is initiated at an internal ribosome entry site (IRES) located at the 5'end of its RNA genome. The HCV IRES is highly structured and greater than 50% of its nucleotides form based-paired helices. We report here that the HCV IRES is an activator of PKR, an interferon-induced enzyme that participates in host cell defense against viral infection. Binding of HCV IRES RNA to PKR leads to a greatly increased (20-fold) rate and level (4.5-fold) of PKR autophosphorylation compared to previously studied dsRNA activators. We have mapped the domains in the IRES required for PKR activation to domains III-IV and demonstrate that the N-terminal double-stranded RNA binding domains of PKR bind to the IRES in a similar manner to other RNA activators. Addition of HCV IRES RNA inhibits cap-dependent translation in lysates via phosphorylation of PKR and eIF2alpha. However, HCV IRES-mediated translation is not inhibited by the phosphorylation of PKR and eIF2alpha. The results presented here suggest that hydrolysis of GTP by eIF2 is not an essential step in IRES-mediated translation. Thus, HCV can use structured RNAs to its advantage in translation, while avoiding the deleterious effects of PKR activation.
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>

            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Marshall, R. A., Aitken, C. E., Puglisi, J. D.</span>
            <a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/2009-MolecularCell-Marshall-GTP.pdf" download><span id="title">GTP Hydrolysis by IF2 Guides Progression of the Ribosome into Elongation</span></a>
            <span id="publisher">MOLECULAR CELL, 2009; 35 (1): 37-47</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/2009-MolecularCell-Marshall-GTP.pdf" download><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span></a>
                Recent structural data have revealed two distinct conformations of the ribosome during initiation. We employed single-molecule fluorescence methods to probe the dynamic relation of these ribosomal conformations in real time. In the absence of initiation factors, the ribosome assembles in two distinct conformations. The initiation factors guide progression of the ribosome to the conformation that can enter the elongation cycle. In particular, IF2 both accelerates the rate of subunit joining and actively promotes the transition to the elongation-competent conformation. Blocking GTP hydrolysis by IF2 results in 70S complexes formed in the conformation unable to enter elongation. We observe that rapid GTP hydrolysis by IF2 drives the transition to the elongation-competent conformation, thus committing the ribosome to enter the elongation cycle.
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>

            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Liu, C. W., Alekseyev, V. Y., Allwardt, J. R., Bankovich, A. J., Cade-Menun, B. J., Davis, R. W., Du, L., Garcia, K. C., Herschlag, D., Khosla, C., Kraut, D. A., Li, Q., Null, B., Puglisi, J. D., Sigala, P. A., Stebbins, J. F., Varani, L.</span>
            <a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/2009-Nato-liu.pdf" download><span id="title">THE DIVERSITY OF NUCLEAR MAGNETIC RESONANCE SPECTROSCOPY</span></a>
            <span id="publisher">NATO Advanced Study Institute on Biophysics and the Challenges of Emerging Threats, SPRINGER. 2009: 65–81</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/2009-Nato-liu.pdf" download><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span></a>
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>

            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Aitken, C. E., Marshall, R. A., Puglisi, J. D.</span>
            <a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/2009-Nato-Aitken-Improved.pdf" download><span id="title">Improved Dye Stability In Single-Molecule Fluorescence Experiments </span></a>
            <span id="publisher">NATO Advanced Study Institute on Biophysics and the Challenges of Emerging Threats, SPRINGER. 2009: 83–99</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/2009-Nato-Aitken-Improved.pdf" download><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span></a>
                Complex biological systems challenge existing single-molecule
methods. In particular, dye stability limits observation time in singlemolecule
fluorescence applications. Current approaches to improving dye
performance involve the addition of enzymatic oxygen scavenging systems
and small molecule additives. We present an enzymatic oxygen scavenging
system that improves dye stability in single-molecule experiments.
Compared to the currently-employed glucose-oxidase/catalase system, the
protocatechuate-3,4-dioxygenase system achieves lower dissolved oxygen
concentration and stabilizes single Cy3, Cy5, and Alexa488 fluorophores.
Moreover, this system possesses none of the limitations associated with the
glucose oxidase/catalase system. We also tested the effects of small molecule
additives in this system. Biological reducing agents significantly destabilize
the Cy5 fluorophore as a function of reducing potential. In contrast, antioxidants
stabilize the Cy3 and Alexa488 fluorophores. We recommend use of
the protocatechuate-3,4,-dioxygenase system with antioxidant additives, and
in the absence of biological reducing agents. This system should have wide
application to single-molecule fluorescence experiments.
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>

            </div>
          </div>

        <div class="year" id="2008">2008</div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Marshall, R. A., Dorywalska, M., Puglisi, J. D.</span>
            <a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/PNAS-2008-Marshall-Irreversible.pdf" download><span id="title">Irreversible chemical steps control intersubunit dynamics during translation</span></a>
            <span id="publisher">PROCEEDINGS OF THE NATIONAL ACADEMY OF SCIENCES OF THE UNITED STATES OF AMERICA, 2008; 105 (40): 15364-15369</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/PNAS-2008-Marshall-Irreversible.pdf" download><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span></a>
                The ribosome, a two-subunit macromolecular machine, deciphers the genetic code and catalyzes peptide bond formation. Dynamic rotational movement between ribosomal subunits is likely required for efficient and accurate protein synthesis, but direct observation of intersubunit dynamics has been obscured by the repetitive, multistep nature of translation. Here, we report a collection of single-molecule fluorescence resonance energy transfer assays that reveal a ribosomal intersubunit conformational cycle in real time during initiation and the first round of elongation. After subunit joining and delivery of correct aminoacyl-tRNA to the ribosome, peptide bond formation results in a rapid conformational change, consistent with the counterclockwise rotation of the 30S subunit with respect to the 50S subunit implied by prior structural and biochemical studies. Subsequent binding of elongation factor G and GTP hydrolysis results in a clockwise rotation of the 30S subunit relative to the 50S subunit, preparing the ribosome for the next round of tRNA selection and peptide bond formation. The ribosome thus harnesses the free energy of irreversible peptidyl transfer and GTP hydrolysis to surmount activation barriers to large-scale conformational changes during translation. Intersubunit rotation is likely a requirement for the concerted movement of tRNA and mRNA substrates during translocation.
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>

            </div>
          </div>


          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Elazor, M., Liu, M., McKenna, S., Liu, P., Gehrig, E. A., Elfert, A., Puglisi, J., Rossignol, J., Glenn, J. S.</span>
            <a href="" download><span id="title">Nitazoxanide (NTZ)is an Inducer EIF2A and PKR Phosphorylation</span></a>
            <span id="publisher">59th Annual Meeting of the American-Association-for-the-Study-of-Liver-Diseases, WILEY-BLACKWELL. 2008: 1151A–1151A</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span>
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>

            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Uemura, S., Iizuka, R., Ueno, T., Shimizu, Y., Taguchi, H., Ueda, T., Puglisi, J. D., Funatsu, T.</span>
            <a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/2008-NucleicAcidResearch-Uemura-Single.pdf" download><span id="title">Single-molecule imaging of full protein synthesis by immobilized ribosomes</span></a>
            <span id="publisher">NUCLEIC ACIDS RESEARCH, 2008; 36 (12)</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/2008-NucleicAcidResearch-Uemura-Single.pdf" download><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span></a>
                How folding of proteins is coupled to their synthesis remains poorly understood. Here, we apply single-molecule fluorescence imaging to full protein synthesis in vitro. Ribosomes were specifically immobilized onto glass surfaces and synthesis of green fluorescent protein (GFP) was achieved using modified commercial Protein Synthesis using Recombinant Elements that lacked ribosomes but contained purified factors and enzyme that are required for translation in Escherichia coli. Translation was monitored using a GFP mutant (F64L/S65T/F99S/M153T/V163A) that has a high fluorophore maturation rate and that contained the Secretion Monitor arrest sequence to prevent dissociation from the ribosome. Immobilized ribosomal subunits were labeled with Cy3 and GFP synthesis was measured by colocalization of GFP fluorescence with the ribosome position. The rate of appearance of colocalized ribosome GFP was equivalent to the rates of fluorescence appearance coupled with translation measured in bulk, and the ribosome-polypeptide complexes were stable for hours. The methods presented here are applicable to single-molecule investigation of translational initiation, elongation and cotranslational folding.
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>

            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Puglisi, J. D., Doudna, J. A.</span>
            <a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/2008-CurrOpinStructBio-Puglisi-Nucleic.pdf" download><span id="title">Nucleic acids and their protein partners</span></a>
            <span id="publisher">CURRENT OPINION IN STRUCTURAL BIOLOGY, 2008; 18 (3): 279-281</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/2008-CurrOpinStructBio-Puglisi-Nucleic.pdf" download><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span></a>
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>

            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Aitken, C. E., Marshall, R. A., Puglisi, J. D.</span>
            <a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/2008-BiophysicsJournal-Aiken-Oxygen.pdf" download><span id="title">An oxygen scavenging system for improvement of dye stability in single-molecule fluorescence experiments</span></a>
            <span id="publisher"> BIOPHYSICAL JOURNAL, 2008; 94 (5): 1826-1835</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/2008-BiophysicsJournal-Aiken-Oxygen.pdf" download><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span></a>
                The application of single-molecule fluorescence techniques to complex biological systems places demands on the performance of single fluorophores. We present an enzymatic oxygen scavenging system for improved dye stability in single-molecule experiments. We compared the previously described protocatechuic acid/protocatechuate-3,4-dioxygenase system to the currently employed glucose oxidase/catalase system. Under standardized conditions, we observed lower dissolved oxygen concentrations with the protocatechuic acid/protocatechuate-3,4-dioxygenase system. Furthermore, we observed increased initial lifetimes of single Cy3, Cy5, and Alexa488 fluorophores. We further tested the effects of chemical additives in this system. We found that biological reducing agents increase both the frequency and duration of blinking events of Cy5, an effect that scales with reducing potential. We observed increased stability of Cy3 and Alexa488 in the presence of the antioxidants ascorbic acid and n-propyl gallate. This new O(2)-scavenging system should have wide application for single-molecule fluorescence experiments.
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>

            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Marshall, R. A., Aitken, C. E., Dorywalska, M., Puglisi, J. D.</span>
            <a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/2008-AnnualReviewBioChem-Marshall-Translation.pdf" download><span id="title">Translation at the single-molecule level</span></a>
            <span id="publisher">ANNUAL REVIEW OF BIOCHEMISTRY, 2008; 77: 177-203</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/2008-AnnualReviewBioChem-Marshall-Translation.pdf" download><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span></a>
                Decades of studies have established translation as a multistep, multicomponent process that requires intricate communication to achieve high levels of speed, accuracy, and regulation. A crucial next step in understanding translation is to reveal the functional significance of the large-scale motions implied by static ribosome structures. This requires determining the trajectories, timescales, forces, and biochemical signals that underlie these dynamic conformational changes. Single-molecule methods have emerged as important tools for the characterization of motion in complex systems, including translation. In this review, we chronicle the key discoveries in this nascent field, which have demonstrated the power and promise of single-molecule techniques in the study of translation.
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>

            </div>
          </div>


        <div class="year" id="2007">2007</div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Puglisi, J. D.</span>
            <a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/2007-StructuralBiology-Puglisi-TheDance.pdf" download><span id="title">Structural biology - The dance of domains</span></a>
            <span id="publisher">NATURE, 2007; 450 (7173): 1171-1172 </span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/2007-StructuralBiology-Puglisi-TheDance.pdf" download><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span></a>
                Static crystal structures provide only clues about the way large biological
                molecules work. A recently developed spectroscopic technique also reveals
                details of their molecular motion, as shown for an RNA molecule from HIV.
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>

            </div>
          </div>


          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Gonzalez, R. L., Chu, S., Puglisi, J. D.</span>
            <a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/RNA-2007-Gonzalez-Thiostrepton.pdf" download><span id="title">Thiostrepton inhibition of tRNA delivery to the ribosome</span></a>
            <span id="publisher">RNA-A PUBLICATION OF THE RNA SOCIETY, 2007; 13 (12): 2091-2097</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/RNA-2007-Gonzalez-Thiostrepton.pdf" download><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span></a>
                Ribosome-stimulated hydrolysis of guanosine-5'-triphosphate (GTP) by guanosine triphosphatase (GTPase) translation factors drives protein synthesis by the ribosome. Allosteric coupling of GTP hydrolysis by elongation factor Tu (EF-Tu) at the ribosomal GTPase center to messenger RNA (mRNA) codon:aminoacyl-transfer RNA (aa-tRNA) anticodon recognition at the ribosomal decoding site is essential for accurate and rapid aa-tRNA selection. Here we use single-molecule methods to investigate the mechanism of action of the antibiotic thiostrepton and show that the GTPase center of the ribosome has at least two discrete functions during aa-tRNA selection: binding of EF-Tu(GTP) and stimulation of GTP hydrolysis by the factor. We separate these two functions of the GTPase center and assign each to distinct, conserved structural regions of the ribosome. The data provide a specific model for the coupling between the decoding site and the GTPase center during aa-tRNA selection as well as a general mechanistic model for ribosome-stimulated GTP hydrolysis by GTPase translation factors.
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>

            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Lindhout, D. A., Mckenna, S. A., Aitken, C. E., Liu, C. W., Puglisi, J. D.</span>
            <a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/2007-ProgressNuclearMagnetic-Lindhout-PKR.pdf" download><span id="title">PKR: A NMR perspective</span></a>
            <span id="publisher">PROGRESS IN NUCLEAR MAGNETIC RESONANCE SPECTROSCOPY, 2007; 51 (3): 199-215</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/2007-ProgressNuclearMagnetic-Lindhout-PKR.pdf" download><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span></a>
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>

            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Puglisi, E. V., Puglisi, J. D.</span>
            <a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/2007-FEBS_Letters_Puglisi_Probing.pdf" download><span id="title">Probing the conformation of human tRNA(3)(Lys) in solution by NMR</span></a>
            <span id="publisher">FEBS LETTERS, 2007; 581 (27): 5307-5314</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/2007-FEBS_Letters_Puglisi_Probing.pdf" download><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span></a>
                Human tRNA(3)(Lys) acts as a primer for the reverse transcription of human immunodeficiency virus genomic RNA. To form an initiation complex with genomic RNA, tRNA(3)(Lys) must reorganize its secondary structure. To provide a starting point for mechanistic studies of the formation of the initiation complex, we here present solution NMR investigations of human tRNA(3)(Lys). We use a straightforward set of NMR experiments to show that tRNA(3)(Lys) adopts a standard transfer ribonucleic acid tertiary structure in solution, and that Mg(2+) is required for this folding. The results underscore the power of NMR to reveal rapidly the conformation of RNAs.
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>

            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Kim, H. D., Puglisi, J. D., Chu, S.</span>
            <a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/2007-BiophysicsJournal-Kim-Fluctuations.pdf" download><span id="title">Fluctuations of transfer RNAs between classical and hybrid states</span></a>
            <span id="publisher">BIOPHYSICAL JOURNAL, 2007; 93 (10): 3575-3582</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/2007-BiophysicsJournal-Kim-Fluctuations.pdf" download><span id="title"><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span></a>
                Adjacent transfer RNAs (tRNAs) in the A- and P-sites of the ribosome are in dynamic equilibrium between two different conformations called classical and hybrid states before translocation. Here, we have used single-molecule fluorescence resonance energy transfer to study the effect of Mg(2+) on tRNA dynamics with and without an acetyl group on the A-site tRNA. When the A-site tRNA is not acetylated, tRNA dynamics do not depend on [Mg(2+)], indicating that the relative positions of the substrates for peptide-bond formation are not affected by Mg(2+). In sharp contrast, when the A-site tRNA is acetylated, Mg(2+) lengthens the lifetime of the classical state but does not change the lifetime of the hybrid state. Based on these findings, the classical state resembles a state with direct stabilization of tertiary structure by Mg(2+) ions whereas the hybrid state resembles a state with little Mg(2+)-assisted stabilization. The antibiotic viomycin, a translocation inhibitor, suppresses tRNA dynamics, suggesting that the enhanced fluctuations of tRNAs after peptide-bond formation drive spontaneous attempts at translocation by the ribosome.
              </div>
                <div id="webof-science"></div>
              <div id="pubmed"></div>

            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Alekseyev, V. Y., Liu, C. W., Cane, D. E., Puglisi, J. D., Khosla, C.</span>
            <a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/2007_Protein_Science_Alekseyev_Solution.pdf" download><span id="title">Solution structure and proposed domain-domain recognition interface of an acyl carrier protein domain from a modular polyketide synthase</span></a>
            <span id="publisher">PROTEIN SCIENCE, 2007; 16 (10): 2093-2107</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/2007_Protein_Science_Alekseyev_Solution.pdf" download><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span></a>
                Polyketides are a medicinally important class of natural products. The architecture of modular polyketide synthases (PKSs), composed of multiple covalently linked domains grouped into modules, provides an attractive framework for engineering novel polyketide-producing assemblies. However, impaired domain-domain interactions can compromise the efficiency of engineered polyketide biosynthesis. To facilitate the study of these domain-domain interactions, we have used nuclear magnetic resonance (NMR) spectroscopy to determine the first solution structure of an acyl carrier protein (ACP) domain from a modular PKS, 6-deoxyerythronolide B synthase (DEBS). The tertiary fold of this 10-kD domain is a three-helical bundle; an additional short helix in the second loop also contributes to the core helical packing. Superposition of residues 14-94 of the ensemble on the mean structure yields an average atomic RMSD of 0.64 +/- 0.09 Angstrom for the backbone atoms (1.21 +/- 0.13 Angstrom for all non-hydrogen atoms). The three major helices superimpose with a backbone RMSD of 0.48 +/- 0.10 Angstrom (0.99 +/- 0.11 Angstrom for non-hydrogen atoms). Based on this solution structure, homology models were constructed for five other DEBS ACP domains. Comparison of their steric and electrostatic surfaces at the putative interaction interface (centered on helix II) suggests a model for protein-protein recognition of ACP domains, consistent with the previously observed specificity. Site-directed mutagenesis experiments indicate that two of the identified residues influence the specificity of ACP recognition.
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>

            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Mckenna, S. A., Lindhout, D. A., Shimoike, T., Aitken, C. E., Puglisi, J. D.</span>
            <a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/2007-JBM-McKenna-Viral.pdf" download><span id="title">Viral dsRNA inhibitors prevent self-association and autophosphorylation of PKR </span></a>
            <span id="publisher">JOURNAL OF MOLECULAR BIOLOGY, 2007; 372 (1): 103-113</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/2007-JBM-McKenna-Viral.pdf" download><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span></a>
                Host response to viral RNA genomes and replication products represents an effective strategy to combat viral invasion. PKR is a Ser/Thr protein kinase that binds to double-stranded (ds)RNA, autophosphorylates its kinase domain, and subsequently phosphorylates eukaryotic initiation factor 2alpha (eIF2alpha). This results in attenuation of protein translation, preventing synthesis of necessary viral proteins. In certain DNA viruses, PKR function can be evaded by transcription of highly structured virus-encoded dsRNA inhibitors that bind to and inactivate PKR. We probe here the mechanism of PKR inhibition by two viral inhibitor RNAs, EBER(I) (from Epstein-Barr) and VA(I) (from human adenovirus). Native gel shift mobility assays and isothermal titration calorimetry experiments confirmed that the RNA-binding domains of PKR are sufficient and necessary for the interaction with dsRNA inhibitors. Both EBER(I) and VA(I) are effective inhibitors of PKR activation by preventing trans-autophosphorylation between two PKR molecules. The RNA inhibitors prevent self-association of PKR molecules, providing a mechanistic basis for kinase inhibition. A variety of approaches indicated that dsRNA inhibitors remain associated with PKR under activating conditions, as opposed to activator dsRNA molecules that dissociate due to reduced affinity for the phosphorylated form of PKR. Finally, we show using a HeLa cell extract system that inhibitors of PKR result in translational recovery by the protein synthesis machinery. These data indicate that inhibitory dsRNAs bind preferentially to the latent, dephosphorylated form of PKR and prevent dimerization that is required for trans-autophosphorylation.
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>

            </div>
          </div>



          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Lee, T., Blanchard, S. C., Kim, H. D., Puglisi, J. D., Chu, S.</span>
            <a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/2007-PNAS-Lee-TheRole.pdf" download><span id="title">The role of fluctuations in tRNA selection by the ribosome</span></a>
            <span id="publisher">PROCEEDINGS OF THE NATIONAL ACADEMY OF SCIENCES OF THE UNITED STATES OF AMERICA, 2007; 104 (34): 13661-13665</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/2007-PNAS-Lee-TheRole.pdf" download><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span></a>
                The detailed mechanism of how the ribosome decodes protein sequence information with an abnormally high accuracy, after 40 years of study, remains elusive. A critical element in selecting correct transfer RNA (tRNA) transferring correct amino acid is "induced fit" between the ribosome and tRNA. By using single-molecule methods, the induced fit mechanism is shown to position favorably the correct tRNA after initial codon recognition. We provide evidence that this difference in positioning and thermal fluctuations constitutes the primary mechanism for the initial selection of tRNA. This work demonstrates thermal fluctuations playing a critical role in the substrate selection by an enzyme.
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>

            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Varani, L., Bankovich, A. J., Liu, C. W., Colf, L. A., Jones, L. L., Kranz, D. M., Puglisi, J. D., Garcia, K. C.</span>
            <a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/2007-PNAS-Varani.pdf" download><span id="title">Solution mapping of T cell receptor docking footprints on peptide-MHC</span></a>
            <span id="publisher">PROCEEDINGS OF THE NATIONAL ACADEMY OF SCIENCES OF THE UNITED STATES OF AMERICA, 2007; 104 (32): 13080-13085</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/2007-PNAS-Varani.pdf" download><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span></a>
                T cell receptor (TCR) recognition of peptide-MHC (pMHC) is central to the cellular immune response. A large database of TCR-pMHC structures is needed to reveal general structural principles, such as whether the repertoire of TCR/MHC docking modes is dictated by a "recognition code" between conserved elements of the TCR and MHC genes. Although approximately 17 cocrystal structures of unique TCR-pMHC complexes have been determined, cocrystallization of soluble TCR and pMHC remains a major technical obstacle in the field. Here we demonstrate a strategy, based on NMR chemical shift mapping, that permits rapid and reliable analysis of the solution footprint made by a TCR when binding onto the pMHC surface. We mapped the 2C TCR binding interaction with its allogeneic ligand H-2Ld-QL9 and identified a group of NMR-shifted residues that delineated a clear surface of the MHC that we defined as the TCR footprint. We subsequently found that the docking footprint described by NMR shifts was highly accurate compared with a recently determined high-resolution crystal structure of the same complex. The same NMR footprint analysis was done on a high-affinity mutant of the TCR. The current work serves as a foundation to explore the molecular dynamics of pMHC complexes and to rapidly determine the footprints of many Ld-specific TCRs.
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>

            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Mckenna, S. A., Lindhout, D. A., Kim, I., Liu, C. W., Gelev, V. M., Wagner, G., Puglisi, J. D.</span>
            <a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/2007-J.+Biol.+Chem.-McKenna-Molecular.pdf" download><span id="title">Molecular framework for the activation of RNA-dependent protein kinase</span></a>
            <span id="publisher">JOURNAL OF BIOLOGICAL CHEMISTRY, 2007; 282 (15): 11474-11486</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/2007-J.+Biol.+Chem.-McKenna-Molecular.pdf" download><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span></a>
                The RNA-dependent protein kinase (PKR) plays an integral role in the antiviral response to cellular infection. PKR contains three distinct domains consisting of two conserved N-terminal double-stranded RNA (dsRNA)-binding domains, a C-terminal Ser-Thr kinase domain, and a central 80-residue linker. Despite rich structural and biochemical data, a detailed mechanistic explanation of PKR activation remains unclear. Here we provide a framework for understanding dsRNA-dependent activation of PKR using nuclear magnetic resonance spectroscopy, dynamic light scattering, gel filtration, and autophosphorylation kinetics. In the latent state, PKR exists as an extended monomer, with an increase in self-affinity upon dsRNA association. Subsequent phosphorylation leads to efficient release of dsRNA followed by a greater increase in self-affinity. Activated PKR displays extensive conformational perturbations within the kinase domain. We propose an updated model for PKR activation in which the communication between RNA binding, central linker, and kinase domains is critical in the propagation of the activation signal and for PKR dimerization.
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>

            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Uemura, S., Dorywalska, M., Lee, T., Kim, H. D., Puglisi, J. D., Chu, S.</span>
            <a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/2007-Nature-Uemura-Peptide.pdf" download><span id="title">Peptide bond formation destabilizes Shine-Dalgarno interaction on the ribosome</span></a>
            <span id="publisher">NATURE, 2007; 446 (7134): 454-457</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/2007-Nature-Uemura-Peptide.pdf" download><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span></a>
                The ribosome is a molecular machine that translates the genetic code contained in the messenger RNA into an amino acid sequence through repetitive cycles of transfer RNA selection, peptide bond formation and translocation. Here we demonstrate an optical tweezer assay to measure the rupture force between a single ribosome complex and mRNA. The rupture force was compared between ribosome complexes assembled on an mRNA with and without a strong Shine-Dalgarno (SD) sequence-a sequence found just upstream of the coding region of bacterial mRNAs, involved in translation initiation. The removal of the SD sequence significantly reduced the rupture force in complexes carrying an aminoacyl tRNA, Phe-tRNA(Phe), in the A site, indicating that the SD interactions contribute significantly to the stability of the ribosomal complex on the mRNA before peptide bond formation. In contrast, the presence of a peptidyl tRNA analogue, N-acetyl-Phe-tRNA(Phe), in the A site, which mimicked the post-peptidyl transfer state, weakened the rupture force as compared to the complex with Phe-tRNA(Phe), and the resultant force was the same for both the SD-containing and SD-deficient mRNAs. These results suggest that formation of the first peptide bond destabilizes the SD interaction, resulting in the weakening of the force with which the ribosome grips an mRNA. This might be an important requirement to facilitate movement of the ribosome along mRNA during the first translocation step.
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>

            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Kim, I., Mckenna, S. A., Puglisi, E. V., Puglisi, J. D.</span>
            <a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/2007-RNA-Kim-Rapid.pdf" download><span id="title">Rapid purification of RNAs using fast performance liquid chromatography (FPLC)</span></a>
            <span id="publisher">RNA-A PUBLICATION OF THE RNA SOCIETY, 2007; 13 (2): 289-294</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/2007-RNA-Kim-Rapid.pdf" download><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span></a>
                We present here an improved RNA purification method using fast performance liquid chromatography (FPLC) size-exclusion chromatography in place of denaturing polyacrylamide gel electrophoresis (PAGE). The method allows preparation of milligram quantities of pure RNA in a single day. As RNA oligonucleotides behave differently from globular proteins in the size-exclusion column, we present standard curves for RNA oligonucleotides of different lengths on both the Superdex 75 column and the Superdex 200 size-exclusion column. Using this approach, we can separate monomer from multimeric RNA species, purify the desired RNA product from hammerhead ribozyme reactions, and isolate refolded RNA that has aggregated after long-term storage. This methodology allows simple and rapid purification of RNA oligonucleotides for structural and biophysical studies.
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>

            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Mckenna, S. A., Lindhout, D. A., Shimoike, T., Puglisi, J. D.</span>
            <a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/2007-MethodsOfEnzymol-McKenna-Biophysical.pdf" download><span id="title">Biophysical and biochemical investigations of dsRNA-activated kinase PKR </span></a>
            <span id="publisher">TRANSLATION INITIATION: RECONSTITUTED SYSTEMS AND BIOPHYSICAL METHODS, 2007; 430: 373-396</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/2007-MethodsOfEnzymol-McKenna-Biophysical.pdf" download><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span></a>
                Protein kinase RNA-activated (PKR) is a serine/threonine kinase that contains an N-terminal RNA-binding domain (dsRNA) and a C-terminal kinase domain. On binding viral dsRNA molecules, PKR can become activated and phosphorylate cellular targets, such as eukaryotic translation initiation factor 2alpha (eIF-2alpha). Phosphorylation of eIF-2alpha results in attenuation of protein translation initiation. Therefore, PKR plays an integral role in the antiviral response to cellular infection. Here we provide a methodological framework for probing PKR function by use of assays for phosphorylation, RNA-protein stability, PKR dimerization, and in vitro translation. These methods are complemented by nuclear magnetic resonance approaches for probing structural features of PKR activation. Considerations required for both PKR and dsRNA sample preparation are also discussed.
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>

            </div>
          </div>



          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Uemura, S., Dorywalska, M., Lee, T., Kim, H. D., Puglisi, J. D., Chu, S.</span>
            <a href="" download><span id="title">Peptide bond formation destabilizes Shine-Dalgarno interaction on the ribosome</span></a>
            <span id="publisher">51st Annual Meeting of the Biophysical-Society, CELL PRESS. 2007: 571A–571A</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span>
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>

            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Mckenna, S. A., Lindhout, D. A., Aitken, C. E., Puglisi, J. D.</span>
            <a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/2007-Nato-McKenna-MolecularInsights.pdf" download><span id="title">Molecular insights into PKR activation by viral double-stranded RNA</span></a>
            <span id="publisher">NATO Advanced Study Institute Course on Structure and Biophysics,SPRINGER. 2007: 99–110</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/2007-Nato-McKenna-MolecularInsights.pdf" download><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span></a>
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>

            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Mckenna, S. A., Kim, I., Puglisi, E. V., Lindhout, D. A., Aitken, C. E., Marshall, R. A., Puglisi, J. D.</span>
            <a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/2007-NatureProtocols-McKenna-Purification.pdf" download><span id="title">Purification and characterization of transcribed RNAs using gel filtration chromatography</span></a>
            <span id="publisher">NATURE PROTOCOLS, 2007; 2 (12): 3270-3277</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/2007-NatureProtocols-McKenna-Purification.pdf" download><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span></a>
                RNA synthesis using in vitro transcription by phage T7 RNA polymerase allows preparation of milligram quantities of RNA for biochemical, biophysical and structural investigations. Previous purification approaches relied on gel electrophoretic or gravity-flow chromatography methods. We present here a protocol for the in vitro transcription of RNAs and subsequent purification using fast-performance liquid chromatography. This protocol greatly facilitates production of RNA in a single day from transcription to purification.
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>
            </div>
          </div>


          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Marshall RA, and Puglisi JD</span>
            <a href="" download><span id="title">NMR structural studies of aminoglycoside:RNA interaction</span></a>
            <span id="publisher">Aminoglycoside Antibiotics, Edited by DP Arya, John Wiley & Sons, 2007: 181-207</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span>
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>

            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">McKenna SA, Lindhout DA, Aitken CE, Puglisi JD</span>
            <a href="" download><span id="title">Molecular Insights Into PKR Activation by Viral Double-Stranded RNA</span></a>
            <span id="publisher">NATO Science Series, Springer, 2007; Ch 8: 99-110</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span>
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>

            </div>
          </div>


          <div class="year" id="2006">2006</div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Lee, T., Blanchard, S. C., Kim, H. D., Puglisi, J. D., Chu, S.</span>
            <a href="" download><span id="title">PHYS 336-Role of dynamics in the initial selection of tRNA by the ribosome</span></a>
            <span id="publisher">AMERICAN CHEMICAL SOCIETY, 2006 </span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span>
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>

            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Kim, I., Liu, C. W., Puglisi, J. D.</span>
            <a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/2006-JMB-Kim-Specific.pdf" download><span id="title">Specific recognition of HIV TAR RNA by the dsRNA binding domains (dsRBD1-dsRBD2) of PKR</span></a>
            <span id="publisher">JOURNAL OF MOLECULAR BIOLOGY, 2006; 358 (2): 430-442</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/2006-JMB-Kim-Specific.pdf" download><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span></a>
                PKR (double-stranded RNA-dependent protein kinase) is an important component of host defense to virus infection. Binding of dsRNA to two dsRBDs (double-stranded RNA binding domains) of PKR modulates its own kinase activation. How structural features of natural target RNAs, such as bulges and loops, have an effect on the binding to two dsRBDs of PKR still remains unclear. By using ITC and NMR, we show here that both the bulge and loop of TAR RNA are necessary for the high affinity binding to dsRBD1-dsRBD2 of PKR with 1:1 stoichiometry. The binding site for the dsRBD1-dsRBD2 spans from upper bulge to lower stem of the TAR RNA, based on chemical shift mapping. The backbone resonances in the 40 kDa TAR.dsRBD1-dsRBD2 were assigned. NMR chemical shift perturbation data suggest that the beta1-beta2 loop of the dsRBD1 interacts with the TAR RNA, whereas that of the dsRBD2 is less involved in the TAR RNA recognition. In addition, the residues of the interdomain linker between the dsRBD1 and the dsRBD2 also show large chemical perturbations indicating that the linker is involved in the recognition of TAR RNA. The results presented here provide the biophysical and spectroscopic basis for high-resolution structural studies, and show how local RNA structural features modulate recognition by dsRBDs.
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>
            </div>
          </div>

        <div class="year" id="2005">2005</div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Underwood, K. A., Swartz, J. R., Puglisi, J. D.</span>
            <a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/2005-Biotechnology_and_Bioengineering-Underwood.pdf" download><span id="title">Quantitative polysome analysis identifies limitations in bacterial cell-free protein synthesis</span></a>
            <span id="publisher">BIOTECHNOLOGY AND BIOENGINEERING, 2005; 91 (4): 425-435</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/2005-Biotechnology_and_Bioengineering-Underwood.pdf" download><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span></a>
                Cell-free protein synthesis (CFPS) is becoming increasingly used for protein production as yields increase and costs decrease. CFPS optimization efforts have focused primarily on energy supply and small molecule metabolism, though little is known about the protein synthesis machinery or what limits protein synthesis rates. Here, quantitative polysome profile analysis was used to characterize cell-free translation, thereby elucidating many kinetic parameters. The ribosome concentration in Escherichia coli-based CFPS reactions was 1.6 +/- 0.1 microM, with 72 +/- 4% actively translating at maximal protein synthesis rate. A translation elongation rate of 1.5 +/- 0.2 amino acids per second per ribosome and an initiation rate of 8.2 x 10(-9) +/- 0.3 x 10(-9) M/s, which correlates to, on average, one initiation every 60 +/- 9 s per mRNA, were determined. The measured CFPS initiation and elongation rates are an order of magnitude lower than the in vivo rates and further analysis identified elongation as the major limitation. Adding purified elongation factors (EFs) to CFPS reactions increased the ribosome elongation rate and protein synthesis rates and yields, as well as the translation initiation rate, indicating a possible coupling between initiation and elongation. Further examination of translation initiation in the cell-free system showed that the first initiation on an mRNA is slower than subsequent initiations. Our results demonstrate that polysome analysis is a valid tool to characterize cell-free translation and to identify limiting steps, that dilution of translation factors is a limitation of CFPS, and that CFPS is a useful platform for making novel observations about translation.
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>

            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Kim I, Lukavsky PJ, Otto GA, Liu CW, Puglisi JD</span>
            <a href="" download><span id="title">Using NMR to Study large RNAs: Case Study of the HCV IRES</span></a>
            <span id="publisher">NATO ASI Science Series, Ios Press, 2005; 364: 75-90</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span>
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>

            </div>
          </div>



          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Dorywalska, M., Blanchard, S. C., Gonzalez, R. L., Kim, H. D., Chu, S., Puglisi, J. D.</span>
            <a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/2005-NucleicAcidResearch-Magdalena-Site-Specific.pdf" download><span id="title">Site-specific labeling of the ribosome for single-molecule spectroscopy</span></a>
            <span id="publisher">NUCLEIC ACIDS RESEARCH, 2005; 33 (1): 182-189</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/2005-NucleicAcidResearch-Magdalena-Site-Specific.pdf" download><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span></a>
                Single-molecule fluorescence spectroscopy can reveal mechanistic and kinetic details that may not be observed in static structural and bulk biochemical studies of protein synthesis. One approach requires site-specific and stable attachment of fluorophores to the components of translation machinery. Fluorescent tagging of the ribosome is a prerequisite for the observation of dynamic changes in ribosomal conformation during translation using fluorescence methods. Modifications of the ribosomal particle are difficult due to its complexity and high degree of sequence and structural conservation. We have developed a general method to label specifically the prokaryotic ribosome by hybridization of fluorescent oligonucleotides to mutated ribosomal RNA. Functional, modified ribosomes can be purified as a homogenous population, and fluorescence can be monitored from labeled ribosomal complexes immobilized on a derivatized quartz surface.
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>

            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Lukavsky, P. J., Puglisi, J. D.</span>
            <a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/2005-MethodsEnzymol-Lukavsky.pdf" download><span id="title">Structure determination of large biological RNAs</span></a>
            <span id="publisher">NUCLEAR MAGNETIC RESONANCE OF BIOLOGICAL MACROMOLECULES, PART C, 2005; 394: 399-416</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/2005-MethodsEnzymol-Lukavsky.pdf" download><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span></a>
                Complex RNA structures regulate many biological processes but are often too large for structure determination by nuclear magnetic resonance (NMR) methods. We determined the solution structure of domain II of the hepatitis C viral internal ribosome entry site (HCV IRES), a 25-kDa RNA, using a novel NMR approach. Conventional short-range, distance, and torsion angle NMR restraints were combined with long-range, angular restraints derived from residual dipolar couplings (RDCs) to improve both the local and global precision of the structure. This powerful approach should be generally applicable to the NMR structure determination of large, modular RNAs.
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>
            </div>
          </div>

          <div class="year" id="2004">2004</div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Otto, G. A., Puglisi, J. D.</span>
            <a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/2004-Cell-Otto.pdf" download><span id="title">The pathway of HCVIRES-mediated translation initiation</span></a>
            <span id="publisher">CELL, 2004; 119 (3): 369-380</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/2004-Cell-Otto.pdf" download><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span></a>
                The HCV internal ribosome entry site (IRES) directly regulates the assembly of translation initiation complexes on viral mRNA by a sequential pathway that is distinct from canonical eukaryotic initiation. The HCV IRES can form a binary complex with an eIF-free 40S ribosomal subunit. Next, a 48S-like complex assembles at the AUG initiation codon upon association of eIF3 and ternary complex. 80S complex formation is rate limiting and follows the GTP-dependent association of the 60S subunit. Efficient assembly of the 48S-like and 80S complexes on the IRES mRNA is dependent upon maintenance of the highly conserved HCV IRES structure. This revised model of HCV IRES translation initiation provides a context to understand the function of different HCV IRES domains during translation initiation.
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>

            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Blanchard, S. C., Gonzalez, R. L., Kim, H. D., Chu, S., Puglisi, J. D.</span>
            <a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/2004_Nsmb_blanchard.pdf" download><span id="title">tRNA selection and kinetic proofreading in translation</span></a>
            <span id="publisher">NATURE STRUCTURAL & MOLECULAR BIOLOGY, 2004; 11 (10): 1008-1014</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/2004_Nsmb_blanchard.pdf" download><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span></a>
                Using single-molecule methods we observed the stepwise movement of aminoacyl-tRNA (aa-tRNA) into the ribosome during selection and kinetic proofreading using single-molecule fluorescence resonance energy transfer (smFRET). Intermediate states in the pathway of tRNA delivery were observed using antibiotics and nonhydrolyzable GTP analogs. We identified three unambiguous FRET states corresponding to initial codon recognition, GTPase-activated and fully accommodated states. The antibiotic tetracycline blocks progression of aa-tRNA from the initial codon recognition state, whereas cleavage of the sarcin-ricin loop impedes progression from the GTPase-activated state. Our data support a model in which ribosomal recognition of correct codon-anticodon pairs drives rotational movement of the incoming complex of EF-Tu-GTP-aa-tRNA toward peptidyl-tRNA during selection on the ribosome. We propose a mechanistic model of initial selection and proofreading.
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>

            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Blanchard, S. C., Kim, H. D., Gonzalez, R. L., Puglisi, J. D., Chu, S.</span>
            <a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/2004-PNAS-Blanchard-tRNA.pdf" download><span id="title">tRNA dynamics on the ribosome during translation</span></a>
            <span id="publisher">PROCEEDINGS OF THE NATIONAL ACADEMY OF SCIENCES OF THE UNITED STATES OF AMERICA, 2004; 101 (35): 12893-12898</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/2004-PNAS-Blanchard-tRNA.pdf" download><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span></a>
                Using single-molecule fluorescence spectroscopy, time-resolved conformational changes between fluorescently labeled tRNA have been characterized within surface-immobilized ribosomes proceeding through a complete cycle of translation elongation. Fluorescence resonance energy transfer was used to observe aminoacyl-tRNA (aa-tRNA) stably accommodating into the aminoacyl site (A site) of the ribosome via a multistep, elongation factor-Tu dependent process. Subsequently, tRNA molecules, bound at the peptidyl site and A site, fluctuate between two configurations assigned as classical and hybrid states. The lifetime of classical and hybrid states, measured for complexes carrying aa-tRNA and peptidyl-tRNA at the A site, shows that peptide bond formation decreases the lifetime of the classical-state tRNA configuration by approximately 6-fold. These data suggest that the growing peptide chain plays a role in modulating fluctuations between hybrid and classical states. Single-molecule fluorescence resonance energy transfer was also used to observe aa-tRNA accommodation coupled with elongation factor G-mediated translocation. Dynamic rearrangements in tRNA configuration are also observed subsequent to the translocation reaction. This work underscores the importance of dynamics in ribosome function and demonstrates single-particle enzymology in a system of more than two components.
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>

            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Lukavsky, P. J., Puglisi, J. D.</span>
            <a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/2004-RNA-LUKAVSKY-Large.pdf" download><span id="title">Large-scale preparation and purification of polyacrylamide-free RNA oligonucleotides</span></a>
            <span id="publisher">RNA-A PUBLICATION OF THE RNA SOCIETY, 2004; 10 (5): 889-893</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/2004-RNA-LUKAVSKY-Large.pdf" download><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span></a>
                We present a fast and simple protocol for large-scale preparation and purification of RNA oligonucleotides. RNA oligonucleotides are prepared by in vitro transcription with T7 RNA polymerase from linearized plasmid DNA templates constructed by PCR. In place of denaturing polyacrylamide gel electrophoresis (PAGE), size-exclusion chromatography is employed to purify the RNA oligonucleotide from the transcription mixture yielding >99% pure RNA product. In contrast to PAGE-based purification, the gel filtration method does not require denaturation of the RNA oligonucleotide, which is desirable for larger RNAs, and the product is free of low-molecular-weight acrylamide contaminants, which greatly benefits NMR, crystallographic, and other biophysical studies of large RNAs and RNA-protein complexes.
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>
            </div>
          </div>


          <div class="year" id="2003">2003</div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Runyon, S. T., Puglisi, J. D.</span>
            <a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/2003-JACS-Runyon.pdf" download><span id="title">Design of a cyclic peptide that targets a viral RNA</span></a>
            <span id="publisher">JOURNAL OF THE AMERICAN CHEMICAL SOCIETY, 2003; 125 (51): 15704-15705</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/2003-JACS-Runyon.pdf" download><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span></a>
                The Tat protein controls transcription in lentiviruses such as HIV. A cyclic peptide analog of the RNA binding domain of the bovine immunodeficiency virus (BIV) Tat protein is shown to bind specifically to its target RNA stem loop. NMR data indicate a similar mode of binding of linear and cyclic peptides.
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>

            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Schroeder, S. J., Fountain, M. A., Kennedy, S. D., Lukavsky, P. J., Puglisi, J. D., Krugh, T. R., Turner, D. H.</span>
            <a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/2003-biochemistry-Schroeder-Therodynamically.pdf" download><span id="title">Thermodynamic stability and structural features of the J4/5 loop in a Pneumocystis carinii group I intron</span></a>
            <span id="publisher">BIOCHEMISTRY, 2003; 42 (48): 14184-14196</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/2003-biochemistry-Schroeder-Therodynamically.pdf" download><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span></a>
                The J4/5 loop of the group I intron in the mouse-derived fungal pathogen Pneumocystis carinii is the docking site for the first step of the RNA-catalyzed self-splicing reaction and thus is a model of a potential drug target. This purine-rich asymmetric internal loop, 5'GGAAG/3'UAGU, is also thermodynamically more stable than other internal loops with two GU closing pairs and three nucleotides opposite two nucleotides. The results from optical melting, nuclear magnetic resonance spectroscopy, and functional group substitution experiments suggest that the GU closing pairs form and that sheared GA pairs form in the internal loop. The NMR spectra show evidence of conformational dynamics, and several GA pairings are possible. Thus, this dynamic loop presents several possible structures for potential binding of drugs that target group I self-splicing introns. The results also contribute to understanding the structural and dynamic basis for the function and thermodynamic stability of this loop.
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>
            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Lukavsky, P. J., Kim, I., Otto, G. A., Puglisi, J. D.</span>
            <a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/2003_NatureStructuralBiology_Lukavsky.pdf" download><span id="title">Structure of HCV IRES domain II determined by NMR</span></a>
            <span id="publisher">NATURE STRUCTURAL BIOLOGY, 2003; 10 (12): 1033-1038</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/2003_NatureStructuralBiology_Lukavsky.pdf" download><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span></a>
                Complex RNA structures regulate many biological processes, but are often too large for structure determination by NMR methods. The 5' untranslated region (5' UTR) of the hepatitis C viral (HCV) RNA genome contains an internal ribosome entry site (IRES) that binds to 40S ribosomal subunits with high affinity and specificity to control translation. Domain II of the HCV IRES forms a 25-kDa folded subdomain that may alter ribosome conformation. We report here the structure of domain II as determined using an NMR approach that combines short- and long-range structural data. Domain II adopts a distorted L-shape structure, and its overall shape in the free form is markedly similar to its 40S subunit-bound form; this suggests how domain II may modulate 40S subunit conformation. The results show how NMR can be used for structural analysis of large biological RNAs.
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>

            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Li, Q., Khosla, C., Puglisi, J. D., Liu, C. W.</span>
            <a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/2003-Biochemistry-li-Solution.pdf" download><span id="title">Solution structure and backbone dynamics of the holo form of the frenolicin acyl carrier protein</span></a>
            <span id="publisher">BIOCHEMISTRY, 2003; 42 (16): 4648-4657</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/2003-Biochemistry-li-Solution.pdf" download><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span></a>
                During polyketide biosynthesis, acyl carrier proteins (ACPs) perform the central role of transferring polyketide intermediates between active sites of polyketide synthase. The 4'-phosphopantetheine prosthetic group of a holo-ACP is a long and flexible arm that can reach into different active sites and provide a terminal sulfhydryl group for the attachment of acyl groups through a thioester linkage. We have determined the solution structure and characterized backbone dynamics of the holo form of the frenolicin acyl carrier protein (fren holo-ACP) by nuclear magnetic resonance (NMR). Unambiguous assignments were made for 433 hydrogen atoms, 333 carbon atoms, and 84 nitrogen atoms, representing a total of 94.6% of the assignable atoms in this protein. From 879 meaningful NOEs and 45 angle constraints, a family of 24 structures has been calculated. The solution structure is composed of three major alpha-helices packed in a bundle with three additional short helices in intervening loops; one of the short helices slowly exchanges between two conformations. Superposition of the major helical regions on the mean structure yields average atomic rmsd values of 0.49 +/- 0.09 and 0.91 +/- 0.08 A for backbone and non-hydrogen atoms, respectively. Although the three-helix bundle fold is conserved among acyl carrier proteins involved in fatty acid synthases and polyketide synthases, a detailed comparison revealed that ACPs from polyketide biosynthetic pathways are more related to each other in tertiary fold than to their homologues from fatty acid biosynthetic pathways. Comparison of the free form of ACPs (NMR structures of fren ACP and the Bacillus subtilis ACP) with the substrate-bound form of ACP (crystal structure of butyryl-ACP from Escherichia coli) suggests that conformational exchange plays a role in substrate binding.
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>

            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Lynch, S. R., Gonzalez, R. L., Puglisi, J. D.</span>
            <a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/2003-Structure-Lynch.pdf" download><span id="title">Comparison of x-ray crystal structure of the 30S subunit-antibiotic complex with NMR structure of decoding site oligonucleotide-paromomycin complex</span></a>
            <span id="publisher">STRUCTURE, 2003; 11 (1): 43-53</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/2003-Structure-Lynch.pdf" download><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span></a>
                Aminoglycoside antibiotics that bind to 16S ribosomal RNA in the aminoacyl-tRNA site (A site) cause misreading of the genetic code and inhibit translocation. Structures of an A site RNA oligonucleotide free in solution and bound to the aminoglycosides paromomycin or gentamicin C1a have been determined by NMR. Recently, the X-ray crystal structure of the entire 30S subunit has been determined, free and bound to paromomycin. Distinct differences were observed in the crystal structure, particularly at A1493. Here, the NMR structure of the oligonucleotide-paromomycin complex was determined with higher precision and is compared with the X-ray crystal structure of the 30S subunit complex. The comparison shows the validity of both structures in identifying critical interactions that affect ribosome function.
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>

            </div>
          </div>

          <div class="year" id="2002">2002</div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Kim, I., Lukavsky, P. J., Puglisi, J. D.</span>
            <a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/2002_JACS_Kim_NMR.pdf" download><span id="title">NMR study of 100 kDa HCV IRES RNA using segmental isotope labeling</span></a>
            <span id="publisher">JOURNAL OF THE AMERICAN CHEMICAL SOCIETY, 2002; 124 (32): 9338-9339</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/2002_JACS_Kim_NMR.pdf" download><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span></a>
                RNA NMR is hindered by the large size of most biological RNAs. We present here a simple method for segmental isotopic labeling of an RNA fragment within the context of a larger RNA. The methodology uses transcription and ribozyme cleavage to prepare appropriate ends for RNA ligase catalyzed ligation. We demonstrate that a 64 nucleotide domain of the Hepatitis C virus internal ribosome entry site (IRES) RNA adopts an independently folded domain within the context of the intact, 100 kDa IRES.
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>
            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Otto, G. A., Lukavsky, P. J., Lancaster, A. M., Sarnow, P., Puglisi, J. D.</span>
            <a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/2002-RNA-Otto-Ribosomal.pdf" download><span id="title">Ribosomal proteins mediate the hepatitis C virus IRES-HeLa 40S interaction</span></a>
            <span id="publisher">RNA-A PUBLICATION OF THE RNA SOCIETY, 2002; 8 (7): 913-923</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/2002-RNA-Otto-Ribosomal.pdf" download><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span></a>
              Translation of the hepatitis C virus genomic RNA is mediated by an internal ribosome entry site (IRES). The 330-nt IRES RNA forms a binary complex with the small 40S ribosomal subunit as a first step in translation initiation. Here chemical probing and 4-thiouridine-mediated crosslinking are used to characterize the interaction of the HCV IRES with the HeLa 40S subunit. No IRES-18S rRNA contacts were detected, but several specific crosslinks to 40S ribosomal proteins were observed. The identity of the crosslinked proteins agrees well with available structural information and provides new insights into HCV IRES function. The protein-rich surface of the 40S subunit thus mediates the IRES-ribosome interaction.
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>
            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Yoshizawa, S., Fourmy, D., Eason, R. G., Puglisi, J. D.</span>
            <a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/2002-ACS-Yoshizawa-Sequence.pdf" download><span id="title">Sequence-specific recognition of the major groove of RNA by deoxystreptamine</span></a>
            <span id="publisher">BIOCHEMISTRY, 2002; 41 (20): 6263-6270</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/2002-ACS-Yoshizawa-Sequence.pdf" download><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span></a>
                Aminoglycoside antibiotics specifically interact with a variety of RNA sequences, and in particular with the decoding region of 16S ribosomal RNA in the aminoacyl tRNA acceptor site (A-site). Ring II of aminoglycosides (2-deoxystreptamine) is the most conserved element among aminoglycoside antibiotics that bind to the A-site. NMR structures of aminoglycoside-A-site RNA complexes suggested that the 2-deoxystreptamine core of aminoglycosides specifically recognizes (5')G-U(3') and potentially (5')G-G(3') or (5')U-G(3') steps in the major groove of RNA. Here, we show that isolated deoxystreptamine specifically interacts with G-U steps within the major groove of the A-site RNA. The bulge residue of A-site RNA is required to open the major groove for accommodation of deoxystreptamine. The chemical groups of deoxystreptamine presented to the RNA by the framework of the 6-carbon ring modulate RNA recognition.
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>
            </div>
          </div>

          <div class="year" id="2001">2001</div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Lukavsky, P. J., Puglisi, J. D.</span>
            <a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/2001-Methods-Lukavsky.pdf" download><span id="title">RNAPack: An integrated NMR approach to RNA structure determination </span></a>
            <span id="publisher">METHODS, 2001; 25 (3): 316-332</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/2001-Methods-Lukavsky.pdf" download><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span></a>
                Over the last decade, a vast number of useful nuclear magnetic resonance (NMR) experiments have been developed and successfully employed to determine the structure and dynamics of RNA oligonucleotides. Despite this progress, high-resolution RNA structure determination by NMR spectroscopy still remains a lengthy process and requires programming and extensive calibrations to perform NMR experiments successfully. To accelerate RNA structure determination by NMR spectroscopy, we have designed and programmed a package of RNA NMR experiments, called RNAPack. The user-friendly package contains a set of semiautomated single, double, and triple resonance NMR experiments, which are fully optimized for high-resolution RNA solution structure determination on Varian NMR spectrometers. RNAPack provides an autocalibration feature that allows rapid calibration of all NMR experiments in a single step and thereby speeds up the NMR data collection and eliminates user errors. In our laboratory, we have successfully employed this technology to solve RNA solution structures of domains of the internal ribosome entry site of the genomic hepatitis C viral RNA in less than 3 months. RNAPack therefore makes NMR spectroscopy an attractive and rapid structural tool and allows integration of atomic resolution structural information into biochemical studies of large RNA systems.
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>
            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Recht, M. I., Puglisi, J. D.</span>
            <a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/2001-AntimicrobAgentsChemother-Recht.pdf" download><span id="title">Aminoglycoside resistance with homogeneous and heterogeneous populations of antibiotic-resistant ribosomes</span></a>
            <span id="publisher">ANTIMICROBIAL AGENTS AND CHEMOTHERAPY, 2001; 45 (9): 2414-2419</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/2001-AntimicrobAgentsChemother-Recht.pdf" download><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span></a>
                Aminoglycosides bind to rRNA in the small subunit of the bacterial ribosome. Mutations in the decoding region of 16S rRNA confer resistance to specific subsets of aminoglycoside antibiotics. The two major classes of 2-deoxystreptamine aminoglycosides are the 4,5- and the 4,6-disubstituted antibiotics. Antibiotics of the 4,5-disubstituted class include neomycin, paromomycin, and ribostamycin. Gentamicins and kanamycins belong to the 4,6-disubstituted class of aminoglycosides. Structural studies indicated the potential importance of position 1406 (Escherichia coli numbering) in the binding of ring III of the 4,6-disubstituted class of aminoglycosides to 16S rRNA. We have introduced a U1406-to-A mutation in a plasmid-encoded copy of E. coli 16S rRNA which has been expressed either in a mixture with wild-type ribosomes or in a strain in which all rRNA is transcribed from the plasmid-encoded rrn operon. High-level resistance to many of the 4,6-disubstituted aminoglycosides is observed only when all the rRNA contains the U1406-to-A mutation. In contrast to the partial dominance of resistance observed with other mutations in the decoding region, there is a dominance of sensitivity with the 1406A mutation. Chemical footprinting experiments indicate that resistance arises from a reduced affinity of the antibiotic for the rRNA target. These results demonstrate that although position 1406 is an important determinant in the binding and action of the 4,6-disubstituted aminoglycosides, other rRNA mutations that perturb the binding of ring I of both classes of 2-deoxystreptamine aminoglycosides confer higher levels of resistance as well as a partial dominance of resistance.
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>
            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Blanchard, S. C., Puglisi, J. D.</span>
            <a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/PNAS-2001-Blanchard-Solution.pdf" download><span id="title">Solution structure of the A loop of 23S ribosomal RNA</span></a>
            <span id="publisher">PROCEEDINGS OF THE NATIONAL ACADEMY OF SCIENCES OF THE UNITED STATES OF AMERICA, 2001; 98 (7): 3720-3725</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/PNAS-2001-Blanchard-Solution.pdf" download><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span></a>
                The A loop is an essential RNA component of the ribosome peptidyltransferase center that directly interacts with aminoacyl (A)-site tRNA. The A loop is highly conserved and contains a ubiquitous 2'-O-methyl ribose modification at position U2552. Here, we present the solution structure of a modified and unmodified A-loop RNA to define both the A-loop fold and the structural impact of the U2552 modification. Solution data reveal that the A-loop RNA has a compact structure that includes a noncanonical base pair between C2556 and U2552. NMR evidence is presented that the N3 position of C2556 has a shifted pKa and that protonation at C2556-N3 changes the C-U pair geometry. Our data indicate that U2552 methylation modifies the A-loop fold, in particular the dynamics and position of residues C2556 and U2555. We compare our structural data with the structure of the A loop observed in a recent 50S crystal structure [Ban, N., Nissen, P., Hansen, J., Moore, P. B. & Steitz, T. A. (2000) Science 289, 905--920; Nissen, P., Hansen, J., Ban, N., Moore, P. B. & Steitz, T. A. (2000) Science 289, 920--930]. The solution and crystal structures of the A loop are dramatically different, suggesting that a structural rearrangement of the A loop must occur on docking into the peptidyltransferase center. Possible roles of this docking event, the shifted pKa of C2556 and the U2552 2'-O-methylation in the mechanism of translation, are discussed.
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>
            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Lynch, S. R., Puglisi, J. D.</span>
            <a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/2001-JMB-Lynch-StructuralOrigins.pdf" download><span id="title">Structural origins of aminoglycoside specificity for prokaryotic ribosomes </span></a>
            <span id="publisher">JOURNAL OF MOLECULAR BIOLOGY, 2001; 306 (5): 1037-1058</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/2001-JMB-Lynch-StructuralOrigins.pdf" download><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span></a>
                Aminoglycoside antibiotics, including paromomycin, neomycin and gentamicin, target a region of highly conserved nucleotides in the decoding region aminoacyl-tRNA site (A site) of 16 S rRNA on the 30 S subunit. Change of a single nucleotide, A1408 to G, reduces the affinity of many aminoglycosides for the ribosome; G1408 distinguishes between prokaryotic and eukaryotic ribosomes. The structures of a prokaryotic decoding region A-site oligonucleotide free in solution and bound to the aminoglycosides paromomycin and gentamicin C1a were determined previously. Here, the structure of a eukaryotic decoding region A-site oligonucleotide bound to paromomycin has been determined using NMR spectroscopy and compared to the prokaryotic A-site-paromomycin structure. A conformational change in three adenosine residues of an internal loop, critical for high-affinity antibiotic binding, was observed in the prokaryotic RNA-paromomycin complex in comparison to its free form. This conformational change is not observed in the eukaryotic RNA-paromomycin complex, disrupting the binding pocket for ring I of the antibiotic. The lack of the conformational change supports footprinting and titration calorimetry data that demonstrate approximately 25-50-fold weaker binding of paromomycin to the eukaryotic decoding-site oligonucleotide. Neomycin, which is much less active against Escherichia coli ribosomes with an A1408G mutation, binds non-specifically to the oligonucleotide. These results suggest that eukaryotic ribosomal RNA has a shallow binding pocket for aminoglycosides, which accommodates only certain antibiotics.
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>
            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Lynch, S. R., Puglisi, J. D.</span>
            <a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/2001-JMB-Lynch-StructureEukaryotic.pdf" download><span id="title">Structure of a eukaryotic decoding region A-site RNA</span></a>
            <span id="publisher">JOURNAL OF MOLECULAR BIOLOGY, 2001; 306 (5): 1023-1035</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span>
                The aminoglycoside antibiotics target a region of highly conserved nucleotides in the aminoacyl-tRNA site (A site) of 16 S RNA on the 30 S subunit. The structures of a prokaryotic decoding region A-site oligonucleotide free in solution and bound to the aminoglycosides paromomycin and gentamicin C1A have been determined. Here, the structure of a eukaryotic decoding region A-site oligonucleotide has been determined using homonuclear and heteronuclear NMR spectroscopy, and compared to the unbound prokaryotic rRNA structure. The two structures are similar, with a U1406-U1495 base-pair, a C1407-G1494 Watson-Crick base-pair, and a G1408-A1493 base-pair instead of the A1408-A1493 base-pair of the prokaryotic structure. The two structures differ in the orientation of the 1408 position with respect to A1493; G1408 is rotated toward the major groove, which is the binding pocket for aminoglycosides. The structures also differ in the stacking geometry of G1494 on A1493, which could have slight long-range conformational effects.
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>
            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Puglisi, J. D., Kim, I., Lukavsky, P., Otto, G., Lancaster, A., Sarnow, P.</span>
            <a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/2001-NucleicAcidResearch-Puglisi-Structural.pdf" download><span id="title">Structural and functional investigation of the hepatitis C virus IRES.</span></a>
            <span id="publisher">Nucleic acids research. Supplement (2001), 2001: 263-?</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/2001-NucleicAcidResearch-Puglisi-Structural.pdf" download><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span></a>

              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>
            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Puglisi, J. D., Yoshizawa, S., Fourmy, D.</span>
            <a href="" download><span id="title">Molecular origins of ribosomal fidelity</span></a>
            <span id="publisher">4th NATO Advanced-Study-Institute on Dynamics, Structure and Function of Biological Macromolecules, I O S PRESS. 2001: 177–185</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span>

              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>
            </div>
          </div>

        <div class="year" id="2000">2000</div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Lukavsky, P. J., Otto, G. A., Lancaster, A. M., Sarnow, P., Puglisi, J. D.</span>
            <a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/2000-NatureAmerica-Lukavsky-Structures.pdf" download><span id="title">Structures of two RNA domains essential for hepatitis C virus internal ribosome entry site function</span></a>
            <span id="publisher">NATURE STRUCTURAL BIOLOGY, 2000; 7 (12): 1105-1110</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/2000-NatureAmerica-Lukavsky-Structures.pdf" download><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span></a>
              Translation of the hepatitis C virus (HCV) polyprotein is initiated at an internal ribosome entry site (IRES) element in the 5' untranslated region of HCV RNA. The HCV IRES element interacts directly with the 40S subunit, and biochemical experiments have implicated RNA elements near the AUG start codon as required for IRES-40S subunit complex formation. The data we present here show that two RNA stem loops, domains IIId and IIIe, are involved in IRES-40S subunit interaction. The structures of the two RNA domains were solved by NMR spectroscopy and reveal structural features that may explain their role in IRES function.
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>
            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Puglisi, J. D., Blanchard, S. C., Green, R.</span>
            <a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/2000-NatureAmerica-Puglisi.pdf" download><span id="title">Approaching translation at atomic resolution </span></a>
            <span id="publisher">NATURE STRUCTURAL BIOLOGY, 2000; 7 (10): 855-861</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/2000-NatureAmerica-Puglisi.pdf" download><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span></a>
                Atomic resolution structures of 50S and 30S ribosomal particles have recently been solved by X-ray diffraction. These ribosomal structures show often unusual folds of ribosomal RNAs and proteins, and provide molecular explanations for fundamental aspects of translation. In the 50S structure, the active site for peptide bond formation was localized and found to consist of RNA. The ribosome is thus a ribozyme. In the 30S structures, tRNA binding sites were located, and molecular mechanisms for ribosomal fidelity were proposed. The 30S subunit particle has three globular domains, and relative movements of these domains may be required for translocation of the ribosome during protein synthesis. The structures are consistent with and rationalize decades of biochemical analysis of translation and usher in a molecular age in understanding the ribosome.
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>
            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Lynch, S. R., Puglisi, J. D.</span>
            <a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/2000-ChemicalSociety-Lynch-Application.pdf" download><span id="title">Application of residual dipolar coupling measurements to identify conformational changes in RNA induced by antibiotics</span></a>
            <span id="publisher">JOURNAL OF THE AMERICAN CHEMICAL SOCIETY, 2000; 122 (32): 7853-7854</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/2000-ChemicalSociety-Lynch-Application.pdf" download><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span></a>

              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>
            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Dahlquist, K. D., Puglisi, J. D.</span>
            <a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/2000-JMB-Dahlquist.pdf" download><span id="title">Interaction of translation initiation factor IF1 with the E-coli ribosomal A site</span></a>
            <span id="publisher">JOURNAL OF MOLECULAR BIOLOGY, 2000; 299 (1): 1-15</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/2000-JMB-Dahlquist.pdf" download><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span></a>
                Initiation Factor 1 (IF1) is required for the initiation of translation in Escherichia coli. However, the precise function of IF1 remains unknown. Current evidence suggests that IF1 is an RNA-binding protein that sits in the A site of the decoding region of 16 S rRNA. IF1 binding to 30 S subunits changes the reactivity of nucleotides in the A site to chemical probes. The N1 position of A1408 is enhanced, while the N1 positions of A1492 and A1493 are protected from reactivity with dimethyl sulfate (DMS). The N1-N2 positions of G530 are also protected from reactivity with kethoxal. Quantitative footprinting experiments show that the dissociation constant for IF1 binding to the 30 S subunit is 0.9 microM and that IF1 also alters the reactivity of a subset of Class III sites that are protected by tRNA, 50 S subunits, or aminoglycoside antibiotics. IF1 enhances the reactivity of the N1 position of A1413, A908, and A909 to DMS and the N1-N2 positions of G1487 to kethoxal. To characterize this RNA-protein interaction, several ribosomal mutants in the decoding region RNA were created, and IF1 binding to wild-type and mutant 30 S subunits was monitored by chemical modification and primer extension with allele-specific primers. The mutations C1407U, A1408G, A1492G, or A1493G disrupt IF1 binding to 30 S subunits, whereas the mutations G530A, U1406A, U1406G, G1491U, U1495A, U1495C, or U1495G had little effect on IF1 binding. Disruption of IF1 binding correlates with the deleterious phenotypic effects of certain mutations. IF1 binding to the A site of the 30 S subunit may modulate subunit association and the fidelity of tRNA selection in the P site through conformational changes in the 16 S rRNA.
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>
            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Puglisi, J. D.</span>
            <a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/2000-NatureAmerica-Puglisi-mRNA.pdf" download><span id="title">mRNA processing: The 3 '-end justifies the means</span></a>
            <span id="publisher">NATURE STRUCTURAL BIOLOGY, 2000; 7 (4): 263-264</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/2000-NatureAmerica-Puglisi-mRNA.pdf" download><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span></a>

              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>
            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Lynch, S. R., Recht, M. I., Puglisi, J. D.</span>
            <a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/2001-RNALigandInteractions-Lynch-Biochemical.pdf" download><span id="title">Biochemical and nuclear magnetic resonance studies of aminoglycoside-RNA complexes</span></a>
            <span id="publisher">RNA-LIGAND INTERACTIONS PT A, 2000; 317: 240-261 </span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/2001-RNALigandInteractions-Lynch-Biochemical.pdf" download><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span></a>

              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>
            </div>
          </div>


          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Puglisi, J. D., Blanchard, S. C., Dahlquist, K. D., Eason, R. G., Fourmy, D., Lynch, S. R., Recht, M. I., Yoshizawa, S.</span>
            <a href="" download><span id="title">Aminoglycoside antibiotics and decoding</span></a>
            <span id="publisher">International Ribosome Conference, AMER SOC MICROBIOLOGY. 2000: 419–429</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span>

              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>
            </div>
          </div>

        <div class="year" id="1999">1999</div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Green, R., Puglisi, J. D.</span>
            <a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/1999-NaureAmerica-Green-TheRibosome.pdf" download><span id="title">The ribosome revealed</span></a>
            <span id="publisher">NATURE STRUCTURAL BIOLOGY, 1999; 6 (11): 999-1003</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/1999-NaureAmerica-Green-TheRibosome.pdf" download><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span></a>
                Several recently reported structures reveal the details of ribosome architecture and provide new insights into the mechanism of protein synthesis.
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>
            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Yoshizawa, S., Fourmy, D., Puglisi, J. D.</span>
            <a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/1999-ScienceMag-Yoshizawa.pdf" download><span id="title">Recognition of the codon-anticodon helix by ribosomal RNA</span></a>
            <span id="publisher">SCIENCE, 1999; 285 (5434): 1722-1725</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/1999-ScienceMag-Yoshizawa.pdf" download><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span></a>
                Translational fidelity is established by ribosomal recognition of the codon-anticodon interaction within the aminoacyl-transfer RNA
                (tRNA) site (A site) of the ribosome. Experiments are presented that reveal possible contacts between 16S ribosomal RNA and the
                codon-anticodon complex. N1 methylation of adenine at position 1492 (A1492) and A1493 interfered with A-site tRNA binding. Mutation
                of A1492 and A1493 to guanine or cytosine also impaired A-site tRNA binding. The deleterious effects of A1492G or A1493G (or both)
                 mutations were compensated by 2'fluorine substitutions in the mRNA codon. The results suggest that the ribosome recognizes the
                 codon-anticodon complex by adenine contacts to the messenger RNA backbone and provide a mechanism for molecular discrimination
                 of correct versus incorrect codon-anticodon pairs.
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>
            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Recht, M. I., Douthwaite, S., Puglisi, J. D.</span>
            <a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/1999-EMBO-Recht-Basis.pdf" download><span id="title">Basis for prokaryotic specificity of action of aminoglycoside antibiotics </span></a>
            <span id="publisher">EMBO JOURNAL, 1999; 18 (11): 3133-3138</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/1999-EMBO-Recht-Basis.pdf" download><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span></a>
                The aminoglycosides, a group of structurally related antibiotics, bind to rRNA in the small subunit of the prokaryotic ribosome. Most aminoglycosides are inactive or weakly active against eukaryotic ribosomes. A major difference in the binding site for these antibiotics between prokaryotic and eukaryotic ribosomes is the identity of the nucleotide at position 1408 (Escherichia coli numbering), which is an adenosine in prokaryotic ribosomes and a guanosine in eukaryotic ribosomes. Expression in E.coli of plasmid-encoded 16S rRNA containing an A1408 to G substitution confers resistance to a subclass of the aminoglycoside antibiotics that contain a 6' amino group on ring I. Chemical footprinting experiments indicate that resistance arises from the lower affinity of the drug for the eukaryotic rRNA sequence. The 1408G ribosomes are resistant to the same subclass of aminoglycosides as previously observed both for eukaryotic ribosomes and bacterial ribosomes containing a methylation at the N1 position of A1408. The results indicate that the identity of the nucleotide at position 1408 is a major determinant of specificity of aminoglycoside action, and agree with prior structural studies of aminoglycoside-rRNA complexes.
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>
            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Recht, M. I., Douthwaite, S., Dahlquist, K. D., Puglisi, J. D.</span>
            <a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/1999-jmb-Recht-effect.pdf" download><span id="title">Effect of mutations in the A site of 16S rRNA on aminoglycoside antibiotic-ribosome interaction</span></a>
            <span id="publisher">JOURNAL OF MOLECULAR BIOLOGY, 1999; 286 (1): 33-43</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/1999-jmb-Recht-effect.pdf" download><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span></a>
              Decoding of genetic information occurs upon interaction of an mRNA codon-tRNA anticodon complex with the small subunit of the ribosome. The ribosomal decoding region is associated with highly conserved sequences near the 3' end of 16 S rRNA. The decoding process is perturbed by the aminoglycoside antibiotics, which also interact with this region of rRNA. Mutations of certain nucleotides in rRNA reduce aminoglycoside binding affinity, as previously demonstrated using a model RNA oligonucleotide system. Here, predictions from the oligonucleotide system were tested in the ribosome by mutation of universally conserved nucleotides at 1406 to 1408 and 1494 to 1495 in the decoding region of plasmid-encoded bacterial 16 S rRNA. Phenotypic changes range from the benign effect of U1406-->A or A1408-->G substitutions, to the highly deleterious 1406G and 1495 mutations that assemble into 30 S subunits but are defective in forming functional ribosomes. Changes in the local conformation of the decoding region caused by these mutations were identified by chemical probing of isolated 30 S subunits. Ribosomes containing 16 S rRNA with mutations at positions 1408, 1407+1494, or 1495 had reduced affinity for the aminoglycoside paromomycin, whereas no discernible reduction in affinity was observed with 1406 mutant ribosomes. These data are consistent with prior NMR structural determination of aminoglycoside interaction with the decoding region, and further our understanding of how aminoglycoside resistance can be conferred.
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>
            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Puglisi JD, Williamson JR</span>
            <a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/1999-TheRNAWorld-Puglisi-RNAInteractions.pdf" download><span id="title">RNA Interaction with Small Ligands and Peptides</span></a>
            <span id="publisher">The RNA World, 2nd Ed.: The Nature of Modern RNA Suggests a Prebiotic RNA World, 1999; 37: 403-425</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/1999-TheRNAWorld-Puglisi-RNAInteractions.pdf" download><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span></a>

              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>
            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Puglisi JD, Blanchard SC, Dahlquist KD, Eason RG, Fourmy D, Lynch SR, Recht MI, Yoshizawa S</span>
            <a href="" download><span id="title">Structural basis for aminoglycoside antibiotic action</span></a>
            <span id="publisher">Ribosomes, 1999; Ch 34: 419-29</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span>

              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>
            </div>
          </div>


        <div class="year" id="1998">1998</div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Puglisi, E. V., Puglisi, J. D.</span>
            <a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/1998-NatureAmerica-EPuglisi-HIV.pdf" download><span id="title">HIV-1 A-rich RNA loop mimics the tRNA anticodon structure</span></a>
            <span id="publisher">NATURE STRUCTURAL BIOLOGY, 1998; 5 (12): 1033-1036</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/1998-NatureAmerica-EPuglisi-HIV.pdf" download><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span></a>
                Interaction of HIV-1 genomic RNA and human tRNA(Lys)3 initiates viral reverse transcription. An adenosine-rich (A-rich) loop in HIV RNA mediates complex formation between tRNA and viral RNA. We have determined the structure of an A-rich loop oligonucleotide using nuclear magnetic resonance spectroscopy. The loop structure is stabilized by a noncanonical G-A pair and a U-turn motif, which leads to stacking of the conserved adenosines. The structure has similarity to the tRNA anticodon structure, and suggests possible mechanisms for its role in initiation of reverse transcription.
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>
            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Yoshizawa, S., Fourmy, D., Puglisi, J. D.</span>
            <a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/1998_EMBO_Yoshizawa.pdf" download><span id="title">Structural origins of gentamicin antibiotic action</span></a>
            <span id="publisher">EMBO JOURNAL, 1998; 17 (22): 6437-6448</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/1998_EMBO_Yoshizawa.pdf" download><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span></a>
                Aminoglycoside antibiotics that bind to the ribosomal A site cause misreading of the genetic code and inhibit translocation. The clinically important aminoglycoside, gentamicin C, is a mixture of three components. Binding of each gentamicin component to the ribosome and to a model RNA oligonucleotide was studied biochemically and the structure of the RNA complexed to gentamicin C1a was solved using magnetic resonance nuclear spectroscopy. Gentamicin C1a binds in the major groove of the RNA. Rings I and II of gentamicin direct specific RNA-drug interactions. Ring III of gentamicin, which distinguishes this subclass of aminoglycosides, also directs specific RNA interactions with conserved base pairs. The structure leads to a general model for specific ribosome recognition by aminoglycoside antibiotics and a possible mechanism for translational inhibition and miscoding. This study provides a structural rationale for chemical synthesis of novel aminoglycosides.
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>
            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Blanchard, S. C., Fourmy, D., Eason, R. G., Puglisi, J. D.</span>
            <a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/1998-Biochemistry-blanchard-rRNA.pdf" download><span id="title">rRNA chemical groups required for aminoglycoside binding</span></a>
            <span id="publisher">BIOCHEMISTRY, 1998; 37 (21): 7716-7724</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/1998-Biochemistry-blanchard-rRNA.pdf" download><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span></a>
                Through an affinity chromatography based modification-interference assay, we have identified chemical groups within
                Escherichia coli 16S ribosomal RNA sequence that are required for binding the aminoglycoside antibiotic paromomycin.
                Paromomycin was covalently linked to solid support via a nine atom spacer from the 6"'-amine of ring IV, and chemical
                modifications to an A-site oligonucleotide that disrupted binding were identified. Positions in the RNA oligonucleotide
                that correspond to G1405(N7), G1491(N7), G1494(N7), A1408(N7), A1493(N7), A1408(N1), A1492(N1), and A1493(N1), as well
                as the pro-R phosphate oxygens of A1492 and A1493 in 16S rRNA are chemical groups that are essential for a high-affinity
                RNA-paromomycin interaction. These data are consistent with genetic, biochemical, and structural studies related to
                neomycin-class antibiotics and provide additional information for establishing an exact model for their interaction with the ribosome.
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>
            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Fourmy, D., Recht, M. I., Puglisi, J. D.</span>
            <a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/1998-JMB-Fourmy-BindofNeomycin.pdf" download><span id="title">Binding of neomycin-class aminoglycoside antibiotics to the A-site of 16 S rRNA</span></a>
            <span id="publisher">JOURNAL OF MOLECULAR BIOLOGY, 1998; 277 (2): 347-362</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/1998-JMB-Fourmy-BindofNeomycin.pdf" download><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span></a>
                Aminoglycoside antibiotics that bind to ribosomal RNA in the aminoacyl-tRNA site (A-site) cause misreading of the genetic code and inhibit translocation. We have recently solved the structure of an A-site RNA-paromomycin complex. The structure suggested that rings I and II, common to all aminoglycosides that bind to the A-site, are the minimum motif for specific ribosome binding to affect translation. This hypothesis was tested biochemically and with a detailed comparative NMR study of interaction of the aminoglycosides paromomycin, neomycin, ribostamycin, and neamine with the A-site RNA. Our NMR data show that rings I and II of neomycin-class aminoglycosides are sufficient to confer specificity to the binding of the antibiotics to the model A-site RNA. Neomycin, paromomycin, ribostamycin and neamine bind in the major groove of the A-site RNA in a unique binding pocket formed by non-canonical base pairs and a bulged nucleotide. Similar NMR properties of the RNA and the diverse antibiotics within the different complexes formed with neomycin, paromomycin, ribostamycin and neamine suggest similar structures for these complexes.
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>
            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Fourmy, D., Yoshizawa, S., Puglisi, J. D.</span>
            <a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/1998-JMB-Fourmy-paramomycin.pdf" download><span id="title">Paromomycin binding induces a local conformational change in the A-site of 16 S rRNA</span></a>
            <span id="publisher">JOURNAL OF MOLECULAR BIOLOGY, 1998; 277 (2): 333-345 </span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/1998-JMB-Fourmy-paramomycin.pdf" download><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span></a>
                Aminoglycoside antibiotics that bind to ribosomal RNA in the aminoacyl-tRNA site (A-site) cause misreading of the genetic code and inhibit translocation. An A-site RNA oligonucleotide specifically binds to aminoglycoside antibiotics and the structure of the RNA-paromomycin complex was previously determined by nuclear magnetic resonance (NMR) spectroscopy. Here, the A-site RNA structure in its free form has been determined using heteronuclear NMR and compared to the structure of the paromomycin-RNA complex. As in the complex with paromomycin, the asymmetric internal loop is closed by a Watson-Crick base-pair (C1407.G1494) and by two non-canonical base-pairs (U1406.U1495, A1408.A1493). A1492 stacks below A1493 and is intercalated between the upper and lower stems. The comparison of the free and bound conformations of the RNA shows that two universally conserved residues of the A site of 16 S rRNA, A1492 and A1493, are displaced towards the minor groove of the RNA helix in presence of antibiotic. These changes in the RNA conformation place the N1 positions of A1492 and A1493 on the minor groove side of the A-site RNA and suggest a mechanism of action of aminoglycosides on translation.
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>
            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Puglisi EV, and Puglisi JD</span>
            <a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/1998-CSHLPress-Puglisi-Nuclear.pdf" download><span id="title">Nuclear magnetic resonance spectroscopy of RNA</span></a>
            <span id="publisher">CSHL Press (In RNA Structure and Function, (Symons, R. W., and Grunberg-Manago, M., eds), 1998: 117-46</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/1998-CSHLPress-Puglisi-Nuclear.pdf" download><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span></a>

              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>
            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Yoshizawa, S., Puglisi, J. D.</span>
            <a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/1998-Nato-Yoshizawa.pdf" download><span id="title">NMR structure determination of an antibiotic-RNA complex</span></a>
            <span id="publisher">NATO Advanced Study Institute and International School of Structural Biology and Magnetic Resonance, 3rd Course on Protein Dynamics. Function, and Design, PLENUM PRESS DIV PLENUM PUBLISHING CORP. 1998: 173–182</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/1998-Nato-Yoshizawa.pdf" download><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span></a>

              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>
            </div>
          </div>

        <div class="year" id="1997">1997</div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Puglisi, E. V., Green, R., Noller, H. F., Puglisi, J. D.</span>
            <a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/1997-NatureStructuralBiology-Puglisi.pdf" download><span id="title">Structure of a conserved RNA component of the peptidyl transferase centre</span></a>
            <span id="publisher">NATURE STRUCTURAL BIOLOGY, 1997; 4 (10): 775-778</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/1997-NatureStructuralBiology-Puglisi.pdf" download><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span></a>
                The structure of a conserved hairpin loop involved in peptidyl-tRNA recognition by 50S ribosomal subunits has been solved by NMR. The loop is closed by a novel G-C base pair and presents guanine residues for RNA recognition.
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>
            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Puglisi JD</span>
            <a href="" download><span id="title">Structural basis for aminoglycoside antibiotic action</span></a>
            <span id="publisher">Many Faces of RNA, 1997: 97-111</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span>

              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>
            </div>
          </div>

        <div class="year" id="1996">1996</div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Fourmy, D., Recht, M. I., Blanchard, S. C., Puglisi, J. D.</span>
            <a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/Fourmyetal_science_1996.pdf" download><span id="title">Structure of the A site of Escherichia coli 16S ribosomal RNA complexed with an aminoglycoside antibiotic </span></a>
            <span id="publisher">SCIENCE, 1996; 274 (5291): 1367-1371</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/Fourmyetal_science_1996.pdf" download><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span></a>
                Aminoglycoside antibiotics that bind to 30S ribosomal A-site RNA cause misreading of the genetic code and inhibit translocation. The aminoglycoside antibiotic paromomycin binds specifically to an RNA oligonucleotide that contains the 30S subunit A site, and the solution structure of the RNA-paromomycin complex was determined by nuclear magnetic resonance spectroscopy. The antibiotic binds in the major groove of the model A-site RNA within a pocket created by an A-A base pair and a single bulged adenine. Specific interactions occur between aminoglycoside chemical groups important for antibiotic activity and conserved nucleotides in the RNA. The structure explains binding of diverse aminoglycosides to the ribosome, their specific activity against prokaryotic organisms, and various resistance mechanisms, and provides insight into ribosome function.
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>
            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Recht, M. I., Fourmy, D., Blanchard, S. C., Dahlquist, K. D., Puglisi, J. D.</span>
            <a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/1996-JMB-Recht-RNASequence.pdf" download><span id="title">RNA sequence determinants for aminoglycoside binding to an A-site rRNA model oligonucleotide</span></a>
            <span id="publisher">JOURNAL OF MOLECULAR BIOLOGY, 1996; 262 (4): 421-436</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract">  <a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/1996-JMB-Recht-RNASequence.pdf" download><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span></a>
                The codon-anticodon interaction on the ribosome occurs in the A site of the 30 S subunit. Aminoglycoside antibiotics, which bind to ribosomal RNA in the A site, cause misreading of the genetic code and inhibit translocation. Biochemical studies and nuclear magnetic resonance spectroscopy were used to characterize the interaction between the aminoglycoside antibiotic paromomycin and a small model oligonucleotide that mimics the A site of Escherichia coli 16 S ribosomal RNA. Upon chemical modification, the RNA oligonucleotide exhibits an accessibility pattern similar to that of 16 S rRNA in the 30 S subunit. In addition, the oligonucleotide binds specifically aminoglycoside antibiotics. The antibiotic binding site forms an asymmetric internal loop, caused by non-canonical base-pairs. Nucleotides that are important for binding of paromomycin were identified by performing quantitative footprinting on oligonucleotide sequence variants and include the C1407.G1494 base-pair, and A.U base-pair at positions 1410/1490, and nucleotides A1408, A1493 and U1495. The asymmetry of the internal loop, which requires the presence of a nucleotide in position 1492, is also crucial for antibiotic binding. Introduction into the oligonucleotide of base changes that are known to confer aminoglycoside resistance in 16 S rRNA result in weaker binding of paromomycin to the oligonucleotide. Oligonucleotides homologous to eukaryotic rRNA sequences show reduced binding of paromomycin, suggesting a physical origin for the species-specific action of aminoglycosides.
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>
            </div>
          </div>

        <div class="year" id="1995">1995</div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Puglisi, J. D., Chen, L., Blanchard, S., Frankel, A. D.</span>
            <a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/1996-Science-Puglisi-Solution.pdf" download><span id="title">SOLUTION STRUCTURE OF A BOVINE IMMUNODEFICIENCY VIRUS TAT-TAR PEPTIDE-RNA COMPLEX</span></a>
            <span id="publisher">SCIENCE, 1995; 270 (5239): 1200-1203</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/1996-Science-Puglisi-Solution.pdf" download><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span></a>
                The Tat protein of bovine immunodeficiency virus (BIV) binds to its target RNA, TAR, and activates transcription. A 14-amino acid arginine-rich peptide corresponding to the RNA-binding domain of BIV Tat binds specifically to BIV TAR, and biochemical and in vivo experiments have identified the amino acids and nucleotides required for binding. The solution structure of the RNA-peptide complex has now been determined by nuclear magnetic resonance spectroscopy. TAR forms a virtually continuous A-form helix with two unstacked bulged nucleotides. The peptide adopts a beta-turn conformation and sits in the major groove of the RNA. Specific contacts are apparent between critical amino acids in the peptide and bases and phosphates in the RNA. The structure is consistent with all biochemical data and demonstrates ways in which proteins can recognize the major groove of RNA.
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>
            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Puglisi, J. D., Wyatt, J. R.</span>
            <a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/1995-NuclearMagResonance-Puglisi.pdf" download><span id="title">Biochemical and NMR studies of RNA conformation with an emphasis on RNA pseudoknots</span></a>
            <span id="publisher">NUCLEAR MAGNETIC RESONANCE AND NUCLEIC ACIDS, 1995; 261: 323-350</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/1995-NuclearMagResonance-Puglisi.pdf" download><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span></a>

              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>
            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Puglisi JD</span>
            <a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/1995-Chemisty%26Biology-Puglisi-Welcome.pdf" download><span id="title">Welcome guests to the RNA World: Proteins that interact with RNA</span></a>
            <span id="publisher">Chemistry and Biology, 1995; 2: 581</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/1995-Chemisty%26Biology-Puglisi-Welcome.pdf" download><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span></a>

              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>
            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Dahlquist, K., Puglisi, J. D.</span>
            <a href="" download><span id="title">Investigating the structure and function of translation initiation factor 1 in Escherichia coli.</span></a>
            <span id="publisher">Nucleic acids symposium series, 1995: 170-171</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span>

              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>
            </div>
          </div>

        <div class="year" id="1994">1994</div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Puglisi, E. V., Puglisi, J. D., Williamson, J. R., RajBhandary, U. L.</span>
            <a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/1994-PNAS-Puglisi-11467-71.pdf" download><span id="title">NMR ANALYSIS OF TRANSFER-RNA ACCEPTOR STEM MICROHELICES - DISCRIMINATOR BASE CHANGE AFFECTS TRANSFER-RNA CONFORMATION AT THE 3' END</span></a>
            <span id="publisher">PROCEEDINGS OF THE NATIONAL ACADEMY OF SCIENCES OF THE UNITED STATES OF AMERICA, 1994; 91 (24): 11467-11471</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/1994-PNAS-Puglisi-11467-71.pdf" download><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span></a>
              An important step in initiation of protein synthesis in Escherichia coli is the specific formylation of the initiator methionyl-tRNA (Met-tRNA) by Met-tRNA transformylase. The determinants for formylation are clustered mostly in the acceptor stem of the initiator tRNA. Here we use NMR spectroscopy to characterize the conformation of two RNA microhelices, which correspond to the acceptor stem of mutants of E. coli initiator tRNA and which differ only at the position corresponding to the "discriminator base" in tRNAs. One of the mutant tRNAs is an extremely poor substrate for Met-tRNA transformylase, whereas the other one is a much better substrate. We show that one microhelix forms a structure in which its 3'-ACCA sequence extends the stacking of the acceptor stem. The other microhelix forms a structure in which its 3'-UCCA sequence folds back such that the 3'-terminal A22 is in close proximity to G1. These results highlight the importance of the discriminator base in determining tRNA conformation at the 3' end. They also suggest a correlation between tRNA structure at the 3' end and its recognition by Met-tRNA transformylase.
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>
            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Puglisi, J. D., Williamson, J. R.</span>
            <a href="" download><span id="title">NMR-STUDIES OF HIV TAR RNA </span></a>
            <span id="publisher">8th Conversation in the Discipline Biomolecular Stereodynamics, ADENINE PRESS. 1994: 285–291</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span>
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>
            </div>
          </div>

        <div class="year" id="1993">1993</div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Putz, J., Puglisi, J. D., Florentz, C., Giege, R.</span>
            <a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/1993-EMBO-Putz.pdf" download><span id="title">ADDITIVE, COOPERATIVE AND ANTI-COOPERATIVE EFFECTS BETWEEN IDENTITY NUCLEOTIDES OF A TRANSFER-RNA</span></a>
            <span id="publisher">EMBO JOURNAL, 1993; 12 (7): 2949-2957</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/1993-EMBO-Putz.pdf" download><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span></a>
                We have investigated the functional relationship between nucleotides in yeast tRNAAsp that are important for aspartylation by yeast aspartyl-tRNA synthetase. Transcripts of tRNAAsp with two or more mutations at identity positions G73, G34, U35, C36 and base pair G10-U25 have been prepared and the steady-state kinetics of their aspartylation were measured. Multiple mutations affect the catalytic activities of the synthetase mainly at the level of the catalytic constant, kcat. Kinetic data were expressed as free energy variation at transition state of these multiple mutants and comparison of experimental values with those calculated from results on single mutants defined three types of relationships between the identity nucleotides of this tRNA. Nucleotides located far apart in the three-dimensional structure of the tRNA act cooperatively whereas nucleotides of the anticodon triplet act either additively or anti-cooperatively. These results are related to the specific interactions of functional groups on identity nucleotides with amino acids in the protein as revealed by the crystal structure of the tRNAAsp/aspartyl-tRNA synthetase complex. These relationships between identity nucleotides may play an important role in the biological function of tRNAs.
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>
            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Puglisi, J. D., Chen, L., Frankel, A. D., Williamson, J. R.</span>
            <a href="" download><span id="title">ROLE OF RNA STRUCTURE IN ARGININE RECOGNITION OF TAR RNA</span></a>
            <span id="publisher">PROCEEDINGS OF THE NATIONAL ACADEMY OF SCIENCES OF THE UNITED STATES OF AMERICA, 1993; 90 (8): 3680-3684</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span>
                The human immunodeficiency virus Tat protein binds specifically to an RNA stem-loop structure (TAR) that contains two helical stem regions separated by a three-nucleotide bulge. A single arginine within the basic region of Tat mediates specific binding to TAR, and arginine as the free amino acid also binds specifically to TAR. We have previously proposed a model in which interaction of the arginine guanidinium group with guanosine-26 (G26) and with a pair of phosphates is stabilized by formation of a base triple between U23 in the bulge and A27.U38 in the upper helix. Here we show by NMR spectroscopy that formation of the base triple is critical for arginine binding to TAR. Mutants of TAR that cannot form the base triple or that remove the guanine contact do not bind arginine specifically. These mutants also showed reduced transactivation by Tat. A triple mutant designed to form an isomorphous base triple between C23 and G27.C38 binds arginine and adopts the same conformation as wild-type TAR. These results demonstrate the importance of RNA structure for arginine binding and further demonstrate the direct correspondence between arginine and Tat binding.
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>
            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Puglisi, J. D., Putz, J., Florentz, C., Giege, R.</span>
            <a href="" download><span id="title">INFLUENCE OF TRANSFER-RNA TERTIARY STRUCTURE AND STABILITY ON AMINOACYLATION BY YEAST ASPARTYL-TRANSFER RNA-SYNTHETASE</span></a>
            <span id="publisher">NUCLEIC ACIDS RESEARCH, 1993; 21 (1): 41-49</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span>
                Mutations have been designed that disrupt the tertiary structure of yeast tRNA(Asp). The effects of these mutations on both tRNA structure and specific aspartylation by yeast aspartyl-tRNA synthetase were assayed. Mutations that disrupt tertiary interactions involving the D-stem or D-loop result in destabilization of the base-pairing in the D-stem, as monitored by nuclease digestion and chemical modification studies. These mutations also decrease the specificity constant (kcat/Km) for aspartylation by aspartyl-tRNA synthetase up to 10(3)-10(4) fold. The size of the T-loop also influences tRNA(Asp) structure and function; change of its T-loop to a tetraloop (-UUCG-) sequence results in a denatured D-stem and an almost 10(4) fold decrease of kcat/Km for aspartylation. The negative effects of these mutations on aspartylation activity are significantly alleviated by additional mutations that stabilize the D-stem. These results indicate that a critical role of tertiary structure in tRNA(Asp) for aspartylation is the maintenance of a base-paired D-stem.
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>
            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Giege, R., Puglisi, J. D., Florentz, C.</span>
            <a href="" download><span id="title">TRANSFER-RNA STRUCTURE AND AMINOACYLATION EFFICIENCY</span></a>
            <span id="publisher">PROGRESS IN NUCLEIC ACID RESEARCH AND MOLECULAR BIOLOGY, VOL 45, 1993; 45: 129-206</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span>

              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>
            </div>
          </div>

        <div class="year" id="1992">1992</div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Batey, R. T., Inada, M., Kujawinski, E., Puglisi, J. D., Williamson, J. R.</span>
            <a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/1992-NucleicAcidResearch-Batey.pdf" download><span id="title">PREPARATION OF ISOTOPICALLY LABELED RIBONUCLEOTIDES FOR MULTIDIMENSIONAL NMR-SPECTROSCOPY OF RNA</span></a>
            <span id="publisher">NUCLEIC ACIDS RESEARCH, 1992; 20 (17): 4515-4523</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/1992-NucleicAcidResearch-Batey.pdf" download><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span></a>
                A general method for large scale preparation of uniformly isotopically labeled ribonucleotides and RNAs is described. Bacteria are grown on isotopic growth medium, and their nucleic acids are harvested and degraded to mononucleotides. These are enzymatically converted into ribonucleoside triphosphates, which are used in transcription reactions in vitro to prepare RNAs for NMR studies. For 15N-labeling, E.coli is grown on 15N-ammonium sulfate, whereas for 13C-labeling, Methylophilus methylotrophus is grown on 13C-methanol, which is more economical than 13C-glucose. To demonstrate the feasibility and utility of this method, uniformly 13C-labeled ribonucleotides were used to synthesize a 31 nucleotide HIV TAR RNA that was analyzed by 3D-NMR. This method should find widespread use in the structural analysis of RNA by NMR.
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>
            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Perret, V., Florentz, C., Puglisi, J. D., Giege, R.</span>
            <a href="" download><span id="title">EFFECT OF CONFORMATIONAL FEATURES ON THE AMINOACYLATION OF TRANSFER-RNAS AND CONSEQUENCES ON THE PERMUTATION OF TRANSFER-RNA SPECIFICITIES</span></a>
            <span id="publisher">JOURNAL OF MOLECULAR BIOLOGY, 1992; 226 (2): 323-333</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span>
              The structure and function of in vitro transcribed tRNA(Asp) variants with inserted conformational features characteristic of yeast tRNA(Phe), such as the length of the variable region or the arrangement of the conserved residues in the D-loop, have been investigated. Although they exhibit significant conformational alterations as revealed by Pb2+ treatment, these variants are still efficiently aspartylated by yeast aspartyl-tRNA synthetase. Thus, this synthetase can accommodate a variety of tRNA conformers. In a second series of variants, the identity determinants of yeast tRNA(Phe) were transplanted into the previous structural variants of tRNA(Asp). The phenylalanine acceptance of these variants improves with increasing the number of structural characteristics of tRNA(Phe), suggesting that phenylalanyl-tRNA synthetase is sensitive to the conformational frame embedding the cognate identity nucleotides. These results contrast with the efficient transplantation of tRNA(Asp) identity elements into yeast tRNA(Phe). This indicates that synthetases respond differently to the detailed conformation of their tRNA substrates. Efficient aminoacylation is not only dependent on the presence of the set of identity nucleotides, but also on a precise conformation of the tRNA.
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>
            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Puglisi, J. D., Tan, R. Y., CALNAN, B. J., Frankel, A. D., Williamson, J. R.</span>
            <a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/1992-science-puglisi-conformation.pdf" download><span id="title">CONFORMATION OF THE TAR RNA-ARGININE COMPLEX BY NMR-SPECTROSCOPY</span></a>
            <span id="publisher">SCIENCE, 1992; 257 (5066): 76-80</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/1992-science-puglisi-conformation.pdf" download><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span></a>
              The messenger RNAs of human immunodeficiency virus-1 (HIV-1) have an RNA hairpin structure, TAR, at their 5' ends that contains a six-nucleotide loop and a three-nucleotide bulge. The conformations of TAR RNA and of TAR with an arginine analog specifically bound at the binding site for the viral protein, Tat, were characterized by nuclear magnetic resonance (NMR) spectroscopy. Upon arginine binding, the bulge changes conformation, and essential nucleotides for binding, U23 and A27.U38, form a base-triple interaction that stabilizes arginine hydrogen bonding to G26 and phosphates. Specificity in the arginine-TAR interaction appears to be derived largely from the structure of the RNA.
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>
            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Rudinger, J., Puglisi, J. D., Putz, J., Schatz, D., Eckstein, F., Florentz, C., Giege, R.</span>
            <a href="" download><span id="title">DETERMINANT NUCLEOTIDES OF YEAST TRANSFER RNA(ASP) INTERACT DIRECTLY WITH ASPARTYL-TRANSFER RNA-SYNTHETASE</span></a>
            <span id="publisher">PROCEEDINGS OF THE NATIONAL ACADEMY OF SCIENCES OF THE UNITED STATES OF AMERICA, 1992; 89 (13): 5882-5886</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span>
The interaction of wild-type and mutant yeast tRNA(Asp) transcripts with yeast aspartyl-tRNA synthetase (AspRS; EC 6.1.1.12) has been probed by using iodine cleavage of phosphorothioate-substituted transcripts. AspRS protects phosphates in the anticodon (G34, U35), D-stem (U25), and acceptor end (G73) that correspond to determinant nucleotides for aspartylation. This protection, as well as that in anticodon stem (C29, U40, G41) and D-stem (U11 to U13), is consistent with direct interaction of AspRS at these phosphates. Other protection, in the variable loop (G45), D-loop (G18, G19), and T-stem and loop (G53, U54, U55), as well as enhanced reactivity at G37, may result from conformational changes of the transcript upon binding to AspRS. Transcripts mutated at determinant positions showed a loss of phosphate protection in the region of the mutation while maintaining the global protection pattern. The ensemble of results suggests that aspartylation specificity arises from both protein-base and protein-phosphate contacts and that different regions of tRNA(Asp) interact independently with AspRS. A mutant transcript of yeast tRNA(Phe) that contains the set of identity nucleotides for specific aspartylation gave a phosphate protection pattern strikingly similar to that of wild-type tRNA(Asp). This confirms that a small number of nucleotides within a different tRNA sequence context can direct specific interaction with synthetase.
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>
            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Puglisi JD, Tan R, Frankel AD, Williamson JR</span>
            <a href="" download><span id="title">Structure and Function of HIV TAR RNA</span></a>
            <span id="publisher">Advances in Life Sciences, (In Structural Tools for the Analysis of Protein-Nucleic Acid Complexes, D Lilley, H Heumann, D Suck, ed.)., 1992: 269-85</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span>

              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>
            </div>
          </div>

        <div class="year" id="1991">1991</div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Wyatt, J. R., Chastain, M., Puglisi, J. D.</span>
            <a href="" download><span id="title">SYNTHESIS AND PURIFICATION OF LARGE AMOUNTS OF RNA OLIGONUCLEOTIDES</span></a>
            <span id="publisher">BIOTECHNIQUES, 1991; 11 (6): 764-769</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span>
            Biophysical studies of RNA oligonucleotides require milligram amounts of RNA of specific length and sequence. Transcription from synthetic DNA templates using T7 RNA polymerase is a convenient method for synthesis of RNA oligonucleotides ranging in size from 9 to about 45 nucleotides. Here we present methods that make the large-scale synthesis of RNA oligonucleotides practical. This paper describes a rapid method for isolating T7 RNA polymerase free from RNases for use in transcription reactions. Protocols are also described for purification of the desired RNA oligonucleotide from the other products of transcription.
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>
            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Putz, J., Puglisi, J. D., Florentz, C., Giege, R.</span>
            <a href="" download><span id="title">IDENTITY ELEMENTS FOR SPECIFIC AMINOACYLATION OF YEAST TRANSFER RNAASP BY COGNATE ASPARTYL-TRANSFER RNA-SYNTHETASE </span></a>
            <span id="publisher">SCIENCE, 1991; 252 (5013): 1696-1699</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span>
            The nucleotides crucial for the specific aminoacylation of yeast tRNA(Asp) by its cognate synthetase have been identified. Steady-state aminoacylation kinetics of unmodified tRNA transcripts indicate that G34, U35, C36, and G73 are important determinants of tRNA(Asp) identity. Mutations at these positions result in a large decrease (19- to 530-fold) of the kinetic specificity constant (ratio of the catalytic rate constant kcat and the Michaelis constant Km) for aspartylation relative to wild-type tRNA(Asp). Mutation to G10-C25 within the D-stem reduced kcat/Km eightfold. This fifth mutation probably indirectly affects the presentation of the highly conserved G10 nucleotide to the synthetase. A yeast tRNA(Phe) was converted into an efficient substrate for aspartyl-tRNA synthetase through introduction of the five identity elements. The identity nucleotides are located in regions of tight interaction between tRNA and synthetase as shown in the crystal structure of the complex and suggest sites of base-specific contacts.
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>
            </div>
          </div>


          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Puglisi, J. D., Wyatt, J. R., Tinoco, I.</span>
            <a href="" download><span id="title">RNA PSEUDOKNOTS</span></a>
            <span id="publisher">ACCOUNTS OF CHEMICAL RESEARCH, 1991; 24 (5): 152-158</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span>

              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>
            </div>
          </div>


        <div class="year" id="1990">1990</div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Perret, V., Garcia, A., Puglisi, J., Grosjean, H., Ebel, J. P., Florentz, C., Giege, R.</span>
            <a href="" download><span id="title">CONFORMATION IN SOLUTION OF YEAST TRANSFER RNAASP TRANSCRIPTS DEPRIVED OF MODIFIED NUCLEOTIDES</span></a>
            <span id="publisher">BIOCHIMIE, 1990; 72 (10): 735-744</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span>
          A synthetic gene of yeast aspartic acid tRNA with a promoter for phage T7 RNA polymerase was cloned in Escherichia coli. The in vitro transcribed tRNA(Asp) molecules are deprived of modified nucleotides and retain their aspartylation capacity. The solution conformation of these molecules was mapped with chemical structural probes and compared to that of fully modified molecules. Significant differences in reactivities were observed in Pb2+ cleavage of the RNAs and in modification of the bases with dimethyl sulphate. The most striking result concerns C56, which becomes reactive in unmodified tRNA(Asp), indicating the disruption of the C56-G19 base pair involved in the D- and T-loop interaction. The chemical data indicate that unmodified tRNA(Asp) transcripts possess a relaxed conformation compared to that of the native tRNA. This conclusion is confirmed by thermal melting experiments. Thus it can be proposed that post-transcriptional modifications of nucleotides in tRNA stabilize the biologically active conformations in these molecules.
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>
            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Puglisi, J. D., Wyatt, J. R., Tinoco, I.</span>
            <a href="" download><span id="title">CONFORMATION OF AN RNA PSEUDOKNOT</span></a>
            <span id="publisher">JOURNAL OF MOLECULAR BIOLOGY, 1990; 214 (2): 437-453</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span>
                The structure of the 5' GCGAUUUCUGACCGCUUUUUUGUCAG 3' RNA oligonucleotide was investigated using biochemical and chemical probes and nuclear magnetic resonance spectroscopy. Formation of a pseudoknot is indicated by the imino proton spectrum. Imino protons are observed consistent with formation of two helical stem regions; nuclear Overhauser enhancements between imino protons show that the two stem regions stack to form a continuous helix. In the stem regions, nucleotide conformations (3'-endo, anti) and internucleotide distances, derived from two-dimensional correlated, spectroscopy and two-dimensional nuclear Overhauser effect spectra, are characteristic of A-form geometry. The data suggest minor distortion in helical stacking at the junctions of stems and loops. The model of the pseudoknot is consistent with the structure originally proposed by Pleij et al.
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>
            </div>
          </div>


          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Wyatt, J. R., Puglisi, J. D., Tinoco, I.</span>
            <a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/1990-JMB-Wyatt-RNAPseudoknots.pdf" download><span id="title">RNA PSEUDOKNOTS - STABILITY AND LOOP SIZE REQUIREMENTS</span></a>
            <span id="publisher">JOURNAL OF MOLECULAR BIOLOGY, 1990; 214 (2): 455-470</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/1990-JMB-Wyatt-RNAPseudoknots.pdf" download><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span></a>
              The effects of ionic conditions, loop size and loop sequence on the formation of pseudoknots by RNA oligonucleotides have been investigated using biochemical and biophysical methods. An oligonucleotide with the sequence 5' GCGAUUUCUGACCGCUUUUUUGUCAG 3' and oligonucleotides with variations in the sequences of the two loop regions, denoted by bold face type, were studied. Each sequence with the potential to form a pseudoknot can also form two stable hairpins. The pseudoknot structure is stabilized relative to the hairpins by addition of Mg2+. Even in the presence of Mg2+, the pseudoknots formed by the sequences investigated are only marginally more stable (1.5 to 2 kcal mol-1 in free energy at 37 degrees C) than either of the constituent hairpins. The pseudoknot structure is the stable conformation in the presence of Mg2+ when the first loop region is at least three nucleotides and the second is at least four nucleotides. Further deletion of nucleotides from the loop regions stabilizes possible hairpin structures relative to the pseudoknot and equilibria among secondary and tertiary structures result.
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>
            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Giege, R., Florentz, C., Garcia, A., Grosjean, H., Perret, V., Puglisi, J., THEOBALDDIETRICH, A., Ebel, J. P.</span>
            <a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/1990-BIOCHIMIE-Giege.pdf" download><span id="title">EXPLORING THE AMINOACYLATION FUNCTION OF TRANSFER-RNA BY MACROMOLECULAR ENGINEERING APPROACHES - INVOLVEMENT OF CONFORMATIONAL FEATURES IN THE CHARGING PROCESS OF YEAST TRANSFER RNAASP</span></a>
            <span id="publisher">BIOCHIMIE, 1990; 72 (6-7): 453-461</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/1990-BIOCHIMIE-Giege.pdf" download><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span></a>
              This report presents the conceptual and methodological framework that presently underlies the experiments designed to decipher the structural features in tRNA important for its aminoacylation by aminoacyl-tRNA synthetases. It emphasizes the importance of conformational features in tRNA for an optimized aminoacylation. This is illustrated by selected examples on yeast tRNA(Asp). Using the phage T7 transcriptional system, a series of tRNA(Asp) variants were created in which conformational elements were modified. It is shown that aspartyl-tRNA synthetase tolerates conformational variability in tRNA(Asp) at the level of the D-loop and variable region, of the tertiary Levitt base-pair 15-48 which can be inverted and in the T-arm in which residue 49 can be excised. However, changing the anticodon region completely abolishes the aspartylation capacity of the variants. Transplanting the phenylalanine identity elements into a different tRNA(Asp) variant presenting conformational characteristics of tRNA(Phe) converts this molecule into a phenylalanine acceptor but is less efficient than wild-type tRNA(Phe). This engineered tRNA completely loses its aspartylation capacity, showing that some aspartic acid and phenylalanine identity determinants overlap. The fact that chimeric tRNA(Asp) molecules with altered anticodon regions lose their aspartylation capacity demonstrates that this region is part of the aspartic acid identity of tRNA(Asp).
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>
            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Puglisi, J. D., Wyatt, J. R., Tinoco, I.</span>
            <a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/1990-biochemistry-puglisi.pdf" download><span id="title">SOLUTION CONFORMATION OF AN RNA HAIRPIN LOOP</span></a>
            <span id="publisher">BIOCHEMISTRY, 1990; 29 (17): 4215-4226</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/1990-biochemistry-puglisi.pdf" download><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span></a>
                The hairpin conformation adopted by the RNA sequence 5'GCGAUUUCUGACCGCC3' has been studied by one- and two-dimensional NMR spectroscopy. Exchangeable imino spectra in 60 mM Na+ indicate that the hairpin has a stem of six base pairs (indicated by boldface type) and a loop of three nucleotides. NOESY spectra of nonexchangeable protons confirm the formation of the stem region. The duplex has an A-conformation and contains an A.C apposition; a G.U base pair closes the loop region. The stem nucleotides have C3'-endo sugar conformations, as expected of an A-form duplex, whereas the three loop nucleotides adopt C2'-endo sugar puckers. Stacking within the loop, C8 upon the sugar of U7, stabilizes the structure. The pH dependence of both the exchangeable and nonexchangeable NMR spectra is consistent with the formation of an A+.C base pair, protonated at the N1 position of adenine. The stability of the hairpin was probed by using absorbance melting curves. The hairpin structure with the A+.C base pair is about +2 kcal/mol less stable in free energy at 37 degrees C than the hairpin formed with an A.U pair replacing the A+.C pair.
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>
            </div>
          </div>

          <div class="year" id="1989">1989</div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Wyatt, J. R., Puglisi, J. D., Tinoco, I.</span>
            <a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/1989-Bioessays-Wyatt.pdf" download><span id="title">RNA FOLDING - PSEUDOKNOTS, LOOPS AND BULGES</span></a>
            <span id="publisher"> BIOESSAYS, 1989; 11 (4): 100-106</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/1989-Bioessays-Wyatt.pdf" download><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span></a>
                  The three-dimensional structures adopted by RNA molecules are crucial to their biological functions. The nucleotides
                  of an RNA molecule interact to form characteristic secondary-structure motifs. Tertiary interactions orient these secondary-structure elements with respect to each other to form the functional RNA. Here we describe the basic structural elements with special emphasis on a novel tertiary motif, the pseudoknot.
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>
            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Puglisi, J. D., Tinoco, I.</span>
            <a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/1989-MethodsOfEnzymol-Puglsi.pdf" download><span id="title">ABSORBENCY MELTING CURVES OF RNA</span></a>
            <span id="publisher">METHODS IN ENZYMOLOGY, 1989; 180: 304-325</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/1989-MethodsOfEnzymol-Puglsi.pdf" download><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span></a>

              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>
            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Tinoco I Jr, Aboul-ela F, Hardin CC, Puglisi JD, Varani G, Walker GT, Wolk S, Wyatt JR</span>
            <a href="" download><span id="title">Nucleic Acids from A to Z</span></a>
            <span id="publisher">Blackwell Scientific Publications, Oxford (In Frontiers of Macromolecular Science, (T Saegusa, T Higashimura and A Abe, eds.)., 1989: 519-24</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span>

              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>
            </div>
          </div>

          <div class="year" id="1988">1988</div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Puglisi, J. D., Wyatt, J. R., Tinoco, I.</span>
            <a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/1988-Nature-Puglisi-Pseudoknot.pdf" download><span id="title">A PSEUDOKNOTTED RNA OLIGONUCLEOTIDE </span></a>
            <span id="publisher">NATURE, 1988; 331 (6153): 283-286</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><a href="https://s3-us-west-2.amazonaws.com/puglisi-website-publications-bucket/1988-Nature-Puglisi-Pseudoknot.pdf" download><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span></a>
                The diverse functions of RNA, which include enzymatic activities, regulatory roles in transcription and translation, are made possible by tertiary structure. Computer algorithms can predict the secondary structure of an RNA molecule using free-energy parameters for base pairing and stacking, loops and bulges. However, with the exception of transfer RNA, little is known about the structures and thermodynamics of interactions involved in the tertiary structure of RNA. Recently, it has been proposed that a novel form of RNA folding called pseudoknotting occurs at the 3' end of certain viral RNAs from plants. A pseudoknot involves intramolecular pairing of bases in a hairpin loop with a few bases outside the stem of the loop to form an additional stem and loop region (Fig. 1). If each stem contained a full helical turn, a true knot would be formed. We present evidence from single-strand specific (S1) and double-strand specific (V1) nuclease digestion, that a short RNA oligonucleotide (19 nucleotides long) adopts a stable pseudoknotted structure. The nuclease digestion and thermodynamic properties of this oligonucleotide were compared with those of oligonucleotides which form hairpin structures containing the two possible stem regions in the pseudoknot. These results show that appropriate sequences can form pseudoknots and indicate that pseudoknots are a significant type of local tertiary structure which must be considered in the folding of complex RNA molecules.
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>
            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Wyatt JR, Puglisi JD, Tinoco I Jr</span>
            <a href="" download><span id="title">Pseudoknotted RNA Oligonucleotides</span></a>
            <span id="publisher">UCLA Symposia Series, Alan R Liss Inc, New York, NY (In Molecular Biology of RNA, T. Cech, ed.), 1988; 94: 25-32</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span>

              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>
            </div>
          </div>

          <div class="year" id="1987">1987</div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Trulson, M. O., Cruz, P., Puglisi, J. D., Tinoco, I., Mathies, R. A.</span>
            <a href="" download><span id="title">RAMAN-SPECTROSCOPIC STUDY OF LEFT-HANDED Z-RNA</span></a>
            <span id="publisher">BIOCHEMISTRY, 1987; 26 (26): 8624-8630</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span>
            The solvent conditions that induce the formation of a left-handed Z form of poly[r(G-C)] have been extended to include 6.5 M NaBr at 35 degrees C and 3.8 M MgCl2 at room temperature. The analysis of the A----Z transition in RNA by circular dichroism (CD), 1H and 31P NMR, and Raman spectroscopy shows that two distinct forms of left-handed RNA exist. The ZR-RNA structure forms in high concentrations of NaBr and NaClO4 and exhibits a unique CD signature. ZD-RNA is found in concentrated MgCl2 and has a CD signature similar to the Z form of poly[d(G-C)]. The loss of Raman intensity of the 813-cm-1 A-form marker band in both the A----ZR-RNA and A----ZD-RNA transitions parallels the loss of intensity at 835 cm-1 in the B----Z transition of DNA. A guanine vibration that is sensitive to the glycosyl torsion angle shifts from 671 cm-1 in A-RNA to 641 cm-1 in both ZD- and ZR-RNA, similar to the B----Z transition in DNA in which this band shifts from 682 to 625 cm-1. Significant differences in the glycosyl angle and sugar pucker between Z-DNA and Z-RNA are suggested by the 16-cm-1 difference in the position of this band. The Raman evidence for structural difference between ZD- and ZR-RNA comes from two groups of bands: First, Raman intensities between 1180 and 1600 cm-1 of ZD-RNA differ from those for ZR-RNA, corroborating the CD evidence for differences in base-stacking geometry. Second, the phosphodiester stretching bands near 815 cm-1 provide evidence of differences in backbone geometry between ZD- and ZR-RNA.
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>
            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Hardin, C. C., Zarling, D. A., Puglisi, J. D., Trulson, M. O., Davis, P. W., Tinoco, I.</span>
            <a href="" download><span id="title">STABILIZATION OF Z-RNA BY CHEMICAL BROMINATION AND ITS RECOGNITION BY ANTI-Z-DNA ANTIBODIES</span></a>
            <span id="publisher">BIOCHEMISTRY, 1987; 26 (16): 5191-5199</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span>
              Limited chemical bromination of poly[r(C-G)] (32% br8G, 26% br5C) results in partial modification of guanine C8 and cytosine C5, producing a mixture of A- and Z-RNA forms. The Z conformation in the brominated polynucleotide is stabilized at much lower ionic strength than in the unmodified polynucleotide. More extensive bromination of poly[r(C-G)] (greater than 49% br8G, 43% br5C) results in stabilization of a form of RNA having a Z-DNA-like (ZD) CD spectrum in low-salt, pH 7.0-7.5 buffers. Raising the ionic strength to 6 M NaBr or NaClO4 results in a transition in Br-poly[r(C-G)] to a Z-RNA (ZR) conformation as judged by CD spectroscopy. At lower ionic strength Z-DNA-like (ZD) and A-RNA conformations are also present. 1H NMR data demonstrate a 1/1 mixture of A- and Z-RNAs in 110 mM NaBr buffer at 37 degrees C. Nuclear Overhauser effect (NOE) experiments permit complete assignments of GH8, CH6, CH5, GH1', and CH1' resonances in both the A- and Z-forms. GH8----GH1' NOEs demonstrate the presence of both A- and Z-form GH8 resonances in slow exchange on the NMR time scale. The NMR results indicate that unbrominated guanine residues undergo transition to the syn conformation (Z-form). Raman scattering data are consistent with a mixture of A- and Z-RNAs in 110 mM NaCl buffer at 37 degrees C. Comparison with the spectrum of Z-DNA indicates that there may be different glycosidic torsion angles in Z-RNA and Z-DNA [Tinoco, I., Jr., Cruz, P., Davis, P., Hall, K., Hardin, C. C., Mathies, R. A., Puglisi, J. D., Trulson, M. O., Johnson, W. C., & Neilson, T. (1986) in Structure and Dynamics of RNA, pp 55-68, Plenum, New York].(ABSTRACT TRUNCATED AT 250 WORDS)
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>
            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Tinoco, I., Davis, P. W., Hardin, C. C., Puglisi, J. D., Walker, G. T., Wyatt, J.</span>
            <a href="" download><span id="title">RNA STRUCTURE FROM A TO Z</span></a>
            <span id="publisher">COLD SPRING HARBOR SYMPOSIA ON QUANTITATIVE BIOLOGY, 1987; 52: 135-146</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span>

              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>
            </div>
          </div>

          <div class="year" id="1986">1986</div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Cruz P, Hall K, Puglisi JD, Davis P, Hardin C, Trulson M, Mathies R, Tinoco, I , Jr, Johnson W Jr, Neilson T</span>
            <a href="" download><span id="title">The Left-Handed Z-Form of Double-Stranded RNA </span></a>
            <span id="publisher">Adenine Press, New York (In Biomolecular Stereodynamics, IV), 1986: 179-200</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span>

              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>
            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Tinoco I Jr., Cruz P, Davis P, Hall K, Hardin CC, Mathies RA, Puglisi JD, Trulson MO, Johnson WC, Neilson T</span>
            <a href="" download><span id="title">Z-RNA: A Left-Handed Double Helix</span></a>
            <span id="publisher">Plenum Press (In Structure and Dynamics of RNA), 1986: 55-68</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span>

              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>
            </div>
          </div>




      </div>
    </div>
      <div id="designed-by">
        Design:
          <a href="https://github.com/lmadigan" target="_blank"><i class="fa fa-github" aria-hidden="true"></i></a>
					<a href="https://www.linkedin.com/in/laurenmadigan/" class="fa fa-linkedin" target="_blank"></a>
      </div>
    </div>

    <script>
    // function offsetAnchor() {
    //     if(location.hash.length !== 0) {
    //       console.log(location.hash);
    //         window.scrollTo(window.scrollX, window.scrollY - 130);
    //     }
    //   }
    //   window.setTimeout(offsetAnchor, 1);
    //

    $('a[href*="#"]:not([href="#"])').click(function() {
    var offset = -150; // <-- change the value here
    if (location.pathname.replace(/^\//,'') == this.pathname.replace(/^\//,'') && location.hostname == this.hostname) {
        var target = $(this.hash);
        target = target.length ? target : $('[name=' + this.hash.slice(1) +']');
        if (target.length) {
            $('html, body').animate({
                scrollTop: target.offset().top + offset
            }, 1000);
            return false;
        }
    }
});

      var el = $('.header');
        var $document = $(document);
        $document.scroll(function() {
          if ($document.scrollTop() >= 230) {
            // user scrolled 50 pixels or more;
            // do stuff
            el.addClass('fix-search').slideDown();
          } else {
            el.removeClass("fix-search");
          }
        });

        $(document).ready(function () {

          $(document).on("click", "div#more-title", function () {
              // $(this).next("div#more").toggleClass("hidden");
              $(this).next("div#more").slideToggle();
          });

          $(document).on("click", "div#more-title", function () {
              $(this).find("i#triangle").toggleClass("rotated");
          });


      });


      var bars = $('#bars');
      bars.click(function() {

        console.log($(this).children().eq(1));
        let menu = $(this).children().eq(1);

        if ($(menu).css('display') == 'none') {
          $(menu).css('display', 'block');
        } else {
          $(menu).css('display', 'none');
        }

      });
			</script>
  </body>
</html>
