<!DOCTYPE html>
<html>
  <head>
    <meta charset="utf-8">
    <title>Puglisi Lab</title>
    <link href="https://fonts.googleapis.com/css?family=Open+Sans+Condensed:300" rel="stylesheet">
    <link href="https://fonts.googleapis.com/css?family=Arimo" rel="stylesheet">
    <script src="https://use.fontawesome.com/7ccc97b483.js"></script>
    <script type="text/javascript" src="jquery-3.2.0.min.js"></script>
    <link rel="stylesheet" href="assets/reset.css" />
    <link rel="stylesheet" href="assets/main.css" />
    <link rel="stylesheet" href="assets/publications.css" />
  </head>
  <body>
    <div id="page-wrapper">

      <header class="header">
        <h1 id="logo"><a href="index.html">
          <div id="logo-img">
          <img src="images/Logo_of_Stanford_University.png" height="50" width="50"></div>
          <div id="puglisi">Puglisi</div>
          <div id="laboratory">Laboratory</div></a></h1>
        <nav id="nav">
          <a href="research.html"><ul id="nav-title">Research</ul></a>
          <a href="people.html"><ul id="nav-title">People</ul></a>
          <a href="publications.html"><ul id="nav-title">Publications</ul></a>
          <a href="abroad.html"><ul id="nav-title">Abroad</ul></a>
          <a href="contact.html"><ul id="nav-title">Contact</ul></a>
        </nav>
      </header>
      <div id="background-photo">
        <div class="welcome 1">Our Publications</div></br>
        <!-- <h3 id="history">Let's make history</h3> -->
      </div>

      <div id="publications-wrapper">
        <div id="year-wrapper">

          <div id="year">2016</div>
          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Coey, A., Larsen, K., Puglisi, J. D., Puglisi, E. V.</span>
            <a href="" download><span id="title">Heterogeneous structures formed by conserved RNA sequences within the HIV reverse transcription initiation site</span></a>
            <span id="publisher">RNA, 2016; 22 (11): 1689-1698</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span>
Reverse transcription is a key process in the early steps of HIV infection. This process initiates within a specific complex formed by the 5' UTR of the HIV genomic RNA (vRNA) and a host primer tRNA(Lys)3 Using nuclear magnetic resonance (NMR) spectroscopy and single-molecule fluorescence spectroscopy, we detect two distinct conformers adopted by the tRNA/vRNA initiation complex. We directly show that an interaction between the conserved 8-nucleotide viral RNA primer activation signal (PAS) and the primer tRNA occurs in one of these conformers. This intermolecular PAS interaction likely induces strain on a vRNA intramolecular helix, which must be broken for reverse transcription to initiate. We propose a mechanism by which this vRNA/tRNA conformer relieves the kinetic block formed by the vRNA intramolecular helix to initiate reverse transcription.
              <div id="webof-science"></div>
              <div id="pubmed"></div>
              </div>

            </div>
          </div>



          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Navon, S. P., Kornberg, G., Chen, J., Schwartzman, T., Tsai, A., Puglisi, E. V., Puglisi, J. D., Adir, N.</span>
            <a href="" download><span id="title">Amino acid sequence repertoire of the bacterial proteome and the occurrence of untranslatable sequences</span></a>
            <span id="publisher"> PROCEEDINGS OF THE NATIONAL ACADEMY OF SCIENCES OF THE UNITED STATES OF AMERICA, 2016; 113 (26): 7166-7170</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span>
                Bioinformatic analysis of Escherichia coli proteomes revealed that all possible amino acid triplet sequences occur at their expected frequencies, with four exceptions. Two of the four underrepresented sequences (URSs) were shown to interfere with translation in vivo and in vitro. Enlarging the URS by a single amino acid resulted in increased translational inhibition. Single-molecule methods revealed stalling of translation at the entrance of the peptide exit tunnel of the ribosome, adjacent to ribosomal nucleotides A2062 and U2585. Interaction with these same ribosomal residues is involved in regulation of translation by longer, naturally occurring protein sequences. The E. coli exit tunnel has evidently evolved to minimize interaction with the exit tunnel and maximize the sequence diversity of the proteome, although allowing some interactions for regulatory purposes. Bioinformatic analysis of the human proteome revealed no underrepresented triplet sequences, possibly reflecting an absence of regulation by interaction with the exit tunnel.
                <div id="webof-science"></div>
                <div id="pubmed"></div>
                
              </div>

            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Chen, J., Choi, J., O'Leary, S. E., Prabhakar, A., Petrov, A., Grosely, R., Puglisi, E. V., Puglisi, J. D.</span>
            <a href="" download><span id="title">The molecular choreography of protein synthesis: translational control, regulation, and pathways </span></a>
            <span id="publisher">QUARTERLY REVIEWS OF BIOPHYSICS, 2016; 49</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span>

              <div id="webof-science"></div>
              <div id="pubmed"></div>
              </div>
            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Petrov, A., Grosely, R., Chen, J., O'Leary, S. E., Puglisi, J. D.</span>
            <a href="" download><span id="title">Multiple Parallel Pathways of Translation Initiation on the CrPV IRES.</span></a>
            <span id="publisher">Molecular cell, 2016; 62 (1): 92-103</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span>
                The complexity of eukaryotic translation allows fine-tuned regulation of protein synthesis. Viruses use internal ribosome entry sites (IRESs) to minimize or, like the CrPV IRES, eliminate the need for initiation factors. Here, by exploiting the CrPV IRES, we observed the entire process of initiation and transition to elongation in real time. We directly tracked the CrPV IRES, 40S and 60S ribosomal subunits, and tRNA using single-molecule fluorescence spectroscopy and identified multiple parallel initiation pathways within the system. Our results distinguished two pathways of 80S:CrPV IRES complex assembly that produce elongation-competent complexes. Following 80S assembly, the requisite eEF2-mediated translocation results in an unstable intermediate that is captured by binding of the elongator tRNA. Whereas initiation can occur in the 0 and +1 frames, the arrival of the first tRNA defines the reading frame and strongly favors 0 frame initiation. Overall, even in the simplest system, an intricate reaction network regulates translation initiation.

              <div id="webof-science"></div>
              <div id="pubmed"></div>
              </div>
            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Choi, J., Ieong, K., Demirci, H., Chen, J., Petrov, A., Prabhakar, A., O'Leary, S. E., Dominissini, D., Rechavi, G., Soltis, S. M., Ehrenberg, M., Puglisi, J. D.</span>
            <a href="" download><span id="title">N(6)-methyladenosine in mRNA disrupts tRNA selection and translation-elongation dynamics.</span></a>
            <span id="publisher">Nature structural & molecular biology, 2016; 23 (2): 110-115</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span>
N(6)-methylation of adenosine (forming m(6)A) is the most abundant post-transcriptional modification within the coding region of mRNA, but its role during translation remains unknown. Here, we used bulk kinetic and single-molecule methods to probe the effect of m(6)A in mRNA decoding. Although m(6)A base-pairs with uridine during decoding, as shown by X-ray crystallographic analyses of Thermus thermophilus ribosomal complexes, our measurements in an Escherichia coli translation system revealed that m(6)A modification of mRNA acts as a barrier to tRNA accommodation and translation elongation. The interaction between an m(6)A-modified codon and cognate tRNA echoes the interaction between a near-cognate codon and tRNA, because delay in tRNA accommodation depends on the position and context of m(6)A within codons and on the accuracy level of translation. Overall, our results demonstrate that chemical modification of mRNA can change translational dynamics.
              <div id="webof-science"></div>
              <div id="pubmed"></div>
              </div>
            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Tsai, A., Puglisi, J. D., Uemura, S.</span>
            <a href="" download><span id="title">Probing the Translation Dynamics of Ribosomes Using Zero-Mode Waveguides. </span></a>
            <span id="publisher">Progress in molecular biology and translational science, 2016; 139: 1-43</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span>
                In order to coordinate the complex biochemical and structural feat of converting triple-nucleotide codons into their corresponding amino acids, the ribosome must physically manipulate numerous macromolecules including the mRNA, tRNAs, and numerous translation factors. The ribosome choreographs binding, dissociation, physical movements, and structural rearrangements so that they synergistically harness the energy from biochemical processes, including numerous GTP hydrolysis steps and peptide bond formation. Due to the dynamic and complex nature of translation, the large cast of ligands involved, and the large number of possible configurations, tracking the global time evolution or dynamics of the ribosome complex in translation has proven to be challenging for bulk methods. Conventional single-molecule fluorescence experiments on the other hand require low concentrations of fluorescent ligands to reduce background noise. The significantly reduced bimolecular association rates under those conditions limit the number of steps that can be observed within the time window available to a fluorophore. The advent of zero-mode waveguide (ZMW) technology has allowed the study of translation at near-physiological concentrations of labeled ligands, moving single-molecule fluorescence microscopy beyond focused model systems into studying the global dynamics of translation in realistic setups. This chapter reviews the recent works using the ZMW technology to dissect the mechanism of translation initiation and elongation in prokaryotes, including complex processes such as translational stalling and frameshifting. Given the success of the technology, similarly complex biological processes could be studied in near-physiological conditions with the controllability of conventional in vitro experiments.

              <div id="webof-science"></div>
              <div id="pubmed"></div>
              </div>
            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Siprashvili, Z., Webster, D. E., Johnston, D., Shenoy, R. M., Ungewickell, A. J., Bhaduri, A., Flockhart, R., Zarnegar, B. J., Che, Y., Meschi, F., Puglisi, J. D., Khavari, P. A.</span>
            <a href="" download><span id="title">The noncoding RNAs SNORD50A and SNORD50B bind K-Ras and are recurrently deleted in human cancer.</span></a>
            <span id="publisher"> Nature genetics, 2016; 48 (1): 53-58</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span>
                Small nucleolar RNAs (snoRNAs) are conserved noncoding RNAs best studied as ribonucleoprotein (RNP) guides in RNA modification. To explore their role in cancer, we compared 5,473 tumor-normal genome pairs to identify snoRNAs with frequent copy number loss. The SNORD50A-SNORD50B snoRNA locus was deleted in 10-40% of 12 common cancers, where its loss was associated with reduced survival. A human protein microarray screen identified direct SNORD50A and SNORD50B RNA binding to K-Ras. Loss of SNORD50A and SNORD50B increased the amount of GTP-bound, active K-Ras and hyperactivated Ras-ERK1/ERK2 signaling. Loss of these snoRNAs also increased binding by farnesyltransferase to K-Ras and increased K-Ras prenylation, suggesting that KRAS mutation might synergize with SNORD50A and SNORD50B loss in cancer. In agreement with this hypothesis, CRISPR-mediated deletion of SNORD50A and SNORD50B in KRAS-mutant tumor cells enhanced tumorigenesis, and SNORD50A and SNORD50B deletion and oncogenic KRAS mutation co-occurred significantly in multiple human tumor types. SNORD50A and SNORD50B snoRNAs thus directly bind and inhibit K-Ras and are recurrently deleted in human cancer.
              <div id="webof-science"></div>
              <div id="pubmed"></div>
              </div>
            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Sierra, R. G., Gati, C., Laksmono, H., Dao, E. H., Gul, S., Fuller, F., Kern, J., Chatterjee, R., Ibrahim, M., Brewster, A. S., Young, I. D., Michels-Clark, T., Aquila, A., Liang, M., Hunter, M. S., Koglin, J. E., Boutet, S., Junco, E. A., Hayes, B., Bogan, M. J., Hampton, C. Y., Puglisi, E. V., Sauter, N. K., Stan, C. A., Zouni, A., Yano, J., Yachandra, V. K., Soltis, S. M., Puglisi, J. D., DeMirci, H.</span>
            <a href="" download><span id="title">Concentric-flow electrokinetic injector enables serial crystallography of ribosome and photosystem II</span></a>
            <span id="publisher">NATURE METHODS,2016; 13 (1): 59-?</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span>

              <div id="webof-science"></div>
              <div id="pubmed"></div>
              </div>
            </div>
          </div>

          <div id="year">2015</div>
          <div id="publication-wrapper">
          <div id="publication">
            <span id="author"></span>
            <a href="" download><span id="title"></span></a>
            <span id="publisher">, </span></div>
            <div id="more-title"><i id="triangle" class="fa fa-caret-right" aria-hidden="true"></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span>
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>
              
            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author"></span>
            <a href="" download><span id="title"></span></a>
            <span id="publisher">, </span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span>
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>
              
            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author"></span>
            <a href="" download><span id="title"></span></a>
            <span id="publisher">, </span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span>
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>
              
            </div>
          </div>


          <div id="publication-wrapper">
          <div id="publication">
            <span id="author"></span>
            <a href="" download><span id="title"></span></a>
            <span id="publisher">, </span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span>
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>
              
            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author"></span>
            <a href="" download><span id="title"></span></a>
            <span id="publisher">, </span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span>
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>
              
            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author"></span>
            <a href="" download><span id="title"></span></a>
            <span id="publisher">, </span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span>
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>
              
            </div>
          </div>

      </div>
    </div>
      <div id="designed-by">
        Design:
          <a href="https://github.com/lmadigan" target="_blank"><i class="fa fa-github" aria-hidden="true"></i></a>
					<a href="https://www.linkedin.com/in/laurenmadigan/" class="fa fa-linkedin" target="_blank"></a>
      </div>
    </div>

    <script>
      var el = $('.header');
        var $document = $(document);
        $document.scroll(function() {
          if ($document.scrollTop() >= 230) {
            // user scrolled 50 pixels or more;
            // do stuff
            el.addClass('fix-search').slideDown();
          } else {
            el.removeClass("fix-search");
          }
        });

        $(document).ready(function () {

          $(document).on("click", "div#more-title", function () {
              // $(this).next("div#more").toggleClass("hidden");
              $(this).next("div#more").slideToggle();
          });

          $(document).on("click", "div#more-title", function () {
              $(this).find("i#triangle").toggleClass("rotated");
          });


      });
			</script>
  </body>
</html>
