<!DOCTYPE html>
<html>
  <head>
    <meta charset="utf-8">
    <title>Puglisi Lab</title>
    <link href="https://fonts.googleapis.com/css?family=Open+Sans+Condensed:300" rel="stylesheet">
    <link href="https://fonts.googleapis.com/css?family=Arimo" rel="stylesheet">
    <script src="https://use.fontawesome.com/7ccc97b483.js"></script>
    <script type="text/javascript" src="jquery-3.2.0.min.js"></script>
    <!-- <link rel="stylesheet" href="assets/reset.css" /> -->
    <link rel="stylesheet" href="assets/main.css" />
    <!-- <link rel="stylesheet" href="assets/publications.css" /> -->
  </head>
  <body>
    <div id="page-wrapper">

      <header class="header">
        <h1 id="logo"><a href="index.html">
          <div id="logo-img">
          <img src="images/Logo_of_Stanford_University.png" height="50" width="50"></div>
          <div id="puglisi">Puglisi</div>
          <div id="laboratory">Lab</div></a></h1>
        <nav id="nav">
          <a href="research.html"><ul id="nav-title">Research</ul></a>
          <a href="people.html"><ul id="nav-title">People</ul></a>
          <a href="publications.html"><ul id="nav-title">Publications</ul></a>
          <a href="abroad.html"><ul id="nav-title">Abroad</ul></a>
          <a href="contact.html"><ul id="nav-title">Contact</ul></a>
        </nav>
      </header>
      <div id="background-photo">
        <div class="welcome 1">Our Publications</div></br>
        <!-- <h3 id="history">Let's make history</h3> -->
      </div>

      <div id="publications-wrapper">
        <div id="year-wrapper">

          <div id="year">2016</div>
          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Coey, A., Larsen, K., Puglisi, J. D., Puglisi, E. V.</span>
            <a href="" download><span id="title">Heterogeneous structures formed by conserved RNA sequences within the HIV reverse transcription initiation site</span></a>
            <span id="publisher">RNA, 2016; 22 (11): 1689-1698</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span>
Reverse transcription is a key process in the early steps of HIV infection. This process initiates within a specific complex formed by the 5' UTR of the HIV genomic RNA (vRNA) and a host primer tRNA(Lys)3 Using nuclear magnetic resonance (NMR) spectroscopy and single-molecule fluorescence spectroscopy, we detect two distinct conformers adopted by the tRNA/vRNA initiation complex. We directly show that an interaction between the conserved 8-nucleotide viral RNA primer activation signal (PAS) and the primer tRNA occurs in one of these conformers. This intermolecular PAS interaction likely induces strain on a vRNA intramolecular helix, which must be broken for reverse transcription to initiate. We propose a mechanism by which this vRNA/tRNA conformer relieves the kinetic block formed by the vRNA intramolecular helix to initiate reverse transcription.
              <div id="webof-science"></div>
              <div id="pubmed"></div>
              </div>

            </div>
          </div>



          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Navon, S. P., Kornberg, G., Chen, J., Schwartzman, T., Tsai, A., Puglisi, E. V., Puglisi, J. D., Adir, N.</span>
            <a href="" download><span id="title">Amino acid sequence repertoire of the bacterial proteome and the occurrence of untranslatable sequences</span></a>
            <span id="publisher"> PROCEEDINGS OF THE NATIONAL ACADEMY OF SCIENCES OF THE UNITED STATES OF AMERICA, 2016; 113 (26): 7166-7170</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span>
                Bioinformatic analysis of Escherichia coli proteomes revealed that all possible amino acid triplet sequences occur at their expected frequencies, with four exceptions. Two of the four underrepresented sequences (URSs) were shown to interfere with translation in vivo and in vitro. Enlarging the URS by a single amino acid resulted in increased translational inhibition. Single-molecule methods revealed stalling of translation at the entrance of the peptide exit tunnel of the ribosome, adjacent to ribosomal nucleotides A2062 and U2585. Interaction with these same ribosomal residues is involved in regulation of translation by longer, naturally occurring protein sequences. The E. coli exit tunnel has evidently evolved to minimize interaction with the exit tunnel and maximize the sequence diversity of the proteome, although allowing some interactions for regulatory purposes. Bioinformatic analysis of the human proteome revealed no underrepresented triplet sequences, possibly reflecting an absence of regulation by interaction with the exit tunnel.
                <div id="webof-science"></div>
                <div id="pubmed"></div>

              </div>

            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Chen, J., Choi, J., O'Leary, S. E., Prabhakar, A., Petrov, A., Grosely, R., Puglisi, E. V., Puglisi, J. D.</span>
            <a href="" download><span id="title">The molecular choreography of protein synthesis: translational control, regulation, and pathways </span></a>
            <span id="publisher">QUARTERLY REVIEWS OF BIOPHYSICS, 2016; 49</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span>

              <div id="webof-science"></div>
              <div id="pubmed"></div>
              </div>
            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Petrov, A., Grosely, R., Chen, J., O'Leary, S. E., Puglisi, J. D.</span>
            <a href="" download><span id="title">Multiple Parallel Pathways of Translation Initiation on the CrPV IRES.</span></a>
            <span id="publisher">Molecular cell, 2016; 62 (1): 92-103</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span>
                The complexity of eukaryotic translation allows fine-tuned regulation of protein synthesis. Viruses use internal ribosome entry sites (IRESs) to minimize or, like the CrPV IRES, eliminate the need for initiation factors. Here, by exploiting the CrPV IRES, we observed the entire process of initiation and transition to elongation in real time. We directly tracked the CrPV IRES, 40S and 60S ribosomal subunits, and tRNA using single-molecule fluorescence spectroscopy and identified multiple parallel initiation pathways within the system. Our results distinguished two pathways of 80S:CrPV IRES complex assembly that produce elongation-competent complexes. Following 80S assembly, the requisite eEF2-mediated translocation results in an unstable intermediate that is captured by binding of the elongator tRNA. Whereas initiation can occur in the 0 and +1 frames, the arrival of the first tRNA defines the reading frame and strongly favors 0 frame initiation. Overall, even in the simplest system, an intricate reaction network regulates translation initiation.

              <div id="webof-science"></div>
              <div id="pubmed"></div>
              </div>
            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Choi, J., Ieong, K., Demirci, H., Chen, J., Petrov, A., Prabhakar, A., O'Leary, S. E., Dominissini, D., Rechavi, G., Soltis, S. M., Ehrenberg, M., Puglisi, J. D.</span>
            <a href="" download><span id="title">N(6)-methyladenosine in mRNA disrupts tRNA selection and translation-elongation dynamics.</span></a>
            <span id="publisher">Nature structural & molecular biology, 2016; 23 (2): 110-115</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span>
N(6)-methylation of adenosine (forming m(6)A) is the most abundant post-transcriptional modification within the coding region of mRNA, but its role during translation remains unknown. Here, we used bulk kinetic and single-molecule methods to probe the effect of m(6)A in mRNA decoding. Although m(6)A base-pairs with uridine during decoding, as shown by X-ray crystallographic analyses of Thermus thermophilus ribosomal complexes, our measurements in an Escherichia coli translation system revealed that m(6)A modification of mRNA acts as a barrier to tRNA accommodation and translation elongation. The interaction between an m(6)A-modified codon and cognate tRNA echoes the interaction between a near-cognate codon and tRNA, because delay in tRNA accommodation depends on the position and context of m(6)A within codons and on the accuracy level of translation. Overall, our results demonstrate that chemical modification of mRNA can change translational dynamics.
              <div id="webof-science"></div>
              <div id="pubmed"></div>
              </div>
            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Tsai, A., Puglisi, J. D., Uemura, S.</span>
            <a href="" download><span id="title">Probing the Translation Dynamics of Ribosomes Using Zero-Mode Waveguides. </span></a>
            <span id="publisher">Progress in molecular biology and translational science, 2016; 139: 1-43</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span>
                In order to coordinate the complex biochemical and structural feat of converting triple-nucleotide codons into their corresponding amino acids, the ribosome must physically manipulate numerous macromolecules including the mRNA, tRNAs, and numerous translation factors. The ribosome choreographs binding, dissociation, physical movements, and structural rearrangements so that they synergistically harness the energy from biochemical processes, including numerous GTP hydrolysis steps and peptide bond formation. Due to the dynamic and complex nature of translation, the large cast of ligands involved, and the large number of possible configurations, tracking the global time evolution or dynamics of the ribosome complex in translation has proven to be challenging for bulk methods. Conventional single-molecule fluorescence experiments on the other hand require low concentrations of fluorescent ligands to reduce background noise. The significantly reduced bimolecular association rates under those conditions limit the number of steps that can be observed within the time window available to a fluorophore. The advent of zero-mode waveguide (ZMW) technology has allowed the study of translation at near-physiological concentrations of labeled ligands, moving single-molecule fluorescence microscopy beyond focused model systems into studying the global dynamics of translation in realistic setups. This chapter reviews the recent works using the ZMW technology to dissect the mechanism of translation initiation and elongation in prokaryotes, including complex processes such as translational stalling and frameshifting. Given the success of the technology, similarly complex biological processes could be studied in near-physiological conditions with the controllability of conventional in vitro experiments.

              <div id="webof-science"></div>
              <div id="pubmed"></div>
              </div>
            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Siprashvili, Z., Webster, D. E., Johnston, D., Shenoy, R. M., Ungewickell, A. J., Bhaduri, A., Flockhart, R., Zarnegar, B. J., Che, Y., Meschi, F., Puglisi, J. D., Khavari, P. A.</span>
            <a href="" download><span id="title">The noncoding RNAs SNORD50A and SNORD50B bind K-Ras and are recurrently deleted in human cancer.</span></a>
            <span id="publisher"> Nature genetics, 2016; 48 (1): 53-58</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span>
                Small nucleolar RNAs (snoRNAs) are conserved noncoding RNAs best studied as ribonucleoprotein (RNP) guides in RNA modification. To explore their role in cancer, we compared 5,473 tumor-normal genome pairs to identify snoRNAs with frequent copy number loss. The SNORD50A-SNORD50B snoRNA locus was deleted in 10-40% of 12 common cancers, where its loss was associated with reduced survival. A human protein microarray screen identified direct SNORD50A and SNORD50B RNA binding to K-Ras. Loss of SNORD50A and SNORD50B increased the amount of GTP-bound, active K-Ras and hyperactivated Ras-ERK1/ERK2 signaling. Loss of these snoRNAs also increased binding by farnesyltransferase to K-Ras and increased K-Ras prenylation, suggesting that KRAS mutation might synergize with SNORD50A and SNORD50B loss in cancer. In agreement with this hypothesis, CRISPR-mediated deletion of SNORD50A and SNORD50B in KRAS-mutant tumor cells enhanced tumorigenesis, and SNORD50A and SNORD50B deletion and oncogenic KRAS mutation co-occurred significantly in multiple human tumor types. SNORD50A and SNORD50B snoRNAs thus directly bind and inhibit K-Ras and are recurrently deleted in human cancer.
              <div id="webof-science"></div>
              <div id="pubmed"></div>
              </div>
            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Sierra, R. G., Gati, C., Laksmono, H., Dao, E. H., Gul, S., Fuller, F., Kern, J., Chatterjee, R., Ibrahim, M., Brewster, A. S., Young, I. D., Michels-Clark, T., Aquila, A., Liang, M., Hunter, M. S., Koglin, J. E., Boutet, S., Junco, E. A., Hayes, B., Bogan, M. J., Hampton, C. Y., Puglisi, E. V., Sauter, N. K., Stan, C. A., Zouni, A., Yano, J., Yachandra, V. K., Soltis, S. M., Puglisi, J. D., DeMirci, H.</span>
            <a href="" download><span id="title">Concentric-flow electrokinetic injector enables serial crystallography of ribosome and photosystem II</span></a>
            <span id="publisher">NATURE METHODS,2016; 13 (1): 59-?</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span>
                We describe a concentric-flow electrokinetic injector for efficiently delivering microcrystals for serial femtosecond X-ray crystallography analysis that enables studies of challenging biological systems in their unadulterated mother liquor. We used the injector to analyze microcrystals of Geobacillus stearothermophilus thermolysin (2.2-Å structure), Thermosynechococcus elongatus photosystem II (<3-Å diffraction) and Thermus thermophilus small ribosomal subunit bound to the antibiotic paromomycin at ambient temperature (3.4-Å structure).
              <div id="webof-science"></div>
              <div id="pubmed"></div>
              </div>
            </div>
          </div>

          <div id="year">2015</div>
          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Chen, J., Coakley, A., O'Connor, M., Petrov, A., O'Leary, S. E., Atkins, J. F., Puglisi, J. D.</span>
            <a href="" download><span id="title">Coupling of mRNA Structure Rearrangement to Ribosome Movement during Bypassing of Non-coding Regions</span></a>
            <span id="publisher">Cell, 2015; 163 (5): 1267-1280</span></div>
            <div id="more-title"><i id="triangle" class="fa fa-caret-right" aria-hidden="true"></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span>
                Nearly half of the ribosomes translating a particular bacteriophage T4 mRNA bypass a region of 50 nt, resuming translation 3′ of this gap. How this large-scale, specific hop occurs and what determines whether a ribosome bypasses remain unclear. We apply single-molecule fluorescence with zero-mode waveguides to track individual Escherichia coli ribosomes during translation of T4’s gene 60 mRNA. Ribosomes that bypass are characterized by a 10- to 20-fold longer pause in a non-canonical rotated state at the take-off codon. During the pause, mRNA secondary structure rearrangements are coupled to ribosome forward movement, facilitated by nascent peptide interactions that disengage the ribosome anticodon-codon interactions for slippage. Close to the landing site, the ribosome then scans mRNA in search of optimal base-pairing interactions. Our results provide a mechanistic and conformational framework for bypassing, highlighting a non-canonical ribosomal state to allow for mRNA structure refolding to drive large-scale ribosome movements.
                </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>

            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Nilsson, O. B., Hedman, R., Marino, J., Wickles, S., Bischoff, L., Johansson, M., Mueller-Lucks, A., Trovato, F., Puglisi, J. D., O'Brien, E. P., Beckmann, R., Von Heijne, G.</span>
            <a href="" download><span id="title">Cotranslational Protein Folding inside the Ribosome Exit Tunnel</span></a>
            <span id="publisher">Cell Reports, 2015; 12 (10): 1533-1540</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span>
                At what point during translation do proteins fold? It is well established that proteins can fold cotranslationally outside the ribosome exit tunnel, whereas studies of folding inside the exit tunnel have so far detected only the formation of helical secondary structure and collapsed or partially structured folding intermediates. Here, using a combination of cotranslational nascent chain force measurements, inter-subunit fluorescence resonance energy transfer studies on single translating ribosomes, molecular dynamics simulations, and cryoelectron microscopy, we show that a small zinc-finger domain protein can fold deep inside the vestibule of the ribosome exit tunnel. Thus, for small protein domains, the ribosome itself can provide the kind of sheltered folding environment that chaperones provide for larger proteins.
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>

            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Puglisi, J. D.</span>
            <a href="" download><span id="title">SYNTHETIC BIOLOGY Ribosomal ties that bind</span></a>
            <span id="publisher">Nature, 2015; 524 (7563): 45-46</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span>
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>

            </div>
          </div>


          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Puglisi, J. D.</span>
            <a href="" download><span id="title">Protein synthesis. The delicate dance of translation and folding.</span></a>
            <span id="publisher">Science, 2015; 348 (6233): 399-400</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span>

The past decade has seen rapid growth in our knowledge of how proteins are synthesized in cells. This includes the basic step of how transfer RNAs (tRNAs) decode messenger RNAs (mRNAs) with high fidelity and speed, how the ribosome moves along mRNA from codon to codon during translation, and how synthesis of the corresponding polypeptide chain is initiated and terminated at specific points on the mRNA (1). Structures of the ribosomal particles—megadalton RNA-protein assemblies—have provided detailed molecular views of the active sites for mRNA decoding and peptide bond formation, and suggested pathways for movement of ligands, factors, and the ribosomal subunits themselves. Obscured in this rich cache of knowledge is the fate of the protein product. How does it fold during mRNA translation and how might protein folding affect translation itself? These questions are addressed by elegant biophysical and biochemical approaches reported by Goldman et al. (2) and Kim et al. (3) on pages 457 and 444, respectively, of this issue, adding to a growing appreciation of cotranslational protein folding (4–6).
                </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>

            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Puglisi, J. D.</span>
            <a href="" download><span id="title">RNA dances to center stage</span></a>
            <span id="publisher">RNA - A Publication of the RNA Society,2015; 21 (4): 712-713 </span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span>
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>

            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Capece, M. C., Kornberg, G. L., Petrov, A., Puglisi, J. D.</span>
            <a href="" download><span id="title">A simple real-time assay for in vitro translation</span></a>
            <span id="publisher">RNA, 2015; 21 (2): 296-305 </span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span>
                A high-throughput assay for real-time measurement of translation rates in cell-free protein synthesis (SNAP assay) is described. The SNAP assay enables quantitative, real-time measurement of overall translation rates in vitro via the synthesis of O(6)-alkylguanine DNA O(6)-alkyltransferase (SNAP). SNAP production is continuously detected by fluorescence produced by the reaction of SNAP with a range of quenched fluorogenic substrates. The capabilities of the assay are exemplified by measurements of the activities of Escherichia coli MRE600 ribosomes and fluorescently labeled E. coli mutant ribosomes in the PURExpress translation system and by determination of the 50% inhibitory concentrations (IC50) of three common macrolide antibiotics.
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>

            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Fuchs, G., Petrov, A. N., Marceau, C. D., Popov, L. M., Chen, J., O'Leary, S. E., Wang, R., Carette, J. E., Sarnow, P., Puglisi, J. D.</span>
            <a href="" download><span id="title">Kinetic pathway of 40S ribosomal subunit recruitment to hepatitis C virus internal ribosome entry site</span></a>
            <span id="publisher">Proceedings of the National Academy of Sciences of the United States of America, 2015; 112 (2): 319-325</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span>
                Translation initiation can occur by multiple pathways. To delineate these pathways by single-molecule methods, fluorescently labeled ribosomal subunits are required. Here, we labeled human 40S ribosomal subunits with a fluorescent SNAP-tag at ribosomal protein eS25 (RPS25). The resulting ribosomal subunits could be specifically labeled in living cells and in vitro. Using single-molecule Förster resonance energy transfer (FRET) between RPS25 and domain II of the hepatitis C virus (HCV) internal ribosome entry site (IRES), we measured the rates of 40S subunit arrival to the HCV IRES. Our data support a single-step model of HCV IRES recruitment to 40S subunits, irreversible on the initiation time scale. We furthermore demonstrated that after binding, the 40S:HCV IRES complex is conformationally dynamic, undergoing slow large-scale rearrangements. Addition of translation extracts suppresses these fluctuations, funneling the complex into a single conformation on the 80S assembly pathway. These findings show that 40S:HCV IRES complex formation is accompanied by dynamic conformational rearrangements that may be modulated by initiation factors.
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>

            </div>
          </div>


          <div id="year">2014</div>
          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Noriega, T. R., Chen, J., Walter, P., Puglisi, J. D.
</span>
            <a href="" download><span id="title">Real-time observation of signal recognition particle binding to actively translating ribosomes</span></a>
            <span id="publisher">Elife, 2014; 3 </span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span>
                The signal recognition particle (SRP) directs translating ribosome-nascent chain complexes (RNCs) that display a signal sequence to protein translocation channels in target membranes. All previous work on the initial step of the targeting reaction, when SRP binds to RNCs, used stalled and non-translating RNCs. This meant that an important dimension of the co-translational process remained unstudied. We apply single-molecule fluorescence measurements to observe directly and in real-time E. coli SRP binding to actively translating RNCs. We show at physiologically relevant SRP concentrations that SRP-RNC association and dissociation rates depend on nascent chain length and the exposure of a functional signal sequence outside the ribosome. Our results resolve a long-standing question: how can a limited, sub-stoichiometric pool of cellular SRP effectively distinguish RNCs displaying a signal sequence from those that are not? The answer is strikingly simple: as originally proposed, SRP only stably engages translating RNCs exposing a functional signal sequence.
                </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>

            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Chen, J., Petrov, A., Johansson, M., Tsai, A., O'Leary, S. E., Puglisi, J. D.</span>
            <a href="https://www.dropbox.com/sh/kfqdilhf8jms13n/AABMIgDaUKlB2uvsDKL8pdsJa?dl=1&lst=&preview=Chen_nature_2014.pdf" download><span id="title">Dynamic pathways of -1 translational frameshifting.</span></a>
            <span id="publisher">Nature, 2014; 512 (7514): 328-332</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span>
                Spontaneous changes in the reading frame of translation are rare (frequency of 10(-3) to 10(-4) per codon), but can be induced by specific features in the messenger RNA (mRNA). In the presence of mRNA secondary structures, a heptanucleotide 'slippery sequence' usually defined by the motif X XXY YYZ, and (in some prokaryotic cases) mRNA sequences that base pair with the 3' end of the 16S ribosomal rRNA (internal Shine-Dalgarno sequences), there is an increased probability that a specific programmed change of frame occurs, wherein the ribosome shifts one nucleotide backwards into an overlapping reading frame (-1 frame) and continues by translating a new sequence of amino acids. Despite extensive biochemical and genetic studies, there is no clear mechanistic description for frameshifting. Here we apply single-molecule fluorescence to track the compositional and conformational dynamics of individual ribosomes at each codon during translation of a frameshift-inducing mRNA from the dnaX gene in Escherichia coli. Ribosomes that frameshift into the -1 frame are characterized by a tenfold longer pause in elongation compared to non-frameshifted ribosomes, which translate through unperturbed. During the pause, interactions of the ribosome with the mRNA stimulatory elements uncouple EF-G catalysed translocation from normal ribosomal subunit reverse-rotation, leaving the ribosome in a non-canonical intersubunit rotated state with an exposed codon in the aminoacyl-tRNA site (A site). tRNA(Lys) sampling and accommodation to the empty A site and EF-G action either leads to the slippage of the tRNAs into the -1 frame or maintains the ribosome into the 0 frame. Our results provide a general mechanistic and conformational framework for -1 frameshifting, highlighting multiple kinetic branchpoints during elongation.
                </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>

            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Noriega, T. R., Tsai, A., Elvekrog, M. M., Petrov, A., Neher, S. B., Chen, J., Bradshaw, N., Puglisi, J. D., Walter, P.</span>
            <a href="" download><span id="title">Signal Recognition Particle-ribosome Binding Is Sensitive to Nascent Chain Length</span></a>
            <span id="publisher">Journal of Biological Chemistry, 2014; 289 (28): 19294-19305</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span>
                The signal recognition particle (SRP) directs ribosome-nascent chain complexes (RNCs) displaying signal sequences to protein translocation channels in the plasma membrane of prokaryotes and endoplasmic reticulum of eukaryotes. It was initially proposed that SRP binds the signal sequence when it emerges from an RNC and that successful binding becomes impaired as translation extends the nascent chain, moving the signal sequence away from SRP on the ribosomal surface. Later studies drew this simple model into question, proposing that SRP binding is unaffected by nascent chain length. Here, we reinvestigate this issue using two novel and independent fluorescence resonance energy transfer assays. We show that the arrival and dissociation rates of SRP binding to RNCs vary according to nascent chain length, resulting in the highest affinity shortly after a functional signal sequence emerges from the ribosome. Moreover, we show that SRP binds RNCs in multiple and interconverting conformations, and that conversely, RNCs exist in two conformations distinguished by SRP interaction kinetics.
                </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>

            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Tsai, A., Kornberg, G., Johansson, M., Chen, J., Puglisi, J. D.</span>
            <a href="" download><span id="title">The Dynamics of SecM-Induced Translational Stalling </span></a>
            <span id="publisher">Cell Reports, 2014; 7 (5): 1521-1533</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span>
                SecM is an E. coli secretion monitor capable of stalling translation on the prokaryotic ribosome without cofactors. Biochemical and structural studies have demonstrated that the SecM nascent chain interacts with the 50S subunit exit tunnel to inhibit peptide bond formation. However, the timescales and pathways of stalling on an mRNA remain undefined. To provide a dynamic mechanism for stalling, we directly tracked the dynamics of elongation on ribosomes translating the SecM stall sequence (FSTPVWISQAQGIRAGP) using single-molecule fluorescence techniques. Within 1 min, three peptide-ribosome interactions work cooperatively over the last five codons of the SecM sequence, leading to severely impaired elongation rates beginning from the terminal proline and lasting four codons. Our results suggest that stalling is tightly linked to the dynamics of elongation and underscore the roles that the exit tunnel and nascent chain play in controlling fundamental steps in translation.
                </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>

            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Johansson, M., Chen, J., Tsai, A., Kornberg, G., Puglisi, J. D.</span>
            <a href="" download><span id="title">Sequence-Dependent Elongation Dynamics on Macrolide-Bound Ribosomes</span></a>
            <span id="publisher">Cell Reports, 2014; 7 (5): 1534-1546</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span>
                The traditional view of macrolide antibiotics as plugs inside the ribosomal nascent peptide exit tunnel (NPET) has lately been challenged in favor of a more complex, heterogeneous mechanism, where drug-peptide interactions determine the fate of a translating ribosome. To investigate these highly dynamic processes, we applied single-molecule tracking of elongating ribosomes during inhibition of elongation by erythromycin of several nascent chains, including ErmCL and H-NS, which were shown to be, respectively, sensitive and resistant to erythromycin. Peptide sequence-specific changes were observed in translation elongation dynamics in the presence of a macrolide-obstructed NPET. Elongation rates were not severely inhibited in general by the presence of the drug; instead, stalls or pauses were observed as abrupt events. The dynamic pathways of nascent-chain-dependent elongation pausing in the presence of macrolides determine the fate of the translating ribosome stalling or readthrough.
                </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>

            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Chen, J., Dalal, R. V., Petrov, A. N., Tsai, A., O'Leary, S. E., Chapin, K., Cheng, J., Ewan, M., Hsiung, P., Lundquist, P., Turner, S. W., Hsu, D. R., Puglisi, J. D.</span>
            <a href="" download><span id="title">High-throughput platform for real-time monitoring of biological processes by multicolor single-molecule fluorescence</span></a>
            <span id="publisher">Proceedings of the National Academy of Sciences of the United States of America, 2014; 111 (2): 664-669</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span>
                Zero-mode waveguides provide a powerful technology for studying single-molecule real-time dynamics of biological systems at physiological ligand concentrations. We customized a commercial
                 zero-mode waveguide-based DNA sequencer for use as a versatile instrument for single-molecule fluorescence detection and showed that the system provides long fluorophore lifetimes with good
                 signal to noise and low spectral cross-talk. We then used a ribosomal translation assay to show real-time fluidic delivery during data acquisition, showing it is possible to follow the conformation
                 and composition of thousands of single biomolecules simultaneously through four spectral channels. This instrument allows high-throughput multiplexed dynamics of single-molecule biological processes
                  over long timescales. The instrumentation presented here has broad applications to single-molecule studies of biological systems and is easily accessible to the biophysical community.
                </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>

            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Noriega, T. R., Chen, J., Walter, P., Puglisi, J. D.</span>
            <a href="" download><span id="title">Real-time observation of signal recognition particle binding to actively translating ribosomes.</span></a>
            <span id="publisher">eLife, 2014; 3</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span>
                The signal recognition particle (SRP) directs translating ribosome-nascent chain complexes (RNCs) that display a signal sequence to protein translocation channels in target membranes. All previous work on the initial step of the targeting reaction, when SRP binds to RNCs, used stalled and non-translating RNCs. This meant that an important dimension of the co-translational process remained unstudied. We apply single-molecule fluorescence measurements to observe directly and in real-time E. coli SRP binding to actively translating RNCs. We show at physiologically relevant SRP concentrations that SRP-RNC association and dissociation rates depend on nascent chain length and the exposure of a functional signal sequence outside the ribosome. Our results resolve a long-standing question: how can a limited, sub-stoichiometric pool of cellular SRP effectively distinguish RNCs displaying a signal sequence from those that are not? The answer is strikingly simple: as originally proposed, SRP only stably engages translating RNCs exposing a functional signal sequence.
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>

            </div>
          </div>

          <div id="year">2013</div>
          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">O'Leary, S. E., Petrov, A., Chen, J., Puglisi, J. D.</span>
            <a href="" download><span id="title">Dynamic Recognition of the mRNA Cap by Saccharomyces cerevisiae eIF4E</span></a>
            <span id="publisher">Structure, 2013; 21 (12): 2197-2207</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span>
                Recognition of the mRNA 5' m⁷G(5')ppp(5')N cap is key to translation initiation for most eukaryotic mRNAs. The cap is bound by the eIF4F complex, consisting
                of a cap-binding protein (eIF4E), a "scaffold" protein (eIF4G), and an RNA helicase (eIF4A). As a central early step in initiation, regulation of eIF4F is crucial for cellular viability.
                 Although the structure and function of eIF4E have been defined, a dynamic mechanistic picture of its activity at the molecular level in the eIF4F·mRNA complex is still unavailable.
                 Here, using single-molecule fluorescence, we measured the effects of Saccharomyces cerevisiae eIF4F factors, mRNA secondary structure, and the poly(A)-binding protein Pab1p on eIF4E-mRNA
                  binding dynamics. Our data provide an integrated picture of how eIF4G and mRNA structure modulate eIF4E-mRNA interaction, and uncover an eIF4G- and poly(A)-independent activity of poly(A)-binding
                  protein that prolongs the eIF4E·mRNA complex lifetime.
                </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>

            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Simonetti, A., Marzi, S., Billas, I. M., Tsai, A., Fabbretti, A., Myasnikov, A. G., Roblin, P., Vaiana, A. C., Hazemann, I., Eiler, D., Steitz, T. A., Puglisi, J. D., Gualerzi, C. O., Klaholz, B. P.</span>
            <a href="" download><span id="title">Involvement of protein IF2 N domain in ribosomal subunit joining revealed from architecture and function of the full-length initiation factor</span></a>
            <span id="publisher">PROCEEDINGS OF THE NATIONAL ACADEMY OF SCIENCES OF THE UNITED STATES OF AMERICA,2013; 110 (39): 15656-15661 </span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span>
                Translation initiation factor 2 (IF2) promotes 30S initiation complex (IC) formation and 50S subunit joining, which produces the 70S IC. The architecture of full-length IF2, determined by small angle X-ray diffraction and cryo electron microscopy, reveals a more extended conformation of IF2 in solution and on the ribosome than in the crystal. The N-terminal domain is only partially visible in the 30S IC, but in the 70S IC, it stabilizes interactions between IF2 and the L7/L12 stalk of the 50S, and on its deletion, proper N-formyl-methionyl(fMet)-tRNA(fMet) positioning and efficient transpeptidation are affected. Accordingly, fast kinetics and single-molecule fluorescence data indicate that the N terminus promotes 70S IC formation by stabilizing the productive sampling of the 50S subunit during 30S IC joining. Together, our data highlight the dynamics of IF2-dependent ribosomal subunit joining and the role played by the N terminus of IF2 in this process.
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>

            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Chen, J., Petrov, A., Tsai, A., O'Leary, S. E., Puglisi, J. D.</span>
            <a href="" download><span id="title">Coordinated conformational and compositional dynamics drive ribosome translocation. </span></a>
            <span id="publisher">Nature structural & molecular biology, 2013; 20 (6): 718-727</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span>
                During translation elongation, the ribosome compositional factors elongation factor G (EF-G; encoded by fusA) and tRNA alternately bind to the ribosome to direct protein synthesis and regulate the conformation of the ribosome. Here, we use single-molecule fluorescence with zero-mode waveguides to directly correlate ribosome conformation and composition during multiple rounds of elongation at high factor concentrations in Escherichia coli. Our results show that EF-G bound to GTP (EF-G-GTP) continuously samples both rotational states of the ribosome, binding with higher affinity to the rotated state. Upon successful accommodation into the rotated ribosome, the EF-G-ribosome complex evolves through several rate-limiting conformational changes and the hydrolysis of GTP, which results in a transition back to the nonrotated state and in turn drives translocation and facilitates release of both EF-G-GDP and E-site tRNA. These experiments highlight the power of tracking single-molecule conformation and composition simultaneously in real time.
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>

            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Tsai, A., Uemura, S., Johansson, M., Puglisi, E. V., Marshall, R. A., Aitken, C. E., Korlach, J., Ehrenberg, M., Puglisi, J. D.</span>
            <a href="" download><span id="title">The Impact of Aminoglycosides on the Dynamics of Translation Elongation</span></a>
            <span id="publisher">CELL REPORTS, 2013; 3 (2): 497-508</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span>
                Inferring antibiotic mechanisms on translation through static structures has been challenging, as biological systems are highly dynamic. Dynamic single-molecule methods are also limited to few simultaneously measurable parameters. We have circumvented these limitations with a multifaceted approach to investigate three structurally distinct aminoglycosides that bind to the aminoacyl-transfer RNA site (A site) in the prokaryotic 30S ribosomal subunit: apramycin, paromomycin, and gentamicin. Using several single-molecule fluorescence measurements combined with structural and biochemical techniques, we observed distinct changes to translational dynamics for each aminoglycoside. While all three drugs effectively inhibit translation elongation, their actions are structurally and mechanistically distinct. Apramycin does not displace A1492 and A1493 at the decoding center, as demonstrated by a solution nuclear magnetic resonance structure, causing only limited miscoding; instead, it primarily blocks translocation. Paromomycin and gentamicin, which displace A1492 and A1493, cause significant miscoding, block intersubunit rotation, and inhibit translocation. Our results show the power of combined dynamics, structural, and biochemical approaches to elucidate the complex mechanisms underlying translation and its inhibition.
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>

            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Tsai, A., Chen, J., Kornberg, G., Korlach, J., Uemura, S., Puglisi, J.</span>
            <a href="" download><span id="title">Observing Prokaryotic Translation Elongation in Real-Time using Single-Molecule Fluorescence</span></a>
            <span id="publisher">CELL PRESS, 2013: 257A-257A</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span>
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>

            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Petrov, A., Tsa, A., Puglisi, J. D.</span>
            <a href="" download><span id="title">Analysis of RNA by Analytical Polyacrylamide Gel Electrophoresis</span></a>
            <span id="publisher">LABORATORY METHODS IN ENZYMOLOGY: RNA, 2013; 530: 301-313</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span>
                Polyacrylamide gel electrophoresis (PAGE) is a powerful tool for analyzing RNA samples. Denaturing PAGE provides information on the sample composition and structural integrity of the individual RNA species. Nondenaturing gel electrophoresis allows separation of the conformers and alternatively folded RNA species. It also can be used to resolve RNA protein complexes and to detect RNA complex formation by analyzing changes in the electrophoretic mobility of the RNA. RNA can be visualized within gels by different methods depending on the nature of the detection reagent. RNA molecules can be stained with various dyes, including toluidine blue, SYBR green, and ethidium bromide. Radioactively labeled RNA molecules are visualized by autoradiography, and fluorescently labeled RNA molecules can be observed with a fluorescence scanner. Generally, gels between 0.4 and 1.5mm thick are used for analytical PAGE. Gels thinner than 1mm are fragile and thus usually are not stained but rather are used for radiolabeled RNA. The gels are dried and the radiolabeled RNA is visualized by autoradiography.
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>

            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Petrov, A., Wu, T., Puglisi, E. V., Puglisi, J. D.</span>
            <a href="" download><span id="title">RNA Purification by Preparative Polyacrylamide Gel Electrophoresis</span></a>
            <span id="publisher">LABORATORY METHODS IN ENZYMOLOGY: RNA, 2013; 530: 315-330</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span>
                Preparative polyacrylamide gel electrophoresis (PAGE) is a powerful tool for purifying RNA samples. Denaturing PAGE allows separation of nucleic acids that differ by a single nucleotide in length. It is commonly used to separate and purify RNA species after in vitro transcription, to purify naturally occurring RNA variants such as tRNAs, to remove degradation products, and to purify labeled RNA species. To preserve RNA integrity following purification, RNA is usually visualized by UV shadowing or stained with ethidium bromide or SYBR green dyes.
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>

            </div>
          </div>


        <div id="year">2012</div>
          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Chen, J., Tsai, A., O'Leary, S. E., Petrov, A., Puglisi, J. D.</span>
            <a href="" download><span id="title">Unraveling the dynamics of ribosome translocation</span></a>
            <span id="publisher">CURRENT OPINION IN STRUCTURAL BIOLOGY,2012; 22 (6): 804-814 </span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span>
                Translocation is one of the key events in translation, requiring large-scale conformational changes in the ribosome, movements of two transfer RNAs (tRNAs) across a distance of more than 20?, and the coupled movement of the messenger RNA (mRNA) by one codon, completing one cycle of peptide-chain elongation. Translocation is catalyzed by elongation factor G (EF-G in bacteria), which hydrolyzes GTP in the process. However, how the conformational rearrangements of the ribosome actually drive the movements of the tRNAs and how EF-G GTP hydrolysis plays a role in this process are still unclear. Fluorescence methods, both single-molecule and bulk, have provided a dynamic view of translocation, allowing us to follow the different conformational changes of the ribosome in real-time. The application of electron microscopy has revealed new conformational intermediates during translocation and important structural rearrangements in the ribosome that drive tRNA movement, while computational approaches have added quantitative views of the translational pathway. These recent advances shed light on the process of translocation, providing insight on how to resolve the different descriptions of translocation in the current literature.
                </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>

            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Petrov, A., Chen, J., O'Leary, S., Tsai, A., Puglisi, J. D.</span>
            <a href="" download><span id="title">Single-Molecule Analysis of Translational Dynamics</span></a>
            <span id="publisher">COLD SPRING HARBOR PERSPECTIVES IN BIOLOGY,2012; 4 (9) </span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span>
                Decades of extensive biochemical and biophysical research have outlined the mechanism of translation. Rich structural studies have provided detailed snapshots of the translational machinery at all phases of the translation cycle. However, the relationship between structural dynamics, composition, and function remains unknown. The multistep nature of each stage of the translation cycle results in rapid desynchronization of individual ribosomes, thus hindering elucidation of the underlying mechanisms by conventional bulk biophysical and biochemical methods. Single-molecule approaches unsusceptible to these complications have led to the first glances at both compositional and conformational dynamics on the ribosome and their impact on translational control. These experiments provide the necessary link between static structure and mechanism, often providing new perspectives. Here we review recent advances in the field and their relationship to structural and biochemical data.
                </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>

            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Harvey, C. J., Puglisi, J. D., Pande, V. S., Cane, D. E., Khosla, C.</span>
            <a href="" download><span id="title">Precursor Directed Biosynthesis of an Orthogonally Functional Erythromycin Analogue: Selectivity in the Ribosome Macrolide Binding Pocket</span></a>
            <span id="publisher">JOURNAL OF THE AMERICAN CHEMICAL SOCIETY, 2012; 134 (29): 12259-12265</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span>
                The macrolide antibiotic erythromycin A and its semisynthetic analogues have been among the most useful antibacterial agents for the treatment of infectious diseases. Using a recently developed chemical genetic strategy for precursor-directed biosynthesis and colony bioassay of 6-deoxyerythromycin D analogues, we identified a new class of alkynyl- and alkenyl-substituted macrolides with activities comparable to that of the natural product. Further analysis revealed a marked and unexpected dependence of antibiotic activity on the size and degree of unsaturation of the precursor. Based on these leads, we also report the precursor-directed biosynthesis of 15-propargyl erythromycin A, a novel antibiotic that not only is as potent as erythromycin A with respect to its ability to inhibit bacterial growth and cell-free ribosomal protein biosynthesis but also harbors an orthogonal functional group that is capable of facile chemical modification.
                </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>

            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Tsai, A., Petrov, A., Marshall, R. A., Korlach, J., Uemura, S., Puglisi, J. D.</span>
            <a href="" download><span id="title">Heterogeneous pathways and timing of factor departure during translation initiation</span></a>
            <span id="publisher">NATURE,2012; 487 (7407): 390-394 </span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span>
                The initiation of translation establishes the reading frame for protein synthesis and is a key point of regulation. Initiation involves factor-driven assembly at a start codon of a messenger RNA of an elongation-competent 70S ribosomal particle (in bacteria) from separated 30S and 50S subunits and initiator transfer RNA. Here we establish in Escherichia coli, using direct single-molecule tracking, the timing of initiator tRNA, initiation factor 2 (IF2; encoded by infB) and 50S subunit joining during initiation. Our results show multiple pathways to initiation, with orders of arrival of tRNA and IF2 dependent on factor concentration and composition. IF2 accelerates 50S subunit joining and stabilizes the assembled 70S complex. Transition to elongation is gated by the departure of IF2 after GTP hydrolysis, allowing efficient arrival of elongator tRNAs to the second codon presented in the aminoacyl-tRNA binding site (A site). These experiments highlight the power of single-molecule approaches to delineate mechanisms in complex multicomponent systems.
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>

            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Puglisi, J. D., Williamson, J. R.</span>
            <a href="" download><span id="title">Digging deep into nucleic acid structure and nucleic acid protein recognition </span></a>
            <span id="publisher">CURRENT OPINION IN STRUCTURAL BIOLOGY, 2012; 22 (3): 249-250</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span>
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>

            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Chen, J., Tsai, A., Petrov, A., Puglisi, J. D.</span>
            <a href="" download><span id="title">Nonfluorescent Quenchers To Correlate Single-Molecule Conformational and Compositional Dynamics</span></a>
            <span id="publisher">JOURNAL OF THE AMERICAN CHEMICAL SOCIETY, 2012; 134 (13): 5734-5737</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span>
                Single-molecule Förster resonance energy transfer (smFRET) is a powerful method for studying the conformational dynamics of a biomolecule in real-time. However, studying how interacting ligands correlate with and regulate the conformational dynamics of the biomolecule is extremely challenging because of the availability of a limited number of fluorescent dyes with both high quantum yield and minimal spectral overlap. Here we report the use of a nonfluorescent quencher (Black Hole Quencher, BHQ) as an acceptor for smFRET. Using a Cy3/BHQ pair, we can accurately follow conformational changes of the ribosome during elongation in real time. We demonstrate the application of single-color FRET to correlate the conformational dynamics of the ribosome with the compositional dynamics of tRNA. We use the normal Cy5 FRET acceptor to observe arrival of a fluorescently labeled tRNA with a concomitant transition of the ribosome from the locked to the unlocked conformation. Our results illustrate the potential of nonfluorescent quenchers in single-molecule correlation studies.
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>

            </div>
          </div>


          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Puglisi, J. D., Chen, J., Kornberg, G., O'Leary, S., Petrov, A., Tsai, A.</span>
            <a href="" download><span id="title">Real-Time Dynamics of Translation</span></a>
            <span id="publisher">FEDERATION AMER SOC EXP BIOL. 2012 </span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span>
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>

            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Masuda, T., Petrov, A. N., Iizuka, R., Funatsu, T., Puglisi, J. D., Uemura, S.</span>
            <a href="" download><span id="title">Initiation factor 2, tRNA, and 50S subunits cooperatively stabilize mRNAs on the ribosome during initiation </span></a>
            <span id="publisher">PROCEEDINGS OF THE NATIONAL ACADEMY OF SCIENCES OF THE UNITED STATES OF AMERICA,2012; 109 (13): 4881-4885 </span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span>
                Initiation factor 2 (IF2) is a key factor in initiation of bacterial protein synthesis. It recruits initiator tRNA to the small ribosomal subunit and facilitates joining of the large ribosomal subunit. Using reconstituted translation system of Escherichia coli and optical tweezers, we directly measure the rupture force between single ribosomal complexes and mRNAs for initiation complexes in the presence and the absence of IF2. We demonstrate that IF2 together with codon recognition by initiator tRNA increases the force required to dislocate mRNA from the ribosome complexes; mRNA stabilization by IF2 required the presence of a joined 50S subunit, and was independent of bound guanine nucleotide. IF2 thus helps lock the 70S ribosome over the start codon during initiation, thus maintaining reading frame. Our results show how mRNA is progressively stabilized on the ribosome through distinct steps of initiation.
                </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>

            </div>
          </div>


          <div id="year">2011</div>
          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">
                Puglisi, E. V., Puglisi, J. D.</span>
            <a href="" download><span id="title">Secondary Structure of the HIV Reverse Transcription Initiation Complex by NMR</span></a>
            <span id="publisher">JOURNAL OF MOLECULAR BIOLOGY,2011; 410 (5): 863-874 </span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span>
                Initiation of reverse transcription of genomic RNA is a key early step in replication of the human immunodeficiency virus (HIV) upon infection of a host cell. Viral reverse transcriptase initiates from a specific RNA-RNA complex formed between a host transfer RNA (tRNA(Lys)(3)) and a region at the 5' end of genomic RNA; the 3' end of the tRNA acts as a primer for reverse transcription of genomic RNA. We report here the secondary structure of the HIV genomic RNA-human tRNA(Lys)(3) initiation complex using heteronuclear nuclear magnetic resonance methods. We show that both RNAs undergo large-scale conformational changes upon complex formation. Formation of the 18-bp primer helix with the 3' end of tRNA(Lys)(3) drives large conformational rearrangements of the tRNA at the 5' end while maintaining the anticodon loop for potential loop-loop interactions. HIV RNA forms an intramolecular helix adjacent to the intermolecular primer helix. This helix, which must be broken by reverse transcription, likely acts as a kinetic block to reverse transcription.
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>

            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Petrov, A., Kornberg, G., O'Leary, S., Tsai, A., Uemura, S., Puglisi, J. D.</span>
            <a href="" download><span id="title">Dynamics of the translational machinery</span></a>
            <span id="publisher"> CURRENT OPINION IN STRUCTURAL BIOLOGY, 2011; 21 (1): 137-145</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span>
                The recent growth in single molecule studies of translation has provided an insight into the molecular mechanism of ribosomal function. Single molecule fluorescence approaches allowed direct observation of the structural rearrangements occurring during translation and revealed dynamic motions of the ribosome and its ligands. These studies demonstrated how ligand binding affects dynamics of the ribosome, and the role of the conformational sampling in large-scale rearrangements intrinsic to translation elongation. The application of time-resolved cryo-electron microscopy revealed new conformational intermediates during back-translocation providing an insight into ribosomal dynamics from an alternative perspective. Recent developments permitted examination of conformational and compositional dynamics of the ribosome in real-time through multiple cycles of elongation at the single molecule level. The zero-mode waveguide approach allowed direct observation of the compositional dynamics of tRNA occupancy on the elongating ribosome. The emergence of single molecule in vivo techniques provided insights into the mechanism and regulation of translation at the organismal level.
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>

            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Uemura, S., Puglisi, J. D.</span>
            <a href="" download><span id="title">Real-time monitoring of single-molecule translation</span></a>
            <span id="publisher">RIBOSOMES: STRUCTURE, FUNCTION, AND DYNAMICS, 2011: 295-302</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span>
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>

            </div>
          </div>


          <div id="year">2010</div>
          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Petrov, A., Puglisi, J. D.</span>
            <a href="" download><span id="title">Site-specific labeling of Saccharomyces cerevisiae ribosomes for single-molecule manipulations</span></a>
            <span id="publisher">NUCLEIC ACIDS RESEARCH, 2010; 38 (13)</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span>
                Site-specific labeling of Escherichia coli ribosomes has allowed application of single-molecule fluorescence spectroscopy and force methods to probe the mechanism of translation. To apply these approaches to eukaryotic translation, eukaryotic ribosomes must be specifically labeled with fluorescent labels and molecular handles. Here, we describe preparation and labeling of the small and large yeast ribosomal subunits. Phylogenetically variable hairpin loops in ribosomal RNA are mutated to allow hybridization of oligonucleotides to mutant ribosomes. We demonstrate specific labeling of the ribosomal subunits, and their use in single-molecule fluorescence and force experiments.
                </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>

            </div>
          </div>


          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Aitken, C. E., Puglisi, J. D.</span>
            <a href="" download><span id="title">Following the intersubunit conformation of the ribosome during translation in real time</span></a>
            <span id="publisher">NATURE STRUCTURAL & MOLECULAR BIOLOGY, 2010; 17 (7): 793-U35</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span>
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>

            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Puglisi, J. D., Williamson, J. R.</span>
            <a href="" download><span id="title">Nucleic acids continue to surprise Editorial overview</span></a>
            <span id="publisher">CURRENT OPINION IN STRUCTURAL BIOLOGY, 2010; 20 (3): 259-261</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span>
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>

            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Uemura, S., Aitken, C. E., Korlach, J., Flusberg, B. A., Turner, S. W., Puglisi, J. D.</span>
            <a href="" download><span id="title">Real-time tRNA transit on single translating ribosomes at codon resolution</span></a>
            <span id="publisher">NATURE, 2010; 464 (7291): 1012-U73</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span>
                Translation by the ribosome occurs by a complex mechanism involving the coordinated interaction of multiple nucleic acid and protein ligands. Here we use zero-mode waveguides (ZMWs) and sophisticated detection instrumentation to allow real-time observation of translation at physiologically relevant micromolar ligand concentrations. Translation at each codon is monitored by stable binding of transfer RNAs (tRNAs)-labelled with distinct fluorophores-to translating ribosomes, which allows direct detection of the identity of tRNA molecules bound to the ribosome and therefore the underlying messenger RNA (mRNA) sequence. We observe the transit of tRNAs on single translating ribosomes and determine the number of tRNA molecules simultaneously bound to the ribosome, at each codon of an mRNA molecule. Our results show that ribosomes are only briefly occupied by two tRNA molecules and that release of deacylated tRNA from the exit (E) site is uncoupled from binding of aminoacyl-tRNA site (A-site) tRNA and occurs rapidly after translocation. The methods outlined here have broad application to the study of mRNA sequences, and the mechanism and regulation of translation.
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>

            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Bokoch, M. P., Zou, Y., Rasmussen, S. G., Liu, C. W., Nygaard, R., Rosenbaum, D. M., Fung, J. J., Choi, H., Thian, F. S., Kobilka, T. S., Puglisi, J. D., Weis, W. I., Pardo, L., Prosser, R. S., Mueller, L., Kobilka, B. K.</span>
            <a href="" download><span id="title">Ligand-specific regulation of the extracellular surface of a G-protein-coupled receptor</span></a>
            <span id="publisher">NATURE, 2010; 463 (7277): 108-U121</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span>
                G-protein-coupled receptors (GPCRs) are seven-transmembrane proteins that mediate most cellular responses to hormones and neurotransmitters. They are the largest group of therapeutic targets for a broad spectrum of diseases. Recent crystal structures of GPCRs have revealed structural conservation extending from the orthosteric ligand-binding site in the transmembrane core to the cytoplasmic G-protein-coupling domains. In contrast, the extracellular surface (ECS) of GPCRs is remarkably diverse and is therefore an ideal target for the discovery of subtype-selective drugs. However, little is known about the functional role of the ECS in receptor activation, or about conformational coupling of this surface to the native ligand-binding pocket. Here we use NMR spectroscopy to investigate ligand-specific conformational changes around a central structural feature in the ECS of the beta(2) adrenergic receptor: a salt bridge linking extracellular loops 2 and 3. Small-molecule drugs that bind within the transmembrane core and exhibit different efficacies towards G-protein activation (agonist, neutral antagonist and inverse agonist) also stabilize distinct conformations of the ECS. We thereby demonstrate conformational coupling between the ECS and the orthosteric binding site, showing that drugs targeting this diverse surface could function as allosteric modulators with high subtype selectivity. Moreover, these studies provide a new insight into the dynamic behaviour of GPCRs not addressable by static, inactive-state crystal structures.
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>

            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Aitken, C. E., Petrov, A., Puglisi, J. D.</span>
            <a href="" download><span id="title">Single Ribosome Dynamics and the Mechanism of Translation</span></a>
            <span id="publisher">ANNUAL REVIEW OF BIOPHYSICS VOL 39, 2010; 39: 491-513</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span>
                Our current understanding of the mechanism of translation is based on nearly fifty years of biochemical and biophysical studies. This mechanism, which requires the ribosome to manipulate tRNA and step repetitively along the mRNA, implies movement. High-resolution structures of the ribosome and its ligands have recently described translation in atomic detail, capturing the endpoints of large-scale rearrangements of the ribosome. Direct observation of the dynamic events that underlie the mechanism of translation is challenged by ensemble averaging in bulk solutions. Single-molecule methods, which eliminate these averaging effects, have emerged as powerful tools to probe the mechanism of translation. Single-molecule fluorescence experiments have described the dynamic motion of the ribosome and tRNA. Single-molecule force measurements have directly probed the forces stabilizing ribosomal complexes. Recent developments have allowed real-time observation of ribosome movement and dynamics during translation. This review covers the contributions of single-molecule studies to our understanding of the dynamic nature of translation.
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>
            </div>
          </div>

        <div id="year">2009</div>
          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Puglisi, J. D.</span>
            <a href="" download><span id="title">Resolving the Elegant Architecture of the Ribosome</span></a>
            <span id="publisher">MOLECULAR CELL, 2009; 36 (5): 720-723</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span>
                This year's Nobel Prize in Chemistry rewards Ada Yonath, Tom Steitz, and Venki Ramakrishnan for their groundbreaking structural studies on the ribosome.
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>

            </div>
          </div>


          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Elazar, M., Liu, M., Mckenna, S. A., Liu, P., Gehrig, E. A., Puglisi, J. D., Rossignol, J., Glenn, J. S.</span>
            <a href="" download><span id="title">
The Anti-Hepatitis C Agent Nitazoxanide Induces Phosphorylation of Eukaryotic Initiation Factor 2 alpha Via Protein Kinase Activated by Double-Stranded RNA Activation</span></a>
            <span id="publisher">GASTROENTEROLOGY, 2009; 137 (5): 1827-1835 </span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span>
                New therapies are needed to treat patients infected with hepatitis C virus (HCV), a major worldwide cause of chronic liver disease. Nitazoxanide (NTZ), originally used to treat cryptosporidiosis infection, recently was shown to have unexpected antiviral activity in the HCV replicon system and in chronically infected patients. A pilot clinical study suggested that NTZ can augment the antiviral effect of interferon (IFN), although the molecular basis for its effect was unknown.We analyzed the effects of NTZ on the regulation of eukaryotic initiation factor-2alpha (eIF2alpha) and its IFN-induced kinase, protein kinase activated by double-stranded RNA (PKR), in cells that support HCV RNA replication and in vitro biochemical assays.NTZ increased eIF2alpha phosphorylation, a modification known to mediate host cell antiviral defenses. The addition of IFN to cell cultures increased NTZ-induced eIF2alpha phosphorylation. NTZ also increased PKR phosphorylation. In vitro, NTZ promoted PKR autophosphorylation, a key step in activating PKR's kinase activity for eIF2alpha. Finally, NTZ-induced eIF2alpha phosphorylation was reduced in the presence of specific inhibitors of PKR autophosphorylation.An important mechanism of NTZ's action involves activation of PKR, a key kinase that regulates the cell's innate antiviral response. These observations could explain the clinical antiviral effect of NTZ. NTZ might represent a new class of small molecules capable of potentiating and recapitulating important antiviral effects of IFN.
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>

            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Shimoike, T., Mckenna, S. A., Lindhout, D. A., Puglisi, J. D.</span>
            <a href="" download><span id="title">Translational insensitivity to potent activation of PKR by HCV IRES RNA</span></a>
            <span id="publisher">ANTIVIRAL RESEARCH, 2009; 83 (3): 228-237</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span>
                Translation of hepatitis C virus (HCV) is initiated at an internal ribosome entry site (IRES) located at the 5'end of its RNA genome. The HCV IRES is highly structured and greater than 50% of its nucleotides form based-paired helices. We report here that the HCV IRES is an activator of PKR, an interferon-induced enzyme that participates in host cell defense against viral infection. Binding of HCV IRES RNA to PKR leads to a greatly increased (20-fold) rate and level (4.5-fold) of PKR autophosphorylation compared to previously studied dsRNA activators. We have mapped the domains in the IRES required for PKR activation to domains III-IV and demonstrate that the N-terminal double-stranded RNA binding domains of PKR bind to the IRES in a similar manner to other RNA activators. Addition of HCV IRES RNA inhibits cap-dependent translation in lysates via phosphorylation of PKR and eIF2alpha. However, HCV IRES-mediated translation is not inhibited by the phosphorylation of PKR and eIF2alpha. The results presented here suggest that hydrolysis of GTP by eIF2 is not an essential step in IRES-mediated translation. Thus, HCV can use structured RNAs to its advantage in translation, while avoiding the deleterious effects of PKR activation.
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>

            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Marshall, R. A., Aitken, C. E., Puglisi, J. D.</span>
            <a href="" download><span id="title">GTP Hydrolysis by IF2 Guides Progression of the Ribosome into Elongation</span></a>
            <span id="publisher">MOLECULAR CELL, 2009; 35 (1): 37-47</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span>
                Recent structural data have revealed two distinct conformations of the ribosome during initiation. We employed single-molecule fluorescence methods to probe the dynamic relation of these ribosomal conformations in real time. In the absence of initiation factors, the ribosome assembles in two distinct conformations. The initiation factors guide progression of the ribosome to the conformation that can enter the elongation cycle. In particular, IF2 both accelerates the rate of subunit joining and actively promotes the transition to the elongation-competent conformation. Blocking GTP hydrolysis by IF2 results in 70S complexes formed in the conformation unable to enter elongation. We observe that rapid GTP hydrolysis by IF2 drives the transition to the elongation-competent conformation, thus committing the ribosome to enter the elongation cycle.
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>

            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Liu, C. W., Alekseyev, V. Y., Allwardt, J. R., Bankovich, A. J., Cade-Menun, B. J., Davis, R. W., Du, L., Garcia, K. C., Herschlag, D., Khosla, C., Kraut, D. A., Li, Q., Null, B., Puglisi, J. D., Sigala, P. A., Stebbins, J. F., Varani, L.</span>
            <a href="" download><span id="title">THE DIVERSITY OF NUCLEAR MAGNETIC RESONANCE SPECTROSCOPY</span></a>
            <span id="publisher">NATO Advanced Study Institute on Biophysics and the Challenges of Emerging Threats, SPRINGER. 2009: 65–81</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span>
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>

            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Aitken, C. E., Marshall, R. A., Puglisi, J. D.</span>
            <a href="" download><span id="title">IMPROVED DYE STABILITY IN SINGLE-MOLECULE FLUORESCENCE EXPERIMENTS </span></a>
            <span id="publisher">NATO Advanced Study Institute on Biophysics and the Challenges of Emerging Threats, SPRINGER. 2009: 83–99</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span>
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>

            </div>
          </div>

        <div id="year">2008</div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Marshall, R. A., Dorywalska, M., Puglisi, J. D.</span>
            <a href="" download><span id="title">Irreversible chemical steps control intersubunit dynamics during translation</span></a>
            <span id="publisher">PROCEEDINGS OF THE NATIONAL ACADEMY OF SCIENCES OF THE UNITED STATES OF AMERICA, 2008; 105 (40): 15364-15369</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span>
                The ribosome, a two-subunit macromolecular machine, deciphers the genetic code and catalyzes peptide bond formation. Dynamic rotational movement between ribosomal subunits is likely required for efficient and accurate protein synthesis, but direct observation of intersubunit dynamics has been obscured by the repetitive, multistep nature of translation. Here, we report a collection of single-molecule fluorescence resonance energy transfer assays that reveal a ribosomal intersubunit conformational cycle in real time during initiation and the first round of elongation. After subunit joining and delivery of correct aminoacyl-tRNA to the ribosome, peptide bond formation results in a rapid conformational change, consistent with the counterclockwise rotation of the 30S subunit with respect to the 50S subunit implied by prior structural and biochemical studies. Subsequent binding of elongation factor G and GTP hydrolysis results in a clockwise rotation of the 30S subunit relative to the 50S subunit, preparing the ribosome for the next round of tRNA selection and peptide bond formation. The ribosome thus harnesses the free energy of irreversible peptidyl transfer and GTP hydrolysis to surmount activation barriers to large-scale conformational changes during translation. Intersubunit rotation is likely a requirement for the concerted movement of tRNA and mRNA substrates during translocation.
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>

            </div>
          </div>


          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Elazor, M., Liu, M., McKenna, S., Liu, P., Gehrig, E. A., Elfert, A., Puglisi, J., Rossignol, J., Glenn, J. S.</span>
            <a href="" download><span id="title">NITAZOXANIDE (NTZ) IS AN INDUCER EIF2A AND PKR PHOSPHORYLATION</span></a>
            <span id="publisher">59th Annual Meeting of the American-Association-for-the-Study-of-Liver-Diseases, WILEY-BLACKWELL. 2008: 1151A–1151A</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span>
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>

            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Uemura, S., Iizuka, R., Ueno, T., Shimizu, Y., Taguchi, H., Ueda, T., Puglisi, J. D., Funatsu, T.</span>
            <a href="" download><span id="title">Single-molecule imaging of full protein synthesis by immobilized ribosomes</span></a>
            <span id="publisher">NUCLEIC ACIDS RESEARCH, 2008; 36 (12)</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span>
                How folding of proteins is coupled to their synthesis remains poorly understood. Here, we apply single-molecule fluorescence imaging to full protein synthesis in vitro. Ribosomes were specifically immobilized onto glass surfaces and synthesis of green fluorescent protein (GFP) was achieved using modified commercial Protein Synthesis using Recombinant Elements that lacked ribosomes but contained purified factors and enzyme that are required for translation in Escherichia coli. Translation was monitored using a GFP mutant (F64L/S65T/F99S/M153T/V163A) that has a high fluorophore maturation rate and that contained the Secretion Monitor arrest sequence to prevent dissociation from the ribosome. Immobilized ribosomal subunits were labeled with Cy3 and GFP synthesis was measured by colocalization of GFP fluorescence with the ribosome position. The rate of appearance of colocalized ribosome GFP was equivalent to the rates of fluorescence appearance coupled with translation measured in bulk, and the ribosome-polypeptide complexes were stable for hours. The methods presented here are applicable to single-molecule investigation of translational initiation, elongation and cotranslational folding.
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>

            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Puglisi, J. D., Doudna, J. A.</span>
            <a href="" download><span id="title">Nucleic acids and their protein partners</span></a>
            <span id="publisher">CURRENT OPINION IN STRUCTURAL BIOLOGY, 2008; 18 (3): 279-281</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span>
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>

            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Aitken, C. E., Marshall, R. A., Puglisi, J. D.</span>
            <a href="" download><span id="title">An oxygen scavenging system for improvement of dye stability in single-molecule fluorescence experiments</span></a>
            <span id="publisher"> BIOPHYSICAL JOURNAL, 2008; 94 (5): 1826-1835</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span>
                The application of single-molecule fluorescence techniques to complex biological systems places demands on the performance of single fluorophores. We present an enzymatic oxygen scavenging system for improved dye stability in single-molecule experiments. We compared the previously described protocatechuic acid/protocatechuate-3,4-dioxygenase system to the currently employed glucose oxidase/catalase system. Under standardized conditions, we observed lower dissolved oxygen concentrations with the protocatechuic acid/protocatechuate-3,4-dioxygenase system. Furthermore, we observed increased initial lifetimes of single Cy3, Cy5, and Alexa488 fluorophores. We further tested the effects of chemical additives in this system. We found that biological reducing agents increase both the frequency and duration of blinking events of Cy5, an effect that scales with reducing potential. We observed increased stability of Cy3 and Alexa488 in the presence of the antioxidants ascorbic acid and n-propyl gallate. This new O(2)-scavenging system should have wide application for single-molecule fluorescence experiments.
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>

            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Marshall, R. A., Aitken, C. E., Dorywalska, M., Puglisi, J. D.</span>
            <a href="" download><span id="title">Translation at the single-molecule level</span></a>
            <span id="publisher">ANNUAL REVIEW OF BIOCHEMISTRY, 2008; 77: 177-203</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span>
                Decades of studies have established translation as a multistep, multicomponent process that requires intricate communication to achieve high levels of speed, accuracy, and regulation. A crucial next step in understanding translation is to reveal the functional significance of the large-scale motions implied by static ribosome structures. This requires determining the trajectories, timescales, forces, and biochemical signals that underlie these dynamic conformational changes. Single-molecule methods have emerged as important tools for the characterization of motion in complex systems, including translation. In this review, we chronicle the key discoveries in this nascent field, which have demonstrated the power and promise of single-molecule techniques in the study of translation.
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>

            </div>
          </div>


        <div id="year">2007</div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Puglisi, J. D.</span>
            <a href="" download><span id="title">Structural biology - The dance of domains</span></a>
            <span id="publisher">NATURE, 2007; 450 (7173): 1171-1172 </span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span>
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>

            </div>
          </div>


          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Gonzalez, R. L., Chu, S., Puglisi, J. D.</span>
            <a href="" download><span id="title">Thiostrepton inhibition of tRNA delivery to the ribosome</span></a>
            <span id="publisher">RNA-A PUBLICATION OF THE RNA SOCIETY, 2007; 13 (12): 2091-2097</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span>
                Ribosome-stimulated hydrolysis of guanosine-5'-triphosphate (GTP) by guanosine triphosphatase (GTPase) translation factors drives protein synthesis by the ribosome. Allosteric coupling of GTP hydrolysis by elongation factor Tu (EF-Tu) at the ribosomal GTPase center to messenger RNA (mRNA) codon:aminoacyl-transfer RNA (aa-tRNA) anticodon recognition at the ribosomal decoding site is essential for accurate and rapid aa-tRNA selection. Here we use single-molecule methods to investigate the mechanism of action of the antibiotic thiostrepton and show that the GTPase center of the ribosome has at least two discrete functions during aa-tRNA selection: binding of EF-Tu(GTP) and stimulation of GTP hydrolysis by the factor. We separate these two functions of the GTPase center and assign each to distinct, conserved structural regions of the ribosome. The data provide a specific model for the coupling between the decoding site and the GTPase center during aa-tRNA selection as well as a general mechanistic model for ribosome-stimulated GTP hydrolysis by GTPase translation factors.
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>

            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Lindhout, D. A., Mckenna, S. A., Aitken, C. E., Liu, C. W., Puglisi, J. D.</span>
            <a href="" download><span id="title">PKR: A NMR perspective</span></a>
            <span id="publisher">PROGRESS IN NUCLEAR MAGNETIC RESONANCE SPECTROSCOPY, 2007; 51 (3): 199-215</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span>
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>

            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Puglisi, E. V., Puglisi, J. D.</span>
            <a href="" download><span id="title">Probing the conformation of human tRNA(3)(Lys) in solution by NMR</span></a>
            <span id="publisher">FEBS LETTERS, 2007; 581 (27): 5307-5314</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span>
                Human tRNA(3)(Lys) acts as a primer for the reverse transcription of human immunodeficiency virus genomic RNA. To form an initiation complex with genomic RNA, tRNA(3)(Lys) must reorganize its secondary structure. To provide a starting point for mechanistic studies of the formation of the initiation complex, we here present solution NMR investigations of human tRNA(3)(Lys). We use a straightforward set of NMR experiments to show that tRNA(3)(Lys) adopts a standard transfer ribonucleic acid tertiary structure in solution, and that Mg(2+) is required for this folding. The results underscore the power of NMR to reveal rapidly the conformation of RNAs.
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>

            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Kim, H. D., Puglisi, J. D., Chu, S.</span>
            <a href="" download><span id="title">Fluctuations of transfer RNAs between classical and hybrid states</span></a>
            <span id="publisher">BIOPHYSICAL JOURNAL, 2007; 93 (10): 3575-3582</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span>
                Adjacent transfer RNAs (tRNAs) in the A- and P-sites of the ribosome are in dynamic equilibrium between two different conformations called classical and hybrid states before translocation. Here, we have used single-molecule fluorescence resonance energy transfer to study the effect of Mg(2+) on tRNA dynamics with and without an acetyl group on the A-site tRNA. When the A-site tRNA is not acetylated, tRNA dynamics do not depend on [Mg(2+)], indicating that the relative positions of the substrates for peptide-bond formation are not affected by Mg(2+). In sharp contrast, when the A-site tRNA is acetylated, Mg(2+) lengthens the lifetime of the classical state but does not change the lifetime of the hybrid state. Based on these findings, the classical state resembles a state with direct stabilization of tertiary structure by Mg(2+) ions whereas the hybrid state resembles a state with little Mg(2+)-assisted stabilization. The antibiotic viomycin, a translocation inhibitor, suppresses tRNA dynamics, suggesting that the enhanced fluctuations of tRNAs after peptide-bond formation drive spontaneous attempts at translocation by the ribosome.
              </div>
                <div id="webof-science"></div>
              <div id="pubmed"></div>

            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Alekseyev, V. Y., Liu, C. W., Cane, D. E., Puglisi, J. D., Khosla, C.</span>
            <a href="" download><span id="title">Solution structure and proposed domain-domain recognition interface of an acyl carrier protein domain from a modular polyketide synthase</span></a>
            <span id="publisher">PROTEIN SCIENCE, 2007; 16 (10): 2093-2107</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span>
                Polyketides are a medicinally important class of natural products. The architecture of modular polyketide synthases (PKSs), composed of multiple covalently linked domains grouped into modules, provides an attractive framework for engineering novel polyketide-producing assemblies. However, impaired domain-domain interactions can compromise the efficiency of engineered polyketide biosynthesis. To facilitate the study of these domain-domain interactions, we have used nuclear magnetic resonance (NMR) spectroscopy to determine the first solution structure of an acyl carrier protein (ACP) domain from a modular PKS, 6-deoxyerythronolide B synthase (DEBS). The tertiary fold of this 10-kD domain is a three-helical bundle; an additional short helix in the second loop also contributes to the core helical packing. Superposition of residues 14-94 of the ensemble on the mean structure yields an average atomic RMSD of 0.64 +/- 0.09 Angstrom for the backbone atoms (1.21 +/- 0.13 Angstrom for all non-hydrogen atoms). The three major helices superimpose with a backbone RMSD of 0.48 +/- 0.10 Angstrom (0.99 +/- 0.11 Angstrom for non-hydrogen atoms). Based on this solution structure, homology models were constructed for five other DEBS ACP domains. Comparison of their steric and electrostatic surfaces at the putative interaction interface (centered on helix II) suggests a model for protein-protein recognition of ACP domains, consistent with the previously observed specificity. Site-directed mutagenesis experiments indicate that two of the identified residues influence the specificity of ACP recognition.
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>

            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Mckenna, S. A., Lindhout, D. A., Shimoike, T., Aitken, C. E., Puglisi, J. D.</span>
            <a href="" download><span id="title">Viral dsRNA inhibitors prevent self-association and autophosphorylation of PKR </span></a>
            <span id="publisher">JOURNAL OF MOLECULAR BIOLOGY, 2007; 372 (1): 103-113</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span>
                Host response to viral RNA genomes and replication products represents an effective strategy to combat viral invasion. PKR is a Ser/Thr protein kinase that binds to double-stranded (ds)RNA, autophosphorylates its kinase domain, and subsequently phosphorylates eukaryotic initiation factor 2alpha (eIF2alpha). This results in attenuation of protein translation, preventing synthesis of necessary viral proteins. In certain DNA viruses, PKR function can be evaded by transcription of highly structured virus-encoded dsRNA inhibitors that bind to and inactivate PKR. We probe here the mechanism of PKR inhibition by two viral inhibitor RNAs, EBER(I) (from Epstein-Barr) and VA(I) (from human adenovirus). Native gel shift mobility assays and isothermal titration calorimetry experiments confirmed that the RNA-binding domains of PKR are sufficient and necessary for the interaction with dsRNA inhibitors. Both EBER(I) and VA(I) are effective inhibitors of PKR activation by preventing trans-autophosphorylation between two PKR molecules. The RNA inhibitors prevent self-association of PKR molecules, providing a mechanistic basis for kinase inhibition. A variety of approaches indicated that dsRNA inhibitors remain associated with PKR under activating conditions, as opposed to activator dsRNA molecules that dissociate due to reduced affinity for the phosphorylated form of PKR. Finally, we show using a HeLa cell extract system that inhibitors of PKR result in translational recovery by the protein synthesis machinery. These data indicate that inhibitory dsRNAs bind preferentially to the latent, dephosphorylated form of PKR and prevent dimerization that is required for trans-autophosphorylation.
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>

            </div>
          </div>



          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Lee, T., Blanchard, S. C., Kim, H. D., Puglisi, J. D., Chu, S.</span>
            <a href="" download><span id="title">The role of fluctuations in tRNA selection by the ribosome</span></a>
            <span id="publisher">PROCEEDINGS OF THE NATIONAL ACADEMY OF SCIENCES OF THE UNITED STATES OF AMERICA, 2007; 104 (34): 13661-13665</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span>
                The detailed mechanism of how the ribosome decodes protein sequence information with an abnormally high accuracy, after 40 years of study, remains elusive. A critical element in selecting correct transfer RNA (tRNA) transferring correct amino acid is "induced fit" between the ribosome and tRNA. By using single-molecule methods, the induced fit mechanism is shown to position favorably the correct tRNA after initial codon recognition. We provide evidence that this difference in positioning and thermal fluctuations constitutes the primary mechanism for the initial selection of tRNA. This work demonstrates thermal fluctuations playing a critical role in the substrate selection by an enzyme.
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>

            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Varani, L., Bankovich, A. J., Liu, C. W., Colf, L. A., Jones, L. L., Kranz, D. M., Puglisi, J. D., Garcia, K. C.</span>
            <a href="" download><span id="title">Solution mapping of T cell receptor docking footprints on peptide-MHC</span></a>
            <span id="publisher">PROCEEDINGS OF THE NATIONAL ACADEMY OF SCIENCES OF THE UNITED STATES OF AMERICA, 2007; 104 (32): 13080-13085</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span>
                T cell receptor (TCR) recognition of peptide-MHC (pMHC) is central to the cellular immune response. A large database of TCR-pMHC structures is needed to reveal general structural principles, such as whether the repertoire of TCR/MHC docking modes is dictated by a "recognition code" between conserved elements of the TCR and MHC genes. Although approximately 17 cocrystal structures of unique TCR-pMHC complexes have been determined, cocrystallization of soluble TCR and pMHC remains a major technical obstacle in the field. Here we demonstrate a strategy, based on NMR chemical shift mapping, that permits rapid and reliable analysis of the solution footprint made by a TCR when binding onto the pMHC surface. We mapped the 2C TCR binding interaction with its allogeneic ligand H-2Ld-QL9 and identified a group of NMR-shifted residues that delineated a clear surface of the MHC that we defined as the TCR footprint. We subsequently found that the docking footprint described by NMR shifts was highly accurate compared with a recently determined high-resolution crystal structure of the same complex. The same NMR footprint analysis was done on a high-affinity mutant of the TCR. The current work serves as a foundation to explore the molecular dynamics of pMHC complexes and to rapidly determine the footprints of many Ld-specific TCRs.
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>

            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Mckenna, S. A., Lindhout, D. A., Kim, I., Liu, C. W., Gelev, V. M., Wagner, G., Puglisi, J. D.</span>
            <a href="" download><span id="title">Molecular framework for the activation of RNA-dependent protein kinase</span></a>
            <span id="publisher">JOURNAL OF BIOLOGICAL CHEMISTRY, 2007; 282 (15): 11474-11486</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span>
                The RNA-dependent protein kinase (PKR) plays an integral role in the antiviral response to cellular infection. PKR contains three distinct domains consisting of two conserved N-terminal double-stranded RNA (dsRNA)-binding domains, a C-terminal Ser-Thr kinase domain, and a central 80-residue linker. Despite rich structural and biochemical data, a detailed mechanistic explanation of PKR activation remains unclear. Here we provide a framework for understanding dsRNA-dependent activation of PKR using nuclear magnetic resonance spectroscopy, dynamic light scattering, gel filtration, and autophosphorylation kinetics. In the latent state, PKR exists as an extended monomer, with an increase in self-affinity upon dsRNA association. Subsequent phosphorylation leads to efficient release of dsRNA followed by a greater increase in self-affinity. Activated PKR displays extensive conformational perturbations within the kinase domain. We propose an updated model for PKR activation in which the communication between RNA binding, central linker, and kinase domains is critical in the propagation of the activation signal and for PKR dimerization.
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>

            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Uemura, S., Dorywalska, M., Lee, T., Kim, H. D., Puglisi, J. D., Chu, S.</span>
            <a href="" download><span id="title">Peptide bond formation destabilizes Shine-Dalgarno interaction on the ribosome</span></a>
            <span id="publisher">NATURE, 2007; 446 (7134): 454-457</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span>
                The ribosome is a molecular machine that translates the genetic code contained in the messenger RNA into an amino acid sequence through repetitive cycles of transfer RNA selection, peptide bond formation and translocation. Here we demonstrate an optical tweezer assay to measure the rupture force between a single ribosome complex and mRNA. The rupture force was compared between ribosome complexes assembled on an mRNA with and without a strong Shine-Dalgarno (SD) sequence-a sequence found just upstream of the coding region of bacterial mRNAs, involved in translation initiation. The removal of the SD sequence significantly reduced the rupture force in complexes carrying an aminoacyl tRNA, Phe-tRNA(Phe), in the A site, indicating that the SD interactions contribute significantly to the stability of the ribosomal complex on the mRNA before peptide bond formation. In contrast, the presence of a peptidyl tRNA analogue, N-acetyl-Phe-tRNA(Phe), in the A site, which mimicked the post-peptidyl transfer state, weakened the rupture force as compared to the complex with Phe-tRNA(Phe), and the resultant force was the same for both the SD-containing and SD-deficient mRNAs. These results suggest that formation of the first peptide bond destabilizes the SD interaction, resulting in the weakening of the force with which the ribosome grips an mRNA. This might be an important requirement to facilitate movement of the ribosome along mRNA during the first translocation step.
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>

            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Kim, I., Mckenna, S. A., Puglisi, E. V., Puglisi, J. D.</span>
            <a href="" download><span id="title">Rapid purification of RNAs using fast performance liquid chromatography (FPLC)</span></a>
            <span id="publisher">RNA-A PUBLICATION OF THE RNA SOCIETY, 2007; 13 (2): 289-294</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span>
                We present here an improved RNA purification method using fast performance liquid chromatography (FPLC) size-exclusion chromatography in place of denaturing polyacrylamide gel electrophoresis (PAGE). The method allows preparation of milligram quantities of pure RNA in a single day. As RNA oligonucleotides behave differently from globular proteins in the size-exclusion column, we present standard curves for RNA oligonucleotides of different lengths on both the Superdex 75 column and the Superdex 200 size-exclusion column. Using this approach, we can separate monomer from multimeric RNA species, purify the desired RNA product from hammerhead ribozyme reactions, and isolate refolded RNA that has aggregated after long-term storage. This methodology allows simple and rapid purification of RNA oligonucleotides for structural and biophysical studies.
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>

            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Mckenna, S. A., Lindhout, D. A., Shimoike, T., Puglisi, J. D.</span>
            <a href="" download><span id="title">Biophysical and biochemical investigations of dsRNA-activated kinase PKR </span></a>
            <span id="publisher">TRANSLATION INITIATION: RECONSTITUTED SYSTEMS AND BIOPHYSICAL METHODS, 2007; 430: 373-396</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span>
                Protein kinase RNA-activated (PKR) is a serine/threonine kinase that contains an N-terminal RNA-binding domain (dsRNA) and a C-terminal kinase domain. On binding viral dsRNA molecules, PKR can become activated and phosphorylate cellular targets, such as eukaryotic translation initiation factor 2alpha (eIF-2alpha). Phosphorylation of eIF-2alpha results in attenuation of protein translation initiation. Therefore, PKR plays an integral role in the antiviral response to cellular infection. Here we provide a methodological framework for probing PKR function by use of assays for phosphorylation, RNA-protein stability, PKR dimerization, and in vitro translation. These methods are complemented by nuclear magnetic resonance approaches for probing structural features of PKR activation. Considerations required for both PKR and dsRNA sample preparation are also discussed.
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>

            </div>
          </div>



          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Uemura, S., Dorywalska, M., Lee, T., Kim, H. D., Puglisi, J. D., Chu, S.</span>
            <a href="" download><span id="title">Peptide bond formation destabilizes Shine-Dalgarno interaction on the ribosome</span></a>
            <span id="publisher">51st Annual Meeting of the Biophysical-Society, CELL PRESS. 2007: 571A–571A</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span>
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>

            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Mckenna, S. A., Lindhout, D. A., Aitken, C. E., Puglisi, J. D.</span>
            <a href="" download><span id="title">Molecular insights into PKR activation by viral double-stranded RNA</span></a>
            <span id="publisher">NATO Advanced Study Institute Course on Structure and Biophysics,SPRINGER. 2007: 99–110</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span>
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>

            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Mckenna, S. A., Kim, I., Puglisi, E. V., Lindhout, D. A., Aitken, C. E., Marshall, R. A., Puglisi, J. D.</span>
            <a href="" download><span id="title">Purification and characterization of transcribed RNAs using gel filtration chromatography</span></a>
            <span id="publisher">NATURE PROTOCOLS, 2007; 2 (12): 3270-3277</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span>
                RNA synthesis using in vitro transcription by phage T7 RNA polymerase allows preparation of milligram quantities of RNA for biochemical, biophysical and structural investigations. Previous purification approaches relied on gel electrophoretic or gravity-flow chromatography methods. We present here a protocol for the in vitro transcription of RNAs and subsequent purification using fast-performance liquid chromatography. This protocol greatly facilitates production of RNA in a single day from transcription to purification.
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>
            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Lindhout DA, McKenna SA, Aitken CE, Liu CW, Puglisi JD</span>
            <a href="" download><span id="title">PKR: A NMR perspective</span></a>
            <span id="publisher">JPNMRS, 2007; 51 (3): 199-215</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span>
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>
            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Marshall RA, and Puglisi JD</span>
            <a href="" download><span id="title">NMR structural studies of aminoglycoside:RNA interaction</span></a>
            <span id="publisher">Aminoglycoside Antibiotics, Edited by DP Arya, John Wiley & Sons, 2007: 181-207</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span>
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>

            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">McKenna SA, Lindhout DA, Aitken CE, Puglisi JD</span>
            <a href="" download><span id="title">Molecular Insights Into PKR Activation by Viral Double-Stranded RNA</span></a>
            <span id="publisher">NATO Science Series, Springer, 2007; Ch 8: 99-110</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span>
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>

            </div>
          </div>


          <div id="year">2006</div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Lee, T., Blanchard, S. C., Kim, H. D., Puglisi, J. D., Chu, S.</span>
            <a href="" download><span id="title">PHYS 336-Role of dynamics in the initial selection of tRNA by the ribosome</span></a>
            <span id="publisher">AMERICAN CHEMICAL SOCIETY, 2006 </span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span>
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>

            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Kim, I., Liu, C. W., Puglisi, J. D.</span>
            <a href="" download><span id="title">Specific recognition of HIV TAR RNA by the dsRNA binding domains (dsRBD1-dsRBD2) of PKR</span></a>
            <span id="publisher">JOURNAL OF MOLECULAR BIOLOGY, 2006; 358 (2): 430-442</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span>
                PKR (double-stranded RNA-dependent protein kinase) is an important component of host defense to virus infection. Binding of dsRNA to two dsRBDs (double-stranded RNA binding domains) of PKR modulates its own kinase activation. How structural features of natural target RNAs, such as bulges and loops, have an effect on the binding to two dsRBDs of PKR still remains unclear. By using ITC and NMR, we show here that both the bulge and loop of TAR RNA are necessary for the high affinity binding to dsRBD1-dsRBD2 of PKR with 1:1 stoichiometry. The binding site for the dsRBD1-dsRBD2 spans from upper bulge to lower stem of the TAR RNA, based on chemical shift mapping. The backbone resonances in the 40 kDa TAR.dsRBD1-dsRBD2 were assigned. NMR chemical shift perturbation data suggest that the beta1-beta2 loop of the dsRBD1 interacts with the TAR RNA, whereas that of the dsRBD2 is less involved in the TAR RNA recognition. In addition, the residues of the interdomain linker between the dsRBD1 and the dsRBD2 also show large chemical perturbations indicating that the linker is involved in the recognition of TAR RNA. The results presented here provide the biophysical and spectroscopic basis for high-resolution structural studies, and show how local RNA structural features modulate recognition by dsRBDs.
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>
            </div>
          </div>

        <div id="year">2005</div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Underwood, K. A., Swartz, J. R., Puglisi, J. D.</span>
            <a href="" download><span id="title">Quantitative polysome analysis identifies limitations in bacterial cell-free protein synthesis</span></a>
            <span id="publisher">BIOTECHNOLOGY AND BIOENGINEERING, 2005; 91 (4): 425-435</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span>
                Cell-free protein synthesis (CFPS) is becoming increasingly used for protein production as yields increase and costs decrease. CFPS optimization efforts have focused primarily on energy supply and small molecule metabolism, though little is known about the protein synthesis machinery or what limits protein synthesis rates. Here, quantitative polysome profile analysis was used to characterize cell-free translation, thereby elucidating many kinetic parameters. The ribosome concentration in Escherichia coli-based CFPS reactions was 1.6 +/- 0.1 microM, with 72 +/- 4% actively translating at maximal protein synthesis rate. A translation elongation rate of 1.5 +/- 0.2 amino acids per second per ribosome and an initiation rate of 8.2 x 10(-9) +/- 0.3 x 10(-9) M/s, which correlates to, on average, one initiation every 60 +/- 9 s per mRNA, were determined. The measured CFPS initiation and elongation rates are an order of magnitude lower than the in vivo rates and further analysis identified elongation as the major limitation. Adding purified elongation factors (EFs) to CFPS reactions increased the ribosome elongation rate and protein synthesis rates and yields, as well as the translation initiation rate, indicating a possible coupling between initiation and elongation. Further examination of translation initiation in the cell-free system showed that the first initiation on an mRNA is slower than subsequent initiations. Our results demonstrate that polysome analysis is a valid tool to characterize cell-free translation and to identify limiting steps, that dilution of translation factors is a limitation of CFPS, and that CFPS is a useful platform for making novel observations about translation.
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>

            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Kim I, Lukavsky PJ, Otto GA, Liu CW, Puglisi JD</span>
            <a href="" download><span id="title">Using NMR to Study large RNAs: Case Study of the HCV IRES</span></a>
            <span id="publisher">NATO ASI Science Series, Ios Press, 2005; 364: 75-90</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span>
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>

            </div>
          </div>



          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Dorywalska, M., Blanchard, S. C., Gonzalez, R. L., Kim, H. D., Chu, S., Puglisi, J. D.</span>
            <a href="" download><span id="title">Site-specific labeling of the ribosome for single-molecule spectroscopy</span></a>
            <span id="publisher">NUCLEIC ACIDS RESEARCH, 2005; 33 (1): 182-189</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span>
                Single-molecule fluorescence spectroscopy can reveal mechanistic and kinetic details that may not be observed in static structural and bulk biochemical studies of protein synthesis. One approach requires site-specific and stable attachment of fluorophores to the components of translation machinery. Fluorescent tagging of the ribosome is a prerequisite for the observation of dynamic changes in ribosomal conformation during translation using fluorescence methods. Modifications of the ribosomal particle are difficult due to its complexity and high degree of sequence and structural conservation. We have developed a general method to label specifically the prokaryotic ribosome by hybridization of fluorescent oligonucleotides to mutated ribosomal RNA. Functional, modified ribosomes can be purified as a homogenous population, and fluorescence can be monitored from labeled ribosomal complexes immobilized on a derivatized quartz surface.
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>

            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Lukavsky, P. J., Puglisi, J. D.</span>
            <a href="" download><span id="title">Structure determination of large biological RNAs</span></a>
            <span id="publisher">NUCLEAR MAGNETIC RESONANCE OF BIOLOGICAL MACROMOLECULES, PART C, 2005; 394: 399-416</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span>
                Complex RNA structures regulate many biological processes but are often too large for structure determination by nuclear magnetic resonance (NMR) methods. We determined the solution structure of domain II of the hepatitis C viral internal ribosome entry site (HCV IRES), a 25-kDa RNA, using a novel NMR approach. Conventional short-range, distance, and torsion angle NMR restraints were combined with long-range, angular restraints derived from residual dipolar couplings (RDCs) to improve both the local and global precision of the structure. This powerful approach should be generally applicable to the NMR structure determination of large, modular RNAs.
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>
            </div>
          </div>

          <div id="year">2004</div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Otto, G. A., Puglisi, J. D.</span>
            <a href="" download><span id="title">The pathway of HCVIRES-mediated translation initiation</span></a>
            <span id="publisher">CELL, 2004; 119 (3): 369-380</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span>
                The HCV internal ribosome entry site (IRES) directly regulates the assembly of translation initiation complexes on viral mRNA by a sequential pathway that is distinct from canonical eukaryotic initiation. The HCV IRES can form a binary complex with an eIF-free 40S ribosomal subunit. Next, a 48S-like complex assembles at the AUG initiation codon upon association of eIF3 and ternary complex. 80S complex formation is rate limiting and follows the GTP-dependent association of the 60S subunit. Efficient assembly of the 48S-like and 80S complexes on the IRES mRNA is dependent upon maintenance of the highly conserved HCV IRES structure. This revised model of HCV IRES translation initiation provides a context to understand the function of different HCV IRES domains during translation initiation.
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>

            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Blanchard, S. C., Gonzalez, R. L., Kim, H. D., Chu, S., Puglisi, J. D.</span>
            <a href="" download><span id="title">tRNA selection and kinetic proofreading in translation</span></a>
            <span id="publisher">NATURE STRUCTURAL & MOLECULAR BIOLOGY, 2004; 11 (10): 1008-1014</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span>
                Using single-molecule methods we observed the stepwise movement of aminoacyl-tRNA (aa-tRNA) into the ribosome during selection and kinetic proofreading using single-molecule fluorescence resonance energy transfer (smFRET). Intermediate states in the pathway of tRNA delivery were observed using antibiotics and nonhydrolyzable GTP analogs. We identified three unambiguous FRET states corresponding to initial codon recognition, GTPase-activated and fully accommodated states. The antibiotic tetracycline blocks progression of aa-tRNA from the initial codon recognition state, whereas cleavage of the sarcin-ricin loop impedes progression from the GTPase-activated state. Our data support a model in which ribosomal recognition of correct codon-anticodon pairs drives rotational movement of the incoming complex of EF-Tu-GTP-aa-tRNA toward peptidyl-tRNA during selection on the ribosome. We propose a mechanistic model of initial selection and proofreading.
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>

            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author"></span>
            <a href="" download><span id="title"></span></a>
            <span id="publisher">, </span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span>
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>

            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author"></span>
            <a href="" download><span id="title"></span></a>
            <span id="publisher">, </span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span>
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>

            </div>
          </div>



          <div id="publication-wrapper">
          <div id="publication">
            <span id="author"></span>
            <a href="" download><span id="title"></span></a>
            <span id="publisher">, </span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span>
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>

            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author"></span>
            <a href="" download><span id="title"></span></a>
            <span id="publisher">, </span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span>
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>

            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author"></span>
            <a href="" download><span id="title"></span></a>
            <span id="publisher">, </span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span>
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>

            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author"></span>
            <a href="" download><span id="title"></span></a>
            <span id="publisher">, </span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span>
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>

            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author"></span>
            <a href="" download><span id="title"></span></a>
            <span id="publisher">, </span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span>
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>

            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author"></span>
            <a href="" download><span id="title"></span></a>
            <span id="publisher">, </span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span>
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>

            </div>
          </div>


















      </div>
    </div>
      <div id="designed-by">
        Design:
          <a href="https://github.com/lmadigan" target="_blank"><i class="fa fa-github" aria-hidden="true"></i></a>
					<a href="https://www.linkedin.com/in/laurenmadigan/" class="fa fa-linkedin" target="_blank"></a>
      </div>
    </div>

    <script>
      var el = $('.header');
        var $document = $(document);
        $document.scroll(function() {
          if ($document.scrollTop() >= 230) {
            // user scrolled 50 pixels or more;
            // do stuff
            el.addClass('fix-search').slideDown();
          } else {
            el.removeClass("fix-search");
          }
        });

        $(document).ready(function () {

          $(document).on("click", "div#more-title", function () {
              // $(this).next("div#more").toggleClass("hidden");
              $(this).next("div#more").slideToggle();
          });

          $(document).on("click", "div#more-title", function () {
              $(this).find("i#triangle").toggleClass("rotated");
          });


      });
			</script>
  </body>
</html>
