<!DOCTYPE html>
<html>
  <head>
    <meta charset="utf-8">
    <title>Puglisi Lab</title>
    <link href="https://fonts.googleapis.com/css?family=Open+Sans+Condensed:300" rel="stylesheet">
    <link href="https://fonts.googleapis.com/css?family=Arimo" rel="stylesheet">
    <script src="https://use.fontawesome.com/7ccc97b483.js"></script>
    <script type="text/javascript" src="jquery-3.2.0.min.js"></script>
    <!-- <link rel="stylesheet" href="assets/reset.css" /> -->
    <link rel="stylesheet" href="assets/main.css" />
    <!-- <link rel="stylesheet" href="assets/publications.css" /> -->
  </head>
  <body>
    <div id="page-wrapper">

      <header class="header">
        <h1 id="logo"><a href="index.html">
          <div id="logo-img">
          <img src="images/Logo_of_Stanford_University.png" height="50" width="50"></div>
          <div id="puglisi">Puglisi</div>
          <div id="laboratory">Lab</div></a></h1>
        <nav id="nav">
          <a href="research.html"><ul id="nav-title">Research</ul></a>
          <a href="people.html"><ul id="nav-title">People</ul></a>
          <a href="publications.html"><ul id="nav-title">Publications</ul></a>
          <a href="abroad.html"><ul id="nav-title">Abroad</ul></a>
          <a href="contact.html"><ul id="nav-title">Contact</ul></a>
        </nav>
      </header>
      <div id="background-photo">
        <div class="welcome 1">Our Publications</div></br>
        <!-- <h3 id="history">Let's make history</h3> -->
      </div>

      <div id="publications-wrapper">
        <div id="year-wrapper">

          <div id="year">2016</div>
          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Coey, A., Larsen, K., Puglisi, J. D., Puglisi, E. V.</span>
            <a href="" download><span id="title">Heterogeneous structures formed by conserved RNA sequences within the HIV reverse transcription initiation site</span></a>
            <span id="publisher">RNA, 2016; 22 (11): 1689-1698</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span>
Reverse transcription is a key process in the early steps of HIV infection. This process initiates within a specific complex formed by the 5' UTR of the HIV genomic RNA (vRNA) and a host primer tRNA(Lys)3 Using nuclear magnetic resonance (NMR) spectroscopy and single-molecule fluorescence spectroscopy, we detect two distinct conformers adopted by the tRNA/vRNA initiation complex. We directly show that an interaction between the conserved 8-nucleotide viral RNA primer activation signal (PAS) and the primer tRNA occurs in one of these conformers. This intermolecular PAS interaction likely induces strain on a vRNA intramolecular helix, which must be broken for reverse transcription to initiate. We propose a mechanism by which this vRNA/tRNA conformer relieves the kinetic block formed by the vRNA intramolecular helix to initiate reverse transcription.
              <div id="webof-science"></div>
              <div id="pubmed"></div>
              </div>

            </div>
          </div>



          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Navon, S. P., Kornberg, G., Chen, J., Schwartzman, T., Tsai, A., Puglisi, E. V., Puglisi, J. D., Adir, N.</span>
            <a href="" download><span id="title">Amino acid sequence repertoire of the bacterial proteome and the occurrence of untranslatable sequences</span></a>
            <span id="publisher"> PROCEEDINGS OF THE NATIONAL ACADEMY OF SCIENCES OF THE UNITED STATES OF AMERICA, 2016; 113 (26): 7166-7170</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span>
                Bioinformatic analysis of Escherichia coli proteomes revealed that all possible amino acid triplet sequences occur at their expected frequencies, with four exceptions. Two of the four underrepresented sequences (URSs) were shown to interfere with translation in vivo and in vitro. Enlarging the URS by a single amino acid resulted in increased translational inhibition. Single-molecule methods revealed stalling of translation at the entrance of the peptide exit tunnel of the ribosome, adjacent to ribosomal nucleotides A2062 and U2585. Interaction with these same ribosomal residues is involved in regulation of translation by longer, naturally occurring protein sequences. The E. coli exit tunnel has evidently evolved to minimize interaction with the exit tunnel and maximize the sequence diversity of the proteome, although allowing some interactions for regulatory purposes. Bioinformatic analysis of the human proteome revealed no underrepresented triplet sequences, possibly reflecting an absence of regulation by interaction with the exit tunnel.
                <div id="webof-science"></div>
                <div id="pubmed"></div>

              </div>

            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Chen, J., Choi, J., O'Leary, S. E., Prabhakar, A., Petrov, A., Grosely, R., Puglisi, E. V., Puglisi, J. D.</span>
            <a href="" download><span id="title">The molecular choreography of protein synthesis: translational control, regulation, and pathways </span></a>
            <span id="publisher">QUARTERLY REVIEWS OF BIOPHYSICS, 2016; 49</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span>

              <div id="webof-science"></div>
              <div id="pubmed"></div>
              </div>
            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Petrov, A., Grosely, R., Chen, J., O'Leary, S. E., Puglisi, J. D.</span>
            <a href="" download><span id="title">Multiple Parallel Pathways of Translation Initiation on the CrPV IRES.</span></a>
            <span id="publisher">Molecular cell, 2016; 62 (1): 92-103</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span>
                The complexity of eukaryotic translation allows fine-tuned regulation of protein synthesis. Viruses use internal ribosome entry sites (IRESs) to minimize or, like the CrPV IRES, eliminate the need for initiation factors. Here, by exploiting the CrPV IRES, we observed the entire process of initiation and transition to elongation in real time. We directly tracked the CrPV IRES, 40S and 60S ribosomal subunits, and tRNA using single-molecule fluorescence spectroscopy and identified multiple parallel initiation pathways within the system. Our results distinguished two pathways of 80S:CrPV IRES complex assembly that produce elongation-competent complexes. Following 80S assembly, the requisite eEF2-mediated translocation results in an unstable intermediate that is captured by binding of the elongator tRNA. Whereas initiation can occur in the 0 and +1 frames, the arrival of the first tRNA defines the reading frame and strongly favors 0 frame initiation. Overall, even in the simplest system, an intricate reaction network regulates translation initiation.

              <div id="webof-science"></div>
              <div id="pubmed"></div>
              </div>
            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Choi, J., Ieong, K., Demirci, H., Chen, J., Petrov, A., Prabhakar, A., O'Leary, S. E., Dominissini, D., Rechavi, G., Soltis, S. M., Ehrenberg, M., Puglisi, J. D.</span>
            <a href="" download><span id="title">N(6)-methyladenosine in mRNA disrupts tRNA selection and translation-elongation dynamics.</span></a>
            <span id="publisher">Nature structural & molecular biology, 2016; 23 (2): 110-115</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span>
N(6)-methylation of adenosine (forming m(6)A) is the most abundant post-transcriptional modification within the coding region of mRNA, but its role during translation remains unknown. Here, we used bulk kinetic and single-molecule methods to probe the effect of m(6)A in mRNA decoding. Although m(6)A base-pairs with uridine during decoding, as shown by X-ray crystallographic analyses of Thermus thermophilus ribosomal complexes, our measurements in an Escherichia coli translation system revealed that m(6)A modification of mRNA acts as a barrier to tRNA accommodation and translation elongation. The interaction between an m(6)A-modified codon and cognate tRNA echoes the interaction between a near-cognate codon and tRNA, because delay in tRNA accommodation depends on the position and context of m(6)A within codons and on the accuracy level of translation. Overall, our results demonstrate that chemical modification of mRNA can change translational dynamics.
              <div id="webof-science"></div>
              <div id="pubmed"></div>
              </div>
            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Tsai, A., Puglisi, J. D., Uemura, S.</span>
            <a href="" download><span id="title">Probing the Translation Dynamics of Ribosomes Using Zero-Mode Waveguides. </span></a>
            <span id="publisher">Progress in molecular biology and translational science, 2016; 139: 1-43</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span>
                In order to coordinate the complex biochemical and structural feat of converting triple-nucleotide codons into their corresponding amino acids, the ribosome must physically manipulate numerous macromolecules including the mRNA, tRNAs, and numerous translation factors. The ribosome choreographs binding, dissociation, physical movements, and structural rearrangements so that they synergistically harness the energy from biochemical processes, including numerous GTP hydrolysis steps and peptide bond formation. Due to the dynamic and complex nature of translation, the large cast of ligands involved, and the large number of possible configurations, tracking the global time evolution or dynamics of the ribosome complex in translation has proven to be challenging for bulk methods. Conventional single-molecule fluorescence experiments on the other hand require low concentrations of fluorescent ligands to reduce background noise. The significantly reduced bimolecular association rates under those conditions limit the number of steps that can be observed within the time window available to a fluorophore. The advent of zero-mode waveguide (ZMW) technology has allowed the study of translation at near-physiological concentrations of labeled ligands, moving single-molecule fluorescence microscopy beyond focused model systems into studying the global dynamics of translation in realistic setups. This chapter reviews the recent works using the ZMW technology to dissect the mechanism of translation initiation and elongation in prokaryotes, including complex processes such as translational stalling and frameshifting. Given the success of the technology, similarly complex biological processes could be studied in near-physiological conditions with the controllability of conventional in vitro experiments.

              <div id="webof-science"></div>
              <div id="pubmed"></div>
              </div>
            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Siprashvili, Z., Webster, D. E., Johnston, D., Shenoy, R. M., Ungewickell, A. J., Bhaduri, A., Flockhart, R., Zarnegar, B. J., Che, Y., Meschi, F., Puglisi, J. D., Khavari, P. A.</span>
            <a href="" download><span id="title">The noncoding RNAs SNORD50A and SNORD50B bind K-Ras and are recurrently deleted in human cancer.</span></a>
            <span id="publisher"> Nature genetics, 2016; 48 (1): 53-58</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span>
                Small nucleolar RNAs (snoRNAs) are conserved noncoding RNAs best studied as ribonucleoprotein (RNP) guides in RNA modification. To explore their role in cancer, we compared 5,473 tumor-normal genome pairs to identify snoRNAs with frequent copy number loss. The SNORD50A-SNORD50B snoRNA locus was deleted in 10-40% of 12 common cancers, where its loss was associated with reduced survival. A human protein microarray screen identified direct SNORD50A and SNORD50B RNA binding to K-Ras. Loss of SNORD50A and SNORD50B increased the amount of GTP-bound, active K-Ras and hyperactivated Ras-ERK1/ERK2 signaling. Loss of these snoRNAs also increased binding by farnesyltransferase to K-Ras and increased K-Ras prenylation, suggesting that KRAS mutation might synergize with SNORD50A and SNORD50B loss in cancer. In agreement with this hypothesis, CRISPR-mediated deletion of SNORD50A and SNORD50B in KRAS-mutant tumor cells enhanced tumorigenesis, and SNORD50A and SNORD50B deletion and oncogenic KRAS mutation co-occurred significantly in multiple human tumor types. SNORD50A and SNORD50B snoRNAs thus directly bind and inhibit K-Ras and are recurrently deleted in human cancer.
              <div id="webof-science"></div>
              <div id="pubmed"></div>
              </div>
            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Sierra, R. G., Gati, C., Laksmono, H., Dao, E. H., Gul, S., Fuller, F., Kern, J., Chatterjee, R., Ibrahim, M., Brewster, A. S., Young, I. D., Michels-Clark, T., Aquila, A., Liang, M., Hunter, M. S., Koglin, J. E., Boutet, S., Junco, E. A., Hayes, B., Bogan, M. J., Hampton, C. Y., Puglisi, E. V., Sauter, N. K., Stan, C. A., Zouni, A., Yano, J., Yachandra, V. K., Soltis, S. M., Puglisi, J. D., DeMirci, H.</span>
            <a href="" download><span id="title">Concentric-flow electrokinetic injector enables serial crystallography of ribosome and photosystem II</span></a>
            <span id="publisher">NATURE METHODS,2016; 13 (1): 59-?</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span>
                We describe a concentric-flow electrokinetic injector for efficiently delivering microcrystals for serial femtosecond X-ray crystallography analysis that enables studies of challenging biological systems in their unadulterated mother liquor. We used the injector to analyze microcrystals of Geobacillus stearothermophilus thermolysin (2.2-Å structure), Thermosynechococcus elongatus photosystem II (<3-Å diffraction) and Thermus thermophilus small ribosomal subunit bound to the antibiotic paromomycin at ambient temperature (3.4-Å structure).
              <div id="webof-science"></div>
              <div id="pubmed"></div>
              </div>
            </div>
          </div>

          <div id="year">2015</div>
          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Chen, J., Coakley, A., O'Connor, M., Petrov, A., O'Leary, S. E., Atkins, J. F., Puglisi, J. D.</span>
            <a href="" download><span id="title">Coupling of mRNA Structure Rearrangement to Ribosome Movement during Bypassing of Non-coding Regions</span></a>
            <span id="publisher">Cell, 2015; 163 (5): 1267-1280</span></div>
            <div id="more-title"><i id="triangle" class="fa fa-caret-right" aria-hidden="true"></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span>
                Nearly half of the ribosomes translating a particular bacteriophage T4 mRNA bypass a region of 50 nt, resuming translation 3′ of this gap. How this large-scale, specific hop occurs and what determines whether a ribosome bypasses remain unclear. We apply single-molecule fluorescence with zero-mode waveguides to track individual Escherichia coli ribosomes during translation of T4’s gene 60 mRNA. Ribosomes that bypass are characterized by a 10- to 20-fold longer pause in a non-canonical rotated state at the take-off codon. During the pause, mRNA secondary structure rearrangements are coupled to ribosome forward movement, facilitated by nascent peptide interactions that disengage the ribosome anticodon-codon interactions for slippage. Close to the landing site, the ribosome then scans mRNA in search of optimal base-pairing interactions. Our results provide a mechanistic and conformational framework for bypassing, highlighting a non-canonical ribosomal state to allow for mRNA structure refolding to drive large-scale ribosome movements.
                </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>

            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Nilsson, O. B., Hedman, R., Marino, J., Wickles, S., Bischoff, L., Johansson, M., Mueller-Lucks, A., Trovato, F., Puglisi, J. D., O'Brien, E. P., Beckmann, R., Von Heijne, G.</span>
            <a href="" download><span id="title">Cotranslational Protein Folding inside the Ribosome Exit Tunnel</span></a>
            <span id="publisher">Cell Reports, 2015; 12 (10): 1533-1540</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span>
                At what point during translation do proteins fold? It is well established that proteins can fold cotranslationally outside the ribosome exit tunnel, whereas studies of folding inside the exit tunnel have so far detected only the formation of helical secondary structure and collapsed or partially structured folding intermediates. Here, using a combination of cotranslational nascent chain force measurements, inter-subunit fluorescence resonance energy transfer studies on single translating ribosomes, molecular dynamics simulations, and cryoelectron microscopy, we show that a small zinc-finger domain protein can fold deep inside the vestibule of the ribosome exit tunnel. Thus, for small protein domains, the ribosome itself can provide the kind of sheltered folding environment that chaperones provide for larger proteins.
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>

            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Puglisi, J. D.</span>
            <a href="" download><span id="title">SYNTHETIC BIOLOGY Ribosomal ties that bind</span></a>
            <span id="publisher">Nature, 2015; 524 (7563): 45-46</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span>
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>

            </div>
          </div>


          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Puglisi, J. D.</span>
            <a href="" download><span id="title">Protein synthesis. The delicate dance of translation and folding.</span></a>
            <span id="publisher">Science, 2015; 348 (6233): 399-400</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span>

The past decade has seen rapid growth in our knowledge of how proteins are synthesized in cells. This includes the basic step of how transfer RNAs (tRNAs) decode messenger RNAs (mRNAs) with high fidelity and speed, how the ribosome moves along mRNA from codon to codon during translation, and how synthesis of the corresponding polypeptide chain is initiated and terminated at specific points on the mRNA (1). Structures of the ribosomal particles—megadalton RNA-protein assemblies—have provided detailed molecular views of the active sites for mRNA decoding and peptide bond formation, and suggested pathways for movement of ligands, factors, and the ribosomal subunits themselves. Obscured in this rich cache of knowledge is the fate of the protein product. How does it fold during mRNA translation and how might protein folding affect translation itself? These questions are addressed by elegant biophysical and biochemical approaches reported by Goldman et al. (2) and Kim et al. (3) on pages 457 and 444, respectively, of this issue, adding to a growing appreciation of cotranslational protein folding (4–6).
                </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>

            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Puglisi, J. D.</span>
            <a href="" download><span id="title">RNA dances to center stage</span></a>
            <span id="publisher">RNA - A Publication of the RNA Society,2015; 21 (4): 712-713 </span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span>
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>

            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Capece, M. C., Kornberg, G. L., Petrov, A., Puglisi, J. D.</span>
            <a href="" download><span id="title">A simple real-time assay for in vitro translation</span></a>
            <span id="publisher">RNA, 2015; 21 (2): 296-305 </span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span>
                A high-throughput assay for real-time measurement of translation rates in cell-free protein synthesis (SNAP assay) is described. The SNAP assay enables quantitative, real-time measurement of overall translation rates in vitro via the synthesis of O(6)-alkylguanine DNA O(6)-alkyltransferase (SNAP). SNAP production is continuously detected by fluorescence produced by the reaction of SNAP with a range of quenched fluorogenic substrates. The capabilities of the assay are exemplified by measurements of the activities of Escherichia coli MRE600 ribosomes and fluorescently labeled E. coli mutant ribosomes in the PURExpress translation system and by determination of the 50% inhibitory concentrations (IC50) of three common macrolide antibiotics.
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>

            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Fuchs, G., Petrov, A. N., Marceau, C. D., Popov, L. M., Chen, J., O'Leary, S. E., Wang, R., Carette, J. E., Sarnow, P., Puglisi, J. D.</span>
            <a href="" download><span id="title">Kinetic pathway of 40S ribosomal subunit recruitment to hepatitis C virus internal ribosome entry site</span></a>
            <span id="publisher">Proceedings of the National Academy of Sciences of the United States of America, 2015; 112 (2): 319-325</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span>
                Translation initiation can occur by multiple pathways. To delineate these pathways by single-molecule methods, fluorescently labeled ribosomal subunits are required. Here, we labeled human 40S ribosomal subunits with a fluorescent SNAP-tag at ribosomal protein eS25 (RPS25). The resulting ribosomal subunits could be specifically labeled in living cells and in vitro. Using single-molecule Förster resonance energy transfer (FRET) between RPS25 and domain II of the hepatitis C virus (HCV) internal ribosome entry site (IRES), we measured the rates of 40S subunit arrival to the HCV IRES. Our data support a single-step model of HCV IRES recruitment to 40S subunits, irreversible on the initiation time scale. We furthermore demonstrated that after binding, the 40S:HCV IRES complex is conformationally dynamic, undergoing slow large-scale rearrangements. Addition of translation extracts suppresses these fluctuations, funneling the complex into a single conformation on the 80S assembly pathway. These findings show that 40S:HCV IRES complex formation is accompanied by dynamic conformational rearrangements that may be modulated by initiation factors.
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>

            </div>
          </div>


          <div id="year">2014</div>
          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Noriega, T. R., Chen, J., Walter, P., Puglisi, J. D.
</span>
            <a href="" download><span id="title">Real-time observation of signal recognition particle binding to actively translating ribosomes</span></a>
            <span id="publisher">Elife, 2014; 3 </span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span>
                The signal recognition particle (SRP) directs translating ribosome-nascent chain complexes (RNCs) that display a signal sequence to protein translocation channels in target membranes. All previous work on the initial step of the targeting reaction, when SRP binds to RNCs, used stalled and non-translating RNCs. This meant that an important dimension of the co-translational process remained unstudied. We apply single-molecule fluorescence measurements to observe directly and in real-time E. coli SRP binding to actively translating RNCs. We show at physiologically relevant SRP concentrations that SRP-RNC association and dissociation rates depend on nascent chain length and the exposure of a functional signal sequence outside the ribosome. Our results resolve a long-standing question: how can a limited, sub-stoichiometric pool of cellular SRP effectively distinguish RNCs displaying a signal sequence from those that are not? The answer is strikingly simple: as originally proposed, SRP only stably engages translating RNCs exposing a functional signal sequence.
                </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>

            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Chen, J., Petrov, A., Johansson, M., Tsai, A., O'Leary, S. E., Puglisi, J. D.</span>
            <a href="https://www.dropbox.com/sh/kfqdilhf8jms13n/AABMIgDaUKlB2uvsDKL8pdsJa?dl=1&lst=&preview=Chen_nature_2014.pdf" download><span id="title">Dynamic pathways of -1 translational frameshifting.</span></a>
            <span id="publisher">Nature, 2014; 512 (7514): 328-332</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span>
                Spontaneous changes in the reading frame of translation are rare (frequency of 10(-3) to 10(-4) per codon), but can be induced by specific features in the messenger RNA (mRNA). In the presence of mRNA secondary structures, a heptanucleotide 'slippery sequence' usually defined by the motif X XXY YYZ, and (in some prokaryotic cases) mRNA sequences that base pair with the 3' end of the 16S ribosomal rRNA (internal Shine-Dalgarno sequences), there is an increased probability that a specific programmed change of frame occurs, wherein the ribosome shifts one nucleotide backwards into an overlapping reading frame (-1 frame) and continues by translating a new sequence of amino acids. Despite extensive biochemical and genetic studies, there is no clear mechanistic description for frameshifting. Here we apply single-molecule fluorescence to track the compositional and conformational dynamics of individual ribosomes at each codon during translation of a frameshift-inducing mRNA from the dnaX gene in Escherichia coli. Ribosomes that frameshift into the -1 frame are characterized by a tenfold longer pause in elongation compared to non-frameshifted ribosomes, which translate through unperturbed. During the pause, interactions of the ribosome with the mRNA stimulatory elements uncouple EF-G catalysed translocation from normal ribosomal subunit reverse-rotation, leaving the ribosome in a non-canonical intersubunit rotated state with an exposed codon in the aminoacyl-tRNA site (A site). tRNA(Lys) sampling and accommodation to the empty A site and EF-G action either leads to the slippage of the tRNAs into the -1 frame or maintains the ribosome into the 0 frame. Our results provide a general mechanistic and conformational framework for -1 frameshifting, highlighting multiple kinetic branchpoints during elongation.
                </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>

            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Noriega, T. R., Tsai, A., Elvekrog, M. M., Petrov, A., Neher, S. B., Chen, J., Bradshaw, N., Puglisi, J. D., Walter, P.</span>
            <a href="" download><span id="title">Signal Recognition Particle-ribosome Binding Is Sensitive to Nascent Chain Length</span></a>
            <span id="publisher">Journal of Biological Chemistry, 2014; 289 (28): 19294-19305</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span>
                The signal recognition particle (SRP) directs ribosome-nascent chain complexes (RNCs) displaying signal sequences to protein translocation channels in the plasma membrane of prokaryotes and endoplasmic reticulum of eukaryotes. It was initially proposed that SRP binds the signal sequence when it emerges from an RNC and that successful binding becomes impaired as translation extends the nascent chain, moving the signal sequence away from SRP on the ribosomal surface. Later studies drew this simple model into question, proposing that SRP binding is unaffected by nascent chain length. Here, we reinvestigate this issue using two novel and independent fluorescence resonance energy transfer assays. We show that the arrival and dissociation rates of SRP binding to RNCs vary according to nascent chain length, resulting in the highest affinity shortly after a functional signal sequence emerges from the ribosome. Moreover, we show that SRP binds RNCs in multiple and interconverting conformations, and that conversely, RNCs exist in two conformations distinguished by SRP interaction kinetics.
                </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>

            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Tsai, A., Kornberg, G., Johansson, M., Chen, J., Puglisi, J. D.</span>
            <a href="" download><span id="title">The Dynamics of SecM-Induced Translational Stalling </span></a>
            <span id="publisher">Cell Reports, 2014; 7 (5): 1521-1533</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span>
                SecM is an E. coli secretion monitor capable of stalling translation on the prokaryotic ribosome without cofactors. Biochemical and structural studies have demonstrated that the SecM nascent chain interacts with the 50S subunit exit tunnel to inhibit peptide bond formation. However, the timescales and pathways of stalling on an mRNA remain undefined. To provide a dynamic mechanism for stalling, we directly tracked the dynamics of elongation on ribosomes translating the SecM stall sequence (FSTPVWISQAQGIRAGP) using single-molecule fluorescence techniques. Within 1 min, three peptide-ribosome interactions work cooperatively over the last five codons of the SecM sequence, leading to severely impaired elongation rates beginning from the terminal proline and lasting four codons. Our results suggest that stalling is tightly linked to the dynamics of elongation and underscore the roles that the exit tunnel and nascent chain play in controlling fundamental steps in translation.
                </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>

            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Johansson, M., Chen, J., Tsai, A., Kornberg, G., Puglisi, J. D.</span>
            <a href="" download><span id="title">Sequence-Dependent Elongation Dynamics on Macrolide-Bound Ribosomes</span></a>
            <span id="publisher">Cell Reports, 2014; 7 (5): 1534-1546</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span>
                The traditional view of macrolide antibiotics as plugs inside the ribosomal nascent peptide exit tunnel (NPET) has lately been challenged in favor of a more complex, heterogeneous mechanism, where drug-peptide interactions determine the fate of a translating ribosome. To investigate these highly dynamic processes, we applied single-molecule tracking of elongating ribosomes during inhibition of elongation by erythromycin of several nascent chains, including ErmCL and H-NS, which were shown to be, respectively, sensitive and resistant to erythromycin. Peptide sequence-specific changes were observed in translation elongation dynamics in the presence of a macrolide-obstructed NPET. Elongation rates were not severely inhibited in general by the presence of the drug; instead, stalls or pauses were observed as abrupt events. The dynamic pathways of nascent-chain-dependent elongation pausing in the presence of macrolides determine the fate of the translating ribosome stalling or readthrough.
                </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>

            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Chen, J., Dalal, R. V., Petrov, A. N., Tsai, A., O'Leary, S. E., Chapin, K., Cheng, J., Ewan, M., Hsiung, P., Lundquist, P., Turner, S. W., Hsu, D. R., Puglisi, J. D.</span>
            <a href="" download><span id="title">High-throughput platform for real-time monitoring of biological processes by multicolor single-molecule fluorescence</span></a>
            <span id="publisher">Proceedings of the National Academy of Sciences of the United States of America, 2014; 111 (2): 664-669</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span>
                Zero-mode waveguides provide a powerful technology for studying single-molecule real-time dynamics of biological systems at physiological ligand concentrations. We customized a commercial
                 zero-mode waveguide-based DNA sequencer for use as a versatile instrument for single-molecule fluorescence detection and showed that the system provides long fluorophore lifetimes with good
                 signal to noise and low spectral cross-talk. We then used a ribosomal translation assay to show real-time fluidic delivery during data acquisition, showing it is possible to follow the conformation
                 and composition of thousands of single biomolecules simultaneously through four spectral channels. This instrument allows high-throughput multiplexed dynamics of single-molecule biological processes
                  over long timescales. The instrumentation presented here has broad applications to single-molecule studies of biological systems and is easily accessible to the biophysical community.
                </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>

            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">Noriega, T. R., Chen, J., Walter, P., Puglisi, J. D.</span>
            <a href="" download><span id="title">Real-time observation of signal recognition particle binding to actively translating ribosomes.</span></a>
            <span id="publisher">eLife, 2014; 3</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span>
                The signal recognition particle (SRP) directs translating ribosome-nascent chain complexes (RNCs) that display a signal sequence to protein translocation channels in target membranes. All previous work on the initial step of the targeting reaction, when SRP binds to RNCs, used stalled and non-translating RNCs. This meant that an important dimension of the co-translational process remained unstudied. We apply single-molecule fluorescence measurements to observe directly and in real-time E. coli SRP binding to actively translating RNCs. We show at physiologically relevant SRP concentrations that SRP-RNC association and dissociation rates depend on nascent chain length and the exposure of a functional signal sequence outside the ribosome. Our results resolve a long-standing question: how can a limited, sub-stoichiometric pool of cellular SRP effectively distinguish RNCs displaying a signal sequence from those that are not? The answer is strikingly simple: as originally proposed, SRP only stably engages translating RNCs exposing a functional signal sequence.
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>

            </div>
          </div>

          <div id="year">2013</div>
          <div id="publication-wrapper">
          <div id="publication">
            <span id="author">O'Leary, S. E., Petrov, A., Chen, J., Puglisi, J. D.</span>
            <a href="" download><span id="title">Dynamic Recognition of the mRNA Cap by Saccharomyces cerevisiae eIF4E</span></a>
            <span id="publisher">Structure, 2013; 21 (12): 2197-2207</span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span>
                Recognition of the mRNA 5' m⁷G(5')ppp(5')N cap is key to translation initiation for most eukaryotic mRNAs. The cap is bound by the eIF4F complex, consisting
                of a cap-binding protein (eIF4E), a "scaffold" protein (eIF4G), and an RNA helicase (eIF4A). As a central early step in initiation, regulation of eIF4F is crucial for cellular viability.
                 Although the structure and function of eIF4E have been defined, a dynamic mechanistic picture of its activity at the molecular level in the eIF4F·mRNA complex is still unavailable.
                 Here, using single-molecule fluorescence, we measured the effects of Saccharomyces cerevisiae eIF4F factors, mRNA secondary structure, and the poly(A)-binding protein Pab1p on eIF4E-mRNA
                  binding dynamics. Our data provide an integrated picture of how eIF4G and mRNA structure modulate eIF4E-mRNA interaction, and uncover an eIF4G- and poly(A)-independent activity of poly(A)-binding
                  protein that prolongs the eIF4E·mRNA complex lifetime.
                </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>

            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author"></span>
            <a href="" download><span id="title"></span></a>
            <span id="publisher">, </span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span>
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>

            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author"></span>
            <a href="" download><span id="title"></span></a>
            <span id="publisher">, </span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span>
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>

            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author"></span>
            <a href="" download><span id="title"></span></a>
            <span id="publisher">, </span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span>
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>

            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author"></span>
            <a href="" download><span id="title"></span></a>
            <span id="publisher">, </span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span>
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>

            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author"></span>
            <a href="" download><span id="title"></span></a>
            <span id="publisher">, </span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span>
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>

            </div>
          </div>

          <div id="publication-wrapper">
          <div id="publication">
            <span id="author"></span>
            <a href="" download><span id="title"></span></a>
            <span id="publisher">, </span></div>
            <div id="more-title"><i id="triangle"class="fa fa-caret-right" aria-hidden="true"></i></i><span>More</span></div>
            <div id="more" class="hidden">
              <div id="abstract"><span id="a">ABSTRACT<i id="download" class="fa fa-arrow-circle-o-down" aria-hidden="true"></i></span>
              </div>
              <div id="webof-science"></div>
              <div id="pubmed"></div>

            </div>
          </div>
      </div>
    </div>
      <div id="designed-by">
        Design:
          <a href="https://github.com/lmadigan" target="_blank"><i class="fa fa-github" aria-hidden="true"></i></a>
					<a href="https://www.linkedin.com/in/laurenmadigan/" class="fa fa-linkedin" target="_blank"></a>
      </div>
    </div>

    <script>
      var el = $('.header');
        var $document = $(document);
        $document.scroll(function() {
          if ($document.scrollTop() >= 230) {
            // user scrolled 50 pixels or more;
            // do stuff
            el.addClass('fix-search').slideDown();
          } else {
            el.removeClass("fix-search");
          }
        });

        $(document).ready(function () {

          $(document).on("click", "div#more-title", function () {
              // $(this).next("div#more").toggleClass("hidden");
              $(this).next("div#more").slideToggle();
          });

          $(document).on("click", "div#more-title", function () {
              $(this).find("i#triangle").toggleClass("rotated");
          });


      });
			</script>
  </body>
</html>
