<!DOCTYPE html>
<html>
  <head>
    <meta charset="utf-8">
    <title>Puglisi Lab</title>
    <link href="https://fonts.googleapis.com/css?family=Open+Sans+Condensed:300" rel="stylesheet">
    <link href="https://fonts.googleapis.com/css?family=Arimo" rel="stylesheet">
    <script src="https://use.fontawesome.com/7ccc97b483.js"></script>
    <!-- <link rel="stylesheet" href="assets/reset.css" /> -->
    <link rel="stylesheet" href="assets/main.css" />
    <script type="text/javascript" src="jquery-3.2.0.min.js"></script>
  </head>
  <body>
    <div id="page-wrapper">

      <header class="header">
        <h1 id="logo"><a href="index.html">
          <div id="logo-img">
          <img src="images/Logo_of_Stanford_University.png" height="50" width="50"></div>
          <div id="puglisi">Puglisi</div>
          <div id="laboratory">Lab</div></a></h1>
          <div id="bars"><i class="fa fa-bars fa-2x" aria-hidden="true"></i>
            <div class="bars-nav" class="hidden">
              <a href="research.html"><ul id="nav-title">Research</ul></a>
              <a href="people.html"><ul id="nav-title">People</ul></a>
              <a href="publications.html"><ul id="nav-title">Publications</ul></a>
              <a href="abroad.html"><ul id="nav-title">Abroad</ul></a>
              <a href="contact.html"><ul id="nav-title">Contact</ul></a>
            </div></div>

        <div class="nav">
          <a href="research.html"><ul id="nav-title">Research</ul></a>
          <a href="people.html"><ul id="nav-title">People</ul></a>
          <a href="publications.html"><ul id="nav-title">Publications</ul></a>
          <a href="abroad.html"><ul id="nav-title">Abroad</ul></a>
          <a href="contact.html"><ul id="nav-title">Contact</ul></a>
        </div>
      </header>


      <div id="research-wrapper">

        <div id="research-item">
          <div id="research-figure"><img src="https://s3.us-east-2.amazonaws.com/puglisi-research-figures/fundamentalmechs.001.jpeg" alt="Fundamental Mechanism - Puglisi Laboratory"></div>
          <div class="research-writing" id="one">
            <div id="research-title">Fundamental mechanisms of translation initiation, elongation, termination and recycling</div>
            <div id="research-description">We are investigating the basic steps of translation initiation, elongation and termination/recycling in bacteria. We have developed single-molecule methods to track ribosomal composition and conformation in real time, which has tracked the dynamics of initiation modulated by IF2, how tRNA selection occurs, how peptide bond formation leads to ribosomal conformational changes, and how EF-G binding and GTP hydrolysis leads to translocation and efficient ribosomal conformational changes. Finally, our recent work has mapped the conformational pathway by which termination and recycling of subunits occur. Our goal is to create a coherent dynamic mechanism of protein synthesis.
</div>
          </div>
        </div>

        <div id="research-item">


          <div class="research-writing" id="two">
            <div id="research-title">Ribosomal pausing induced by RNA modifications, RNA structure and nascent peptides and proteins. </div>
            <div id="research-description">Protein synthesis is not continuous and regular, but involves pausing and stalling during elongation. Single-molecule fluorescence allows direct monitoring of elongation dynamics. We have shown that elongation pausing occurs through mRNA modifications (m6A), secondary structure (hairpins), and nascent peptide interactions in the ribosomal exit channel. The nature and duration of the paused state differs with blocks in tRNA binding or in ribosomal movement. We continue to explore the remarkable interplay of mRNA sequence, modification, structure and protein folding during translation elongation. </div>
          </div>
            <div id="research-figure"><img src="https://s3.us-east-2.amazonaws.com/puglisi-research-figures/Ribosomal+Pausing.001.jpeg" alt="Fundamental Mechanism - Puglisi Laboratory"></div>
        </div>

        <div id="research-item">
          <div id="research-figure"><img src="https://s3.us-east-2.amazonaws.com/puglisi-research-figures/FRAMESHIFTING.002.jpeg" alt="Fundamental Mechanism - Puglisi Laboratory"></div>
          <div class="research-writing" id="three">
            <div id="research-title">Recoding events, including frameshifting and bypassing.</div>
            <div id="research-description">Translational recoding, where the ribosome changes reading frame during elongation, remains an intense area of focus. We have used our single-molecule methods to track -1 and +1 frameshifting, as well as translational bypassing where the ribosome hops over >50nts of mRNA. Our results show how long translational pausing in a rotated, pre-translocation state, is a prelude to recoding driven by EF-G. </div>
          </div>
        </div>

        <div id="research-item">

          <div class="research-writing" id="four">
            <div id="research-title">Drug-ribosome interactions and mechanism. </div>
            <div id="research-description">Many antibiotics target the ribosome, binding directly to functionally important sites. We have a longstanding interest in drug-ribosome interactions, dating to our NMR studies of aminoglycoside-ribosome interactions in the 1990s.  Currently we are exploring how drug binding disrupts ribosome dynamics, perturbing energy landscapes in either global or codon-specific manner. Drug-ribosome interactions are a model for how to target RNA processes therapeutically.</div>
          </div>
            <div id="research-figure"><img src="https://s3.us-east-2.amazonaws.com/puglisi-research-figures/drug-action.001.jpeg" alt="Fundamental Mechanism - Puglisi Laboratory"></div>
        </div>

        <div id="research-item">
          <div id="research-figure"><img src="https://s3.us-east-2.amazonaws.com/puglisi-research-figures/euktranslation.001.jpeg" alt="Fundamental Mechanism - Puglisi Laboratory"></div>
          <div class="research-writing" id="five">
            <div id="research-title">Mechanism of eukaryotic translation initiation—5’end recognition, scanning, start codon recognition. </div>
            <div id="research-description">Translation initiation in eukaryotic organisms is a complex process involving >10 initiation factors that must recognize the 5’ cap structure, assembly a 40S ribosomal initiation complex that then scans to a start codon; upon start codon recognition, 60S subunit joining is catalyzed to create an elongation competent 80S ribosome. We are using single-molecule and structural methods to probe the dynamic pathways of translation initiation, as a foundation for delineating mechanisms of regulation.</div>
          </div>
        </div>

        <div id="research-item">

          <div class="research-writing" id="six">
            <div id="research-title">HCV IRES mediated translation initiation.</div>
            <div id="research-description">Despite new therapies, Hepatitis C virus (HCV) remains a major health problem worldwide. HCV is a positive sense RNA virus, whose genome is translated in the cytoplasm upon viral entry. HCV uses a structured RNA element in its 5’ genomic region to circumvent host cap-dependent translation and initiate by binding 40S subunits directly through a subset of initiation factors. We are applying structural and single-molecule methods to understand how the HCV IRES can direct translation initiation in human cells. </div>
          </div>
            <div id="research-figure"><img src="https://s3.us-east-2.amazonaws.com/puglisi-research-figures/HCV-IRES001.jpeg" alt="Fundamental Mechanism - Puglisi Laboratory"></div>
        </div>

        <div id="research-item">
          <div id="research-figure"><img src="https://s3.us-east-2.amazonaws.com/puglisi-research-figures/3'-regulation.004.jpeg" alt="Fundamental Mechanism - Puglisi Laboratory"></div>
          <div class="research-writing" id="seven">
            <div id="research-title">3’end regulation of translation.</div>
            <div id="research-description">Translational control often occurs through protein-RNA or RNA-RNA interactions in the 3’ untranslated regions of mRNAs. The current model of translation suggests that factors at the 3’end interact with initiation complexes at the 5’ end of a mRNA to control rates and efficiencies of initiation. We are exploring the mechanistic basis for this long range communication in a variety of mRNAs.</div>
          </div>
        </div>

        <div id="research-item">

          <div class="research-writing" id="eight">
            <div id="research-title">Role of translational processes in human disease.</div>
            <div id="research-description">Mistranslation of proteins underlies many diseases. For example, in ALS, triplet repeat expansions in the gene C9orf72 has been shown to cause disease. Although in a normally untranslated region, these repeats have been shown to be translated, leading to toxic protein aggregates. We are investigating the mechanism of “repeat associated non-AUG” translation of these repeat sequences, to test a hypothesis that these products are produced by an IRES-like pathway that does not require a start codon. Many other processes in disease involve translational defects and are potential targets for therapeutic intervention. </div>
          </div>
            <div id="research-figure"><img src="https://s3.us-east-2.amazonaws.com/puglisi-research-figures/role-of-translation.001.jpeg" alt="Fundamental Mechanism - Puglisi Laboratory"></div>
        </div>

        <div id="research-item">
          <div id="research-figure"><img src="https://s3.us-east-2.amazonaws.com/puglisi-research-figures/Single-molecule.001.jpeg" alt="Fundamental Mechanism - Puglisi Laboratory"></div>
          <div class="research-writing" id="nine">
            <div id="research-title">Single-molecule methods. </div>
            <div id="research-description">To probe the dynamics of RNA processes, new approaches are required. We originally used NMR spectroscopy to probe RNA structure and function. In the past decade, we have developed single-molecuel approaches by merging ribosome genetics and biochemistry, novel nanostructure (ZMWs) and instrumentation (modified PacBioRS) and improvements in photophysical behavior of dyes. We continue to develop improved methods towards our goal of real time molecular movies of complex biological systems.</div>
          </div>
        </div>


      </div>

      <div id="">
      </div>

    </div>
      <div id="designed-by">
        Design:
          <a href="https://github.com/lmadigan" target="_blank"><i class="fa fa-github" aria-hidden="true"></i></a>
					<a href="https://www.linkedin.com/in/laurenmadigan/" class="fa fa-linkedin" target="_blank"></a>
      </div>
    </div>
    <script>

    function offsetAnchor() {
        if(location.hash.length !== 0) {
            window.scrollTo(window.scrollX, window.scrollY - 130);
        }
      }
      window.setTimeout(offsetAnchor, 1);

      var el = $('.header');
        var $document = $(document);
        $document.scroll(function() {
          if ($document.scrollTop() >= 230) {
            // user scrolled 50 pixels or more;
            // do stuff
            el.addClass('fix-search');
          } else {
            el.removeClass("fix-search");
          }
        });


      var bars = $('#bars');
      bars.click(function() {

        console.log($(this).children().eq(1));
        let menu = $(this).children().eq(1);

        if ($(menu).css('display') == 'none') {
          $(menu).css('display', 'block');
        } else {
          $(menu).css('display', 'none');
        }

      });
      </script>
  </body>
</html>
